<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>PGC Video Textbook - Chapter 9.4: Interactions with Environmental Factors (Video Transcript)</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar docked nav-fixed">


<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">PGC Video Textbook</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link active" href="./index.html" rel="" target="" aria-current="page">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html" rel="" target="">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./contact.html" rel="" target="">
 <span class="menu-text">Contact us</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://pgc.unc.edu/" rel="" target="">
 <span class="menu-text">PGC Website</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item">Chapter 9.4: Interactions with Environmental Factors (Video Transcript)</li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
      <a href="./index.html" class="sidebar-logo-link">
      <img src="./pgc_logo_website_v3.jpeg" alt="" class="sidebar-logo py-0 d-lg-inline d-none">
      </a>
      <div class="sidebar-tools-main">
    <div class="dropdown">
      <a href="" title="" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label=""><i class="bi bi-github"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="">
            Source code
            </a>
          </li>
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="">
            Report a bug
            </a>
          </li>
      </ul>
    </div>
</div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./welcome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Welcome to the PGC Video Textbook!</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./toc.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Table of Contents</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Chapters</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 1: Introduction</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 2: The Genome</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter3.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 3: Technologies</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter4.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 4: Study Designs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter5.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 5: GWAS Analysis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter6.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 6: Polygenic Scores</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter7.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 7: Ancestry-Specific Analyses and Considerations</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter8.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 8: Post-GWAS Bioinformatics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter9.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 9: Advanced Topics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter10.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 10: Other Considerations</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true">
 <span class="menu-text">Software Tutorials</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_cnvs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CNVs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_conditional.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Conditional Analysis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_datasets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Datasets</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_ewas.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">EWAS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_geneset.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gene Set Identification</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_gwas.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">GWAS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_genomicSEM.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Genomic SEM</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_MR.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Mendelian Randomization</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_mtag.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">MTAG</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_prs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PRS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_correlation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">SNP Heritability and Genetic Correlation</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true">
 <span class="menu-text">Additional Resources</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Glossary</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tools/Software Resources</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="https://pgc.unc.edu/for-researchers/download-results/" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PGC Summary Statistics</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#statistics-of-the-interaction-term" id="toc-statistics-of-the-interaction-term" class="nav-link active" data-scroll-target="#statistics-of-the-interaction-term">Statistics of the Interaction Term</a>
  <ul class="collapse">
  <li><a href="#sec-video1" id="toc-sec-video1" class="nav-link" data-scroll-target="#sec-video1">Dummy Variables: Interaction Terms Explanation</a></li>
  <li><a href="#sec-video2" id="toc-sec-video2" class="nav-link" data-scroll-target="#sec-video2">Continuous Variables: Interaction Term Interpretation</a></li>
  </ul></li>
  <li><a href="#sec-video3" id="toc-sec-video3" class="nav-link" data-scroll-target="#sec-video3">Gene-Environment Interaction in Psychiatric Genetics</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Chapter 9.4: Interactions with Environmental Factors (Video Transcript)</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<hr>
<section id="statistics-of-the-interaction-term" class="level1">
<h1>Statistics of the Interaction Term</h1>
<section id="sec-video1" class="level2">
<h2 class="anchored" data-anchor-id="sec-video1">Dummy Variables: Interaction Terms Explanation</h2>
<p><strong>Title</strong>: Dummy variables: interaction terms explanation</p>
<p><strong>Presenter(s)</strong>: Ben Lambert, PhD (College of Engineering, Mathematics and Physical Sciences, University of Exeter)</p>
<p><strong>Ben Lambert</strong>:</p>
<p>So, let’s think back to our example we had in the last video. Let’s say we were interested in how wage rates varied between, let’s say, male and female individuals. So, the idea is that we regress wage on, let’s say, now that we sort of implicitly assume that we have all these other variables. We’re not going to include them explicitly. We’re just going to have alpha plus, let’s say, beta 1 times the number of years of education plus beta 2 times our sort of sex variable. Our sex variable takes on the value of 1 if the individual is female and it takes on a value of 0 if the individual is male. But then we included a further term, which was, let’s say, beta 2 3 where we multiplied sex times education. So, the idea is that we have collected all these variables across our individuals or in the population or in our sample, rather, and we have included a multiplicative term in our regression specification. So, what does this multiplicative term mean? How do we interpret this beta 3?</p>
<p>Well, let’s think about again what this average wage rate would be for a female and compare it with the average wage rate for a male. The average wage rate for a female, if they had a given number of years of education, would be alpha plus beta 1 times the number of years of education which they had, plus, well, this sex variable now takes on the value of 1, so I’ve just got plus beta 2, and now our sex variable here is taking on a value of 1 as well, so I’ve got plus beta 3 times the number of years of education. Okay, and then we can sort of simplify this. If we should have noticed that our alpha and our beta 2 are those constants here, so you’re writing those both the stars of the model, we get sort of alpha plus beta 2, let’s say. And then we recognize that we have essentially got two education terms, we’ve got this one and this one, so I could simplify these as well by writing them or by combining them. I just get beta 1 plus beta 3 times the number of years of education. Okay, so that’s for the female case.</p>
<p>What do we have for the male? So, the idea for the males in our sample is that the average wage rate is given by alpha plus beta 1 times the number of years of education because our sex variable takes on a value of 0. So, these second two terms actually cancel or don’t exist for the male. So now we can think about what the effects of our sex variable have been on our specification and our interpretation. So, what does beta 2 3 represent here? Well, beta 2 represents the additional premium which females would have over males if they had zero years’ worth of education because if they had two years’ worth of education then both of these two terms would disappear and the only difference between males and females, in fact, and when it would, in fact, actually be our beta 2. So just like we proved in the last video, that is actually the wage premium which females have over males in the case, in this case, of having zero years’ worth of education.</p>
<p>Okay, so what does beta 2 3 represent? Well, notice that the only difference between these two specifications in terms of the education variable is that essentially the partial effect of education for females has been boosted by an amount beta 2 3 relative to the males. So, what does that mean? If beta 3 was greater than zero, it means that the additional effect of one more year of education for females was, in fact, greater than that for males. If it was less than zero, then it would be the other way around. So, the additional effects of having one more year of education, on average, would tend to cause a smaller increase in female wage than it would do for males.</p>
<p>So, we can sort of think about what these cross terms mean in our regression specification. Essentially, what they mean is if I’m interacting a dummy variable with a continuous variable, it allows us to have different slopes of that particular continuous variable across the two different values which our dummy variable can take on. So, that’s quite an inappropriate assumption to make in a whole host of different situations. In this particular situation, you might suppose that there might be a different effect of education between males and females, but there are a whole host of other ways in which this could be true across other types of models.</p>
<hr>
</section>
<section id="sec-video2" class="level2">
<h2 class="anchored" data-anchor-id="sec-video2">Continuous Variables: Interaction Term Interpretation</h2>
<p><strong>Title</strong>: Continuous variables: interaction term interpretation</p>
<p><strong>Presenter(s)</strong>: Ben Lambert, PhD (College of Engineering, Mathematics and Physical Sciences, University of Exeter)</p>
<p><strong>Ben Lambert</strong>:</p>
<p>Hi there! In this video, I wanted to explain the interpretation when we have two continuous variables multiplied together in a regression model. Okay, so let’s think about a particular example. Let’s say we were trying to explain a company’s level of sales, but we’re trying to do that in terms of the effective price and the effect of advertising. Yes, this might be the company’s level of advertising spend, and this is just the company’s price set for a particular product. So, traditional theory would expect us to have a downward-sloping demand curve. We would expect beta 1 to be less than zero because if you lower the price, then sales increase. And we would expect that if we spend more on advertising, then sales tend to be higher as well. So, we’ve got beta 1 being less than 0 and beta 2 being greater than zero.</p>
<p>But let’s say we included a third term here, which is beta 3, and now included the product of price and the company’s spend on advertising. What interpretation can we actually give to this beta 3? Well, let’s think about this in two different situations. Let’s say that the company was spending a hundred thousand on advertising, and let’s think about what the company’s expected sales would be under that situation. The idea is that the company’s level of sales we would expect if advertising was 100 would be equal to alpha plus beta 1 times the price plus now we’re going to get 100 times beta 2 for this third term and then we’re going to get plus 100 times beta 2 3 times the price.</p>
<p>Okay, so what does this show us? Well, we can actually think about the effective price because price is appearing twice in our model here. We can combine the price variables to create an aggregate effective price. So here we would have the aggregate effective price would be beta 1 plus 100 times beta 3, and then that would all be multiplied by the price. One interpretation is beta 3, and what sign would we expect beta 3 to have in this case? Well, we would actually expect that beta 3 would be greater than zero. Why would we expect that? Well, the idea here is that if you spend more money on advertising, then that tends to decrease the sensitivity of your consumers to price changes in that product. Notice that this appears because beta 1 is less than zero. So if we’re adding 100 times beta 3 where beta 3 is greater than zero, then the idea is actually we have decreased the sensitivity of consumers to price changes or we have made our customers less reactive to price changes, which is something which you might expect companies to exhibit. You might expect if you spend more money on advertising, you increase the intangible brand value or the non-tangible effect which consumers consider when they’re thinking about your brand. That might make them less price-sensitive. Okay, so that’s kind of what beta 3 is representing in this case.</p>
<p>Let’s think about another example whereby let’s say we had the price level set to ten, and let’s see what we might predict the company’s sales to be in that case. The idea is that the company’s sales on average when the price was ten would be equal to alpha plus ten times beta 1 plus beta 2 times the level of advertising, which we haven’t specified. Plus now we can have 10 times beta 2 3 times the level of advertising. Notice that again here, we have two terms which essentially have the same variable. So, we can combine these. So now we have an aggregate effective advertising being denoted by beta 2 plus 10 times beta 3. What does beta 3 represent in this case? Well, remember that we found from the first example that beta 3 by theory should be greater than zero. Well, what does it say in this case? It says if your price is higher, represented by this 10 here, then the effect of advertising tends to be greater. So that might be the case. If you have a higher premium price product, you might have to demonstrate to consumers that it’s worthwhile buying, so the effect of advertising is greater than if you have, let’s say, a low-price product which consumers would flock towards anyway.</p>
<p>So, beta 3 generally, what does it mean? What have we learned from considering these two cases? It shows that the effect of price depends on the level of advertising spend, and the effect of advertising tends to be determined by or affected by the level of price. So, beta 3 is a way of adjusting the effect of price and advertising to take into account their multiplicative effects on one another.</p>
<hr>
</section>
</section>
<section id="sec-video3" class="level1">
<h1>Gene-Environment Interaction in Psychiatric Genetics</h1>
<p><strong>Title</strong>: Gene-environment interaction analysis</p>
<p><strong>Presenter(s)</strong>: Kenny Westerman, PhD (Broad Institute of Harvard and MIT)</p>
<p><strong>Kenny Westerman</strong>:</p>
<p>Yeah, so I’m a little bit daunted by the scope of presenting in general on gene-environment interaction analysis because there’s so much that can be encompassed here. But I’ll do my best to provide a broad overview and I’m happy to go into any of the topics in more depth.</p>
<p><em>Outline</em></p>
<p>All right, so to start off here, I’ll generally present this in three chunks. Here we can start with gene-environment interactions in general: what are they, why are we interested in them, and how might we go about assessing or measuring them? The second part will involve a little bit of investigation into why statistical power is a particular issue for gene-environment interactions (GxE) and talk about some of the opportunities for addressing those and improving power. Lastly, part three will be a bit of a grab bag of some additional topics that are relevant and potentially interesting for future investigation.</p>
<p><em>Phenylketonuria: a high-impact gene- environment interaction</em></p>
<p>So, I want to start with an example here, and that example is of phenylketonuria, which is known as PKU. This is an example of a high-impact gene-environment interaction. Basically, the idea here is that phenylketonuria is a disease in which the phenylalanine hydroxylase gene, responsible for converting phenylalanine to tyrosine, loses most or almost all of its function. This leads to phenylalanine buildup, which can downstream result in intellectual disability and developmental problems quite early in life, especially in babies if not addressed. It’s so impactful that it has long been a part of routine neonatal testing. This is partly because it can be treated by using a low phenylalanine diet in babies and throughout life, which can largely alleviate these symptoms.</p>
<p>The interesting thing here and the reason I call it a gene-environment interaction is that if we look at the top row of this table, we see that having a normal phenylalanine hydroxylase gene doesn’t necessarily make a difference in terms of the disease phenotype. Whether on a low phenylalanine diet or a standard diet, the phenotype remains unaffected. Likewise, in the case of a low phenylalanine diet, you will see a normal phenotype regardless of whether the gene is normal or dysfunctional. It’s only in this two-hit synergistic manner that having both the dysfunctional phenylalanine hydroxylase gene and a standard diet will result in these disease symptoms.</p>
<p><em>What is a gene-environment interaction (GxE)?</em></p>
<p>And this takes us towards the general idea of a gene-environment interaction, which can be viewed from one of two perspectives. One, I call it the genotype-centric perspective, where we’re interested in genetic effects, but those genetic effects depend on some exposure. This could be in terms of either the direction of effect or its magnitude. Likewise, we can think of it, although it’s equivalent, in an exposure-centric perspective, where we care about the effect or the association of some exposure with an outcome, but that depends on a genotype.</p>
<p>So, what does this look like in practice? Here’s my little cartoon to illustrate it. If we look at the left panel, there’s an effect of the genotype where the phenotype increases as we go from genotype A to B. The phenotype also increases as we go from environment 1 to 2. However, there’s no interaction here because these effects are independent of each other. Another way to say this is that these lines are parallel.</p>
<p>In the right panel, we have an example of a gene-environment interaction. In genotype A, we see that the difference between environment 1 and 2 is different than in genotype B, and vice versa.</p>
<p><em>Other common concepts in genetics can reflect GxE</em></p>
<p>So, just to expand this intuition just a little bit, other common concepts in genetics can reflect gene-environment interactions in a way that we might not typically think of. Other concepts that might encompass this include variable penetrance. So, we talk about how a genetic mutation is impactful in some people but not in others. That might be due to interactions with the environment.</p>
<p>When we stratify genetic associations, let’s say by sex, and we say there’s a genetic association in males that’s different than in females, this is an interaction between sex and the genetic variant. Also, pharmacogenomics, because the outcome of interest in pharmacogenomics is the response to some exposure, which is a pharmaceutical drug in this case. This can also be thought of as a gene-environment interaction.</p>
<p><em>Why do we care about GxE?</em></p>
<p>So, why do we care? In general, there are a whole host of reasons why gene-environment interactions could be interesting. First, it’s simply a better understanding of biology. If we see that a genetic variant, we think we understand how it’s acting, and it turns out that it depends on the environment, that could give really interesting clues as to refining our understanding of biology.</p>
<p>There, if we think in this genotype-centric manner that I talked about earlier, we might be interested in understanding genetic effects but in the context of the environment. Maybe this helps us explain heritability. You know, we can’t find the right genetic effects to explain heritability because they are dependent on some environmental factors. Or it might help us detect genetic associations that we otherwise wouldn’t, because again, they’re hidden in some way.</p>
<p>If we move to the exposure-centric perspective where we care about the effects of exposure being modulated by genotype, this is where we get into the realm of precision medicine. The idea is that for me versus for you, based on our genotype, we should be taking different drugs, or we should be eating different diets, or something like that.</p>
<p><em>Statistical model</em></p>
<p>So, the statistical model to set up, we can start with a standard genome-wide association study. You know, we’re going to put one that’s very straightforward. Here, we have a genotype term and we have covariates. The G by E extension is not too complicated. Essentially, we’re going to add one term, an exposure term, which is just like a covariate, and it’s going to be treated as such. Then, we have a G by E term, and this is the nuts and bolts of it, right? We have a product term, just a point-wise product between the genotype and the environment. It’s that beta G by E that’s often of interest when we evaluate such a model.</p>
<p>But once we’ve trained or fit such a model, we can then get quite a lot of things from it in some sense. So, one, we have an interaction test, and that would be evaluating this beta G by E. And we would maybe, for that, be trying to understand biology or understand, again specifically, this synergy between the gene and the environment in impacting the outcome.</p>
<p>We can also evaluate jointly, and there are statistical ways to do this. We can jointly evaluate both the gene genotype and the gene-environment interaction term, and this gives us some idea that maybe we can better detect genetic associations by incorporating both of these terms jointly than only the genetic term by itself. And likewise, though this is not a statistical test that’s often performed, if we’re thinking about precision medicine or an exposure-centric manner, we’re, in some sense, trying to holistically evaluate both the interaction term and the effect of the exposure, so that we understand, let’s say, some notion of a context-specific impact overall of the exposure.</p>
<p>Note: we can use “environment” liberally. I’ll make one note here. We can use “environment.” We’re using the term “gene-environment interaction.” For me, that’s largely historical in the way I use that term. But we can expand far beyond what we think of as an environment, let’s say, that might be pollution or social factors. But we can expand to lifestyle factors like smoking or diet, for example. We can expand to demographic factors like sex. We can expand to even physiological factors that are sometimes used as outcomes, like BMI. Those could be used as exposures in a gene-environment interaction test. Here, specifically, I’m going to focus just because of my area of expertise and in order to constrain the space of where I’m getting samples from, I’ll be focusing on specifically dietary exposures and using BMI as an outcome. But that’s by no means the limits of the scope of gene-environment interactions.</p>
<p><em>GxE discovery as intended: saturated fat, APOA2, and BMI</em></p>
<p>So, the PKU (phenylketonuria) example is a particularly stark one because we have a phenotype that’s very risky, that’s very notable, and it’s very responsive to this interaction. But in practice, we expect to find more modest effects. So, I’ll give one little vignette here of a series of studies looking into one interaction that might give a little intuition for how we maybe hope or expect that this could work in practice.</p>
<p>In a study in 2009 in the Framingham Heart Study, a group was able to look at stratifying individuals by both their saturated fat intake, which you see on the groups here on the x-axis, as well as genotype at a variant in APOE2. You see those different colored bars, the different shaded bars here, are those genotypes. They’re interested in seeing how those different strata and that interaction of saturated fat and the genotype at APOE2 influenced BMI.</p>
<p>What we can see here in this diagram is that in individuals with low saturated fat intake, this genotype, while there might be a little decrease, we don’t see a statistically significant difference in BMI. However, if we look at the high saturated fat consumers, more than 22 grams per day, we see there is a significant difference in BMI between the two genotypes (CC carriers versus everyone else).</p>
<p>They originally discovered this in the Framingham Heart Study. And here, you don’t need to look at everything for these figures. Let’s just focus on the relative heights of the bars. We can see that they were able to see a relatively similar type of pattern in the GOLDN Study, which is also in the US, as well as the Boston Puerto Rican Study, which is different ancestry individuals but also from the US.</p>
<p>We move on. Additional studies by this group and others were able to see that they could replicate again the same sort of shape of these relative bars — an increase based on genotype in high but not low saturated fat consumers — in additional populations. We have a Mediterranean population, Chinese and Asian Indians, and Iranian individuals with type 2 diabetes. So, in all sorts of ancestry, ethnicities, and even physiological states like type 2 diabetes, this interaction seems to replicate.</p>
<p>And finally, a mechanistic follow-up type study was able to be performed, and they understood that genotype and saturated fat come to mutually influence a regulatory region in APOE2 that then downstream is able to affect DNA methylation, then transcription, then affect certain metabolites, especially tryptophan and branched-chain amino acid metabolites, ultimately downstream affecting BMI. And they are looking into a clinical trial to test this interaction.</p>
<p>So, this is just an example of how we hope that interaction analysis will work. And I can step in in a moment into maybe how things often turn out in practice, but I’ll stop there for questions just at the moment. Thank you, Kenny. I don’t see any questions just yet, so I will let you know at the next break point. Sounds great. All right, getting into sort of if reality hits a little bit, we often have some statistical power difficulties in gene-environment interactions GWAS discovery.</p>
<p><em>Statistical power is a major obstacle</em></p>
<p>This plot on the left, some of you may have seen this, this is even from four years ago or so. We’ve noted that GWAS (genome-wide association study) discovery, in step with both time passing and sample sizes increasing, we see a nice increase in locus discovery. But G by E (gene-environment interaction) analysis has just lagged a little bit behind this. So, one major reason for this is that statistical power is a major obstacle to gene-environment interaction. At least in the case of a binary exposure, one heuristic we often use is that interaction detection statistically will often require almost four times the sample size compared to a main effect that has a similar magnitude of effect on the outcome.</p>
<p>And so, to get a little handle of this, we can look at just some quick ad hoc power calculations that I performed for this. If we are able to simulate an effect size of 0.05 standard deviations in some, let’s say, continuous outcome, if we want intuition for this, we could think about BMI and think about the FTO variant that is, you know, one of the best-known GWAS variants and one of the strongest that associates with about 0.4 BMI units per allele. So, we’d be talking about an interaction effect that would correspond to about half of that, let’s say 0.2 BMI units that this interaction is responsible for, in some sense.</p>
<p>And so, we can see across different minor allele frequencies, if we’re looking towards rare variants of, let’s say, 0.005 all the way up to 2.5, these are naturally given the same effect size, they’re explaining different amounts of variance and will require different sample sizes. So, if we’re conducting a GWAS to try to detect this type of sample size at a genome-wide significance threshold, for the 0.5 minor allele frequency, let’s say, maybe 34,000-ish individuals here is what’s necessary. And then we’re moving all the way up for this rarer variant. If we’re not aggregating, it might take quite a lot of individuals, even for a GWAS, almost reaching 2 million to detect this effect.</p>
<p>Now, if we extend to G by E, suddenly we’re expanding. This is not exactly a factor of four, but we need a substantially greater number of individuals based on a set of reasonable assumptions to get to the same type of power for the interaction. And we’re getting towards… here we see moving from 34,000, which is very common these days for a GWAS, we’re getting towards the need for biobank-scale population sizes in order to even identify an interaction effect for a common variant. And we’re getting towards maybe even unfeasible sample sizes for something like a very rare variant that might be, you know, the 0.005 minor allele frequency with current sample sizes that are available.</p>
<p><em>Exposure measurement can be an additional limiting factor</em></p>
<p>So, in addition beyond simply that general difficulty in statistical power, we have the additional obstacle of exposure measurement. A couple of obstacles this poses: one is that we can have imprecise or biased measurement. You know, a lot of exposures are self-reported, and this can result in both lower power and less strong interaction effect estimates. We have cultural and environmental heterogeneity resulting in more difficult replication, and we have the potential for reverse causation.</p>
<p>In addition, this could lead, not necessarily that, you know, maybe an interaction effect is equally strong, but it could lead us to conclude incorrectly if we are thinking in one direction like diet soda affects obesity, and maybe it turns out to be the opposite.</p>
<p><em>Approaches to increase statistical power</em></p>
<p><em>Variant prioritization for two-stage testing</em></p>
<p>So, taking these two obstacles into account, we have a couple of types of approaches that can be used to increase statistical power in this context. We’re going to focus on saying, “Okay, we have some genotype or genotypes, some exposures, and they’re interacting to affect body mass index.” We can focus on the genotypes or the exposures in trying to increase statistical power.</p>
<p>First, focusing on the genotypes, a very common type of approach is to use a two-stage testing in which we prioritize variants in stage one that, for some reason, we think are more likely to participate in interactions. Then, in stage two, we can test for gene-environment interactions and greatly reduce the multiple testing burden necessary. So, the most straightforward approach is to choose variants or variants from genes based on some prior biological knowledge we have—genes related to BMI, let’s use those. That’s a very simple, straightforward approach and is often done.</p>
<p>You can take one step forward and say, based on a general idea we have, we have some reason to believe that interactions will often show up maybe in known loci that are already found from a standard GWAS. This is one analysis looking at waist-to-hip ratio as a function of genes, sex, and the interaction between genes and sex. This is from one of our recent papers. What we’re showing here is a Manhattan plot for interaction p-values going up on the top and marginal genetic effects going down. What you generally see here is that almost every peak that we see for interaction also shows up as a marginal effect locus as well, indicating that maybe we can simply prioritize GWAS loci and then look for interaction effects at those loci.</p>
<p><em>Variant prioritization: variance-QTLs</em></p>
<p>Yet another idea would be the concept of Variance QTLs (VQTLs). The point here is that interactions can induce genetic associations with the variance of some phenotype. So, let’s look at the panel on the right, in B here, and point out that if there’s a genetic interaction, let’s say the genotype increases the phenotype in one environment (blue), and in the red environment, it’s actually decreasing the phenotype. This may not happen a lot in practice, but let’s suppose it does. We can then look at the actual distribution of the values of these phenotypes. Now, look to the left and see that if we were to test for a difference in variance across genotypes (AA, AG, and GG), we might actually detect that this locus is interesting, even if we don’t know or haven’t measured the red, green, blue environment in panel B there.</p>
<p>Multiple studies have looked at this. For example, one study took a look at finding these VQTLs and then ultimately looking at those VQTLs. If we find interactions and try to replicate them internally, how well do they replicate? This indicates how enriched these variants might be for actual interactions. If they take a bunch of random SNPs, they see essentially negligible replication of the interactions they find, as might be expected. If they take main effect only, like what I was talking about in the last slide, where maybe a GWAS locus is found, they see better replication. For instance, if they find a certain set of interactions, then maybe about 25% of them are replicating in this held-out internal sample.</p>
<p>Finally, if they choose VQTLs, we can see that we’re getting a lot higher replication rate here. We’re more than doubling the amount of interactions that seem to be real or that seem to replicate. So, this is quite a promising strategy. I’ll actually be talking in the main MPG talk today about one investigation leveraging this idea.</p>
<p><em>Polygenic scores for interaction</em></p>
<p>Another genotype-focused approach is to use polygenic scores, and this is probably the most straightforward of all. Here’s an example of sugar-sweetened beverages, genotypes, and BMI. This study was quite well-known from 2012. They simply constructed a BMI polygenic score, not based on interaction, just using the typical polygenic score methods. They were able to observe that on the y-axis here, we have the effect size of sugar-sweetened beverages on BMI. Across each of these three cohorts, and when we combine them, there’s this step increase. As individuals’ genetic predisposition to higher BMI increases, the effect of sugar-sweetened beverages goes from negative or essentially zero, up to a quite substantial effect. So, this is another quite promising path that’s quite often used to increase power for interaction detection.</p>
<p><em>Exposure-focused strategies</em></p>
<p>In addition, we have exposure-focused strategies. I’ll put this in two categories. First, we can think about the datasets that we actually collect. One approach is better measurement. For example, wearable technologies for physical activity, sleep, etc. These might give us better estimates of some of these behavioral components than we might get from self-report. We can incorporate more diverse populations where ethnic and cultural heterogeneity leads to differences in the distribution of environments and exposures. This can actually be quite helpful in some instances in aid discovery, especially in meta-analysis and such. Finally, we can collect longitudinal data, which will also help us better pin down estimates of the outcome and the exposure in different individuals.</p>
<p>On the analysis side, there are multiple very interesting methods cropping up related to the use of multiple environments at once. There’s the idea of a lifestyle risk score that’s come out recently from the charged gene-lifestyle interaction group. They’re trying to aggregate and say, “Well, do multiple healthy behaviors in some sort of score interact with genotype?” Likewise, another approach taken in a 2019 paper here was this idea of high-dimensional environments, where we construct an environment relationship matrix that’s similar to what we might think of as a genomic relationship matrix and use that to encompass a similarity of environments across individuals across a wide scope of these exposures. Finally, longitudinal methods in the analysis will help us better take advantage of the longitudinal data that I mentioned above to again better identify these interactions.</p>
<p>So, I will take a break here for a moment and I saw a couple questions here. Thank you very much. Yes, I will address the one that’s here and ask anyone else who has questions to please post them in the Q&amp;A. So, the question here is, “Isn’t the assumption that interactions are more likely to be found in main effects flawed because the larger variance introduced by interaction effects would lower the likelihood that a given variant reaches significance? Thus, main effect GWAS could be biased toward non-interacting loci.”</p>
<p>I think this is a really good point and something that I glossed over at the beginning when I introduced interactions and I think is a key piece here is this distinction between what we’ll often call quantitative interactions, where the genotype’s effect of the environment is just being tuned up or down. It’s more strong of an effect in one environmental stratum than compared to the other. Then, we have this idea of qualitative interactions. If we look at panel B on the left here, that’s an example. What’s being shown here is what we might call a qualitative interaction. In this particular instance, the person asking the question is totally right that in this particular instance, we’d expect a potentially strong interaction and no marginal genetic effect potentially whatsoever. What I would point out is that we don’t expect to often see this in practice. It at least intuitively seems a bit more likely that we’d find a genetic variant that’s sort of tuning the effect. For example, if we go to our saturated fat-BMI example, we might not expect that in high or low saturated fat intake, the effect of a genotype is going to totally reverse direction. We might rather expect that it’s going to be modest in one group of the population and more substantial in another group. So, I think that is a great point, and that’s why it’s useful to conduct genome-wide studies of interaction and not just neglect that. But for certain hypotheses, especially depending on your hypothesis of what the interaction might look like, it will often still be worth the trade-off to use main effect, for instance. Thank you very much. And there’s one other question before you go on. The attendee thanks you for your talk and asks, “I’d like to know how to conduct the joint test g by e plus g and the other test g by e plus e.” Yeah, the joint test specifically, there are a couple of different software tools. Our lab has particularly released one software recently called the GEM tool, and you can read about that in our paper right here, the bioinformatics paper that is in press at the moment.</p>
<p><em>Variant prioritization marginal genetic effects</em></p>
<p>So, that is one approach: to use one of a couple of different software. Some software that’s used for calculating gene-environment interactions, especially genome-wide, will be able to output directly the joint test. Another approach, though, in general, if you have, let’s say, one interaction, is that ultimately the joint test is simply a chi-squared test with two degrees of freedom. That is, taking into account both the genetic effect, its variance, the g-by-e effect, its variance, and the covariance between those two terms. So, if it’s maybe for one locus or something, you can sort of construct this yourself if need be.</p>
<p>A final thing I’ll note: there’s also a patch that my advisor Alisa Manning and her collaborator Han Chen input to METAL. Where METAL, you can use METAL in a way that performs a meta-analysis and can take both the interaction terms and the main effects, and will output this joint test as well. I’m happy to, feel free to get in touch with me after, and I’m happy to talk specifics. Thank you so much. Those are the questions for the moment.</p>
<p>All right, so here, as I mentioned before, we can step into a little bit of a grab bag here and discuss some interesting or thought-provoking questions in interaction testing that I think are worth thinking about in this context.</p>
<p><em>Interactions masquerading as main effects</em></p>
<p>So, what I’m calling interactions masquerading as main effects, and the idea here: we’re going to set up a situation where we simulate a pure interaction. That’s what was done – this was not my work, but was in the Hugo Aertsod’s paper from 2016. If we simulate a pure interaction, we don’t expect that the genotype by itself or the environment by itself is explaining any effects per se, only insofar as they participate in the interaction. But our question that we’re going to ask here, and we’re going to look at in this plot, is how much additional variance is explained by the interaction beyond what we could find by environment and genotype alone.</p>
<p>What we can see, if we make some basic assumptions here – for example, we’re going to assume that the mean of the environment is non-zero – what you can see in this plot is that as we move across the risk allele frequency spectrum, there is a large change in some of these parameters: how much variance is explained and which components are explaining it – the interaction, the genotype by itself, or the environment. But what you’ll note most prominently here is that the vast majority of the variance explained across the allele frequency spectrum is explained by things that are not the interaction. So, the idea is – for intuition – we can think about the fact that if, let’s say, as we assumed here, the mean of the environment is two, or let’s just say it’s non-zero, the interaction term becomes correlated with the gene term and/or the environment term. Because of that, we expect that performing a simple GWAS might very well uncover loci that in reality are pure interactions but we’re going to uncover them even in a test for a marginal genetic effect. This also gets back to the question that was asked earlier – about whether we expect interaction effects to show up in GWAS loci or not – this is one of the reasons why we expect that.</p>
<p>So, just to sort of summarize our takeaway at this point: we don’t necessarily expect that G by E will explain a huge amount of additional variance because of this effect that we’re seeing here to the left where GWAS will find interactions even though it’s not explicitly looking for them. So, we might not expect that incorporation of G by E is going to hugely improve prediction. But what it can do is help us identify causal players, and that’s why that’s one of the specific reasons why this interaction analysis is particularly interesting. Even if it’s not going to improve prediction so much for some phenotypes, at least we can expect that it might reveal interesting causal associations.</p>
<p><em>Additive vs.&nbsp;multiplicative interactions</em></p>
<p>Okay, so second idea here is additive versus multiplicative interactions. And so, here we’re going to set up another hypothetical example. We have an exposure that has two categories, and we have a genotype that has two categories. Just to simplify here, let’s say each of them doubles risk for some binary outcome. So, we move on to the genotype equals zero row – we move from exposure one to exposure zero to one – we double our risk. Move from genotype 0 to genotype 1 in the exposure zero stratum – double our risk. If this continues to double risk in that final bottom-right column, then we expect that the relative risk here is four.</p>
<p>Now, if this is the case, then modifying the exposure in some sense matters more in risk genotype carriers than it does in non-risk genotype carriers. If that’s the case, then we might care more about it for, let’s say, public health reasons, right? Because if we think that this is some disease, for example, and then maybe if we can find risk carriers, they’re going to generally have a greater burden of the disease. And changing the exposure, even if there’s no multiplicative interaction or there’s no interaction where we find something new biologically, it still might be more compelling to adjust the exposure if we have limited resources, something like that, in people who have the risk genotype. And so, this is what we call an additive interaction. Where if you have effects of both the genotype and the exposure, it might be of interest to understand how non-additive they are. And in this case, they’re not multiplicative. There’s no multiplicative interaction. It’s simply multiplying a relative risk of two with a relative risk of two. But that doesn’t mean that the two aren’t important for each other in determining relative amounts of absolute risk that’s being changed. This is particularly relevant for binary outcomes. There, additive and multiplicative interactions can exist. For continuous outcomes, they tend to be less relevant.</p>
<p>Just a quick note on some other ongoing areas of investigation. One is that we have whole-genome approaches to heritability and variants explained due to interactions. As I mentioned before, there are some particular considerations in thinking about whether we expect to uncover lots of new heritability. But it still might be quite interesting to get a large-scale view. Whether we know the environments, some of these methods need to know the environment, for some of them we don’t even need to know the environment at all. But there are a whole host of methods that can look in a genome-wide way at the heritability contributed to by interactions. In some sense, for rare variants, we have the same trade-off that exists in GWAS. We expect that we very well might find some higher impact rare variants. But this is going to be traded off with the fact that, as we already described, power is quite an obstacle for gene-environment interaction analysis, and even more so when we start to reduce the minor allele frequency as we saw in that simulation. So, rare variants are certainly an ongoing area of interest, but the power trade-off has to be taken into account.</p>
<p>And finally, something I just think is a lot of fun – I don’t know if this is an official method or not – but a group from the UK has looked, for a couple of different studies, into this idea of Mendelian randomization G by E, where the idea is that Mendelian randomization and G by E interaction analysis can help prove the robustness of an MR analysis. So, in our example here, if we have an MR analysis that’s proxying smoking and trying to understand a causal effect of smoking, if we can look at it from a gene-environment interaction analysis perspective, where there’s some modifying factor which is, does the population smoke at all? We’d expect that if an even genetic variant proxying smoking will not have any effect and should not affect the outcome if it’s a population where there is no smoking, for example. So, it’s just another interesting area for investigation as well.</p>
<p><em>Summary</em></p>
<p>And so, just to sum up here, gene-environment interaction analysis can be a critical building block for biological understanding as well as informing personalized medicine. I talked about that in general. The biological understanding may be from just looking specifically at the interaction term and understanding synergies between environments writ large and genotypes, as well as for informing personalized medicine. And that, again, comes from this idea of being able to look at the effect of the exposure holistically, conditional on some genotype. The statistical power, as I talked about in the second section, is the major obstacle. And we saw that that’s due to both general difficulties in power for gene-environment interactions, as well as the fact that often the exposure will not be measured as well as the genotype or the outcome. And if that’s the case, then that will further dilute statistical power. But we have a series of approaches that can help deal with that. We can prioritize variants in any number of ways: GWAS variants, variants-QTLs, etc. We can use multi-genotype approaches like a polygenic risk score to collapse genotypes into one. And we can use multi-exposure approaches; this could be like a lifestyle risk score or this could be like some of the more advanced, maybe high-dimensional environment random effect type of approaches, to understanding the impact of multiple environments interacting at once.</p>
<p>To make progress on this, a lot of these takeaways are similar to what we might say for GWAS, right? We can use larger samples; we can try to actively seek more diverse populations. And I think, unique from GWAS, is the fact that particularly in one thing we can mean by diversity here, even more than for GWAS, is that we can specifically see populations where there’s diversity of these exposures—whether sort of environmental differences in these groups. And finally, better exposure measurement will always be helpful in pinpointing the measurement and identification.</p>
<p>Thank you for your attention. Happy to take some additional questions here.</p>
<p>Thank you so much. That was really wonderful. Um, there are a couple of questions, and before I voice them, um, a reminder to everyone who’s attending that if you have any questions, please continue posting them in the Q&amp;A. Um, so the first question is posted there, and, um, it’s a question about terminology. The attendee says, ‘Are the additive rather than multiplicative interactions sometimes called gene-environment correlations rather than interactions?’</p>
<p>Yeah, yeah. So I’m aware of, although I don’t… I’m not hugely… I don’t… I’m not too familiar with the literature on this. There’s, I think, a lot, especially in the social sciences, there’s a lot of discussion of and references to this gene-environment correlation, where we’re thinking about if our exposure, let’s say, is some behavior… Let’s say it’s a behavior, um, where that can have impacts on both our… our statistical inference and how we think about the mechanism of an interaction. So that is definitely worth thinking about, and it’s something that can be a really key component to consider when you’re understanding, when you’re testing for interactions and trying to understand what’s going on. With that said, I don’t know, and I’m happy to hear if there’s an element that I’m not thinking about, but the additive versus multiplicative… I don’t know that those two pieces are necessarily correlated because, at least in the way I presented it here, if we’re thinking about this hypothetical setup, we’re not necessarily claiming that there’s any impact of the genotype on the exposure, but simply that changing one of them will have more impact on the absolute number of people, let’s say, getting some disease in one strata versus the other of exposure or genotype, etc. Got it. Great, thank you. And to the attendee, if you wanted to follow up, please, of course, post things in the Q&amp;A. I will follow up with one question also. That’s something you’ve already touched on a bit, but it’s something that really frequently occurs to me, which is that often times, even just with, you know, a simple GWAS or, you know, a just genotype-phenotype needs, there’s often varying levels of confidence in the phenotypes. So sometimes you’re absolutely certain, as well as one can be, of a phenotype, and for other individuals, you may have a decent guess, but you have some probability associated with that. And then, of course, when you add the environmental component, that adds a whole new opportunity for variation in the certainty among individuals.</p>
<p><em>Other ongoing areas of investigation</em></p>
<p>I know you’ve addressed aspects of this before. I’m wondering what your sense is if you have complete, as much as that can be the case, phenotype and environment data for a subset of individuals, and then only phenotype data for others. Is there a way to include all of the individuals, bearing in mind that the amount or the types of information is variable, or the certainty is variable among them? Or is it better just to exclude individuals that fall far below the information standards for the others?</p>
<p>Yeah, I think this is a really tricky one, and I think my mind’s going a couple different ways. One is the idea of unsupervised learning. If we’re thinking in a machine learning context, we have some labels, we have some not labels, and can we use all the data in a useful way? I think one way naively that we could address a situation like this is, let’s just say we decide half of our individuals are just straight out, we can’t use them whatsoever. For these individuals, we might still be able to use that sample in some way. When we think about some of the variant prioritization approaches that I talked about, if we think about, “Okay, let’s stratify our sample, let’s use the first half who don’t have as good exposure data,” so let’s think about, for example, variance QTLs. That’s what I was talking about. You might very well be able to take individuals from the same population but for whom you don’t know the environment. You might be able to prioritize ten percent of your variance, five percent of your variance. You could use a very liberal cut-off and just say, “Alright, we’re trying to narrow down by some substantial amount the set of variants that we’re looking at,” and then you could look at that in individuals with the exposure.</p>
<p>Another way, in general, and especially for more noisily measured exposures like let’s say self-reported physical activity or diet, something like that, an approach I’ve seen explored is to take a look at only extremes. And where we say, “You know, binarization or sort of discrete discretization of individuals where the environment isn’t measured as well, maybe could still allow us to use that information, and maybe be diluted a little bit less.” So, these are all kinds of approaches towards that.</p>
<p>Great! Thank you so much. That’s really thought-provoking. And I think we have time for one more question. This goes back to the previous question. To go back to the earlier question about traditional GWAS being biased toward main effects for any interacting variant, not just the example you mentioned, the standard deviation will be larger and thus the wider confidence intervals may impair achieving genome-wide significance for the interacting variant compared to a non-interacting variant with the same main effect. Correct? Yes, I would… Is that the question or that’s the first part? That is the question as written. And again, please follow up if you would like to ask more questions. But yes. So, I think the idea there would be, yes, we might expect that the confidence interval could be a little more difficult to detect that variant compared to a main effect variant of the same effect. But I think that the important comparison here is that GWAS power, more power is always helpful, more sample size is always helpful. But we have pretty solid sample sizes for a lot of phenotypes for GWAS, so much so that we might expect that even those that are a little more difficult to identify, that at this point we might be getting towards identifying those. And that the comparison of that with a main GWAS is less important than the comparison between that and an interaction test, where we have maybe mismeasurement of exposure, etc. And so, I think that all contributes.</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
          // target, if specified
          link.setAttribute("target", "_blank");
          // default icon
          link.classList.add("external");
      }
    }
});
</script>
</div> <!-- /content -->



</body></html>