<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>PGC Video Textbook - Chapter 1.4 Psychiatric genomics: State-of-the-science (Video Transcript)</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar docked nav-fixed">


<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">PGC Video Textbook</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link active" href="./index.html" rel="" target="" aria-current="page">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html" rel="" target="">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./contact.html" rel="" target="">
 <span class="menu-text">Contact us</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://pgc.unc.edu/" rel="" target="">
 <span class="menu-text">PGC Website</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item">Chapter 1.4 Psychiatric genomics: State-of-the-science (Video Transcript)</li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
      <a href="./index.html" class="sidebar-logo-link">
      <img src="./pgc_logo_website_v3.jpeg" alt="" class="sidebar-logo py-0 d-lg-inline d-none">
      </a>
      <div class="sidebar-tools-main">
    <div class="dropdown">
      <a href="" title="" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label=""><i class="bi bi-github"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="">
            Source code
            </a>
          </li>
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="">
            Report a bug
            </a>
          </li>
      </ul>
    </div>
</div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./welcome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Welcome to the PGC Video Textbook!</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./toc.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Table of Contents</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Chapters</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 1: Introduction</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 2: The Genome</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter3.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 3: Technologies</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter4.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 4: Study designs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter5.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 5: GWAS analysis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter6.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 6: Polygenic Scores</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter7.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 7: Ancestry-Specific Analyses and Considerations</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter8.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 8: Post-GWAS bioinformatics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter9.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 9: Advanced Topics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter10.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 10: Other Considerations</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true">
 <span class="menu-text">Software Tutorials</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_cnvs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CNVs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_conditional.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Conditional Analysis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_crossdisorder.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Cross-disorder Analysis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_datasets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Datasets</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_ewas.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">EWAS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_geneset.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gene Set Identification</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_gwas.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">GWAS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_genomicSEM.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Genomic SEM</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_imaging.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Imaging</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_MR.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Mendelian Randomization</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_mtag.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">MTAG</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_prs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PRS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_correlation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">SNP Heritability and Genetic Correlation</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true">
 <span class="menu-text">Additional Resources</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Glossary</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tools/Software Resources</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./addreading.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Additional Reading</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="https://pgc.unc.edu/for-researchers/download-results/" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PGC Summary Statistics</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#sec-psychgen" id="toc-sec-psychgen" class="nav-link active" data-scroll-target="#sec-psychgen">The Psychiatric Genomics Consortium</a></li>
  <li><a href="#sec-adhd" id="toc-sec-adhd" class="nav-link" data-scroll-target="#sec-adhd">ADHD</a></li>
  <li><a href="#sec-anxiety" id="toc-sec-anxiety" class="nav-link" data-scroll-target="#sec-anxiety">Anxiety disorders</a></li>
  <li><a href="#sec-asd" id="toc-sec-asd" class="nav-link" data-scroll-target="#sec-asd">Autism Spectrum Disorder</a></li>
  <li><a href="#sec-bipolar" id="toc-sec-bipolar" class="nav-link" data-scroll-target="#sec-bipolar">Bipolar Disorder</a></li>
  <li><a href="#sec-dementia" id="toc-sec-dementia" class="nav-link" data-scroll-target="#sec-dementia">Dementia (Alzheimer’s, Parkinson’s)</a>
  <ul class="collapse">
  <li><a href="#alzheimers-disease" id="toc-alzheimers-disease" class="nav-link" data-scroll-target="#alzheimers-disease">Alzheimer’s Disease</a></li>
  <li><a href="#genetics-of-frontal-temporal-dementia" id="toc-genetics-of-frontal-temporal-dementia" class="nav-link" data-scroll-target="#genetics-of-frontal-temporal-dementia">Genetics of Frontal Temporal Dementia</a></li>
  <li><a href="#lewy-body-dementiaparkinsons-disease" id="toc-lewy-body-dementiaparkinsons-disease" class="nav-link" data-scroll-target="#lewy-body-dementiaparkinsons-disease">Lewy Body Dementia/Parkinson’s Disease</a></li>
  </ul></li>
  <li><a href="#sec-eatingdisorders" id="toc-sec-eatingdisorders" class="nav-link" data-scroll-target="#sec-eatingdisorders">Eating Disorders</a></li>
  <li><a href="#sec-mdd" id="toc-sec-mdd" class="nav-link" data-scroll-target="#sec-mdd">Major Depression</a></li>
  <li><a href="#sec-ocd" id="toc-sec-ocd" class="nav-link" data-scroll-target="#sec-ocd">Obsessive-Compulsive Disorder</a></li>
  <li><a href="#sec-ptsd" id="toc-sec-ptsd" class="nav-link" data-scroll-target="#sec-ptsd">Post-Traumatic Stress Disorder</a></li>
  <li><a href="#sec-schizophrenia" id="toc-sec-schizophrenia" class="nav-link" data-scroll-target="#sec-schizophrenia">Schizophrenia</a></li>
  <li><a href="#sec-sharedgenetic" id="toc-sec-sharedgenetic" class="nav-link" data-scroll-target="#sec-sharedgenetic">Shared Genetic Architecture</a></li>
  <li><a href="#sec-sud" id="toc-sec-sud" class="nav-link" data-scroll-target="#sec-sud">Substance Use Disorder</a></li>
  <li><a href="#sec-suicide" id="toc-sec-suicide" class="nav-link" data-scroll-target="#sec-suicide">Suicidal Thoughts and Behaviours</a></li>
  <li><a href="#sec-tourette" id="toc-sec-tourette" class="nav-link" data-scroll-target="#sec-tourette">Tourette Syndrome</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Chapter 1.4 Psychiatric genomics: State-of-the-science (Video Transcript)</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<hr>
<section id="sec-psychgen" class="level1">
<h1>The Psychiatric Genomics Consortium</h1>
<p><strong>Title</strong>: PGC: Future Perspectives</p>
<p><strong>Presenter(s)</strong>: Patrick Sullivan</p>
<p><strong>Gerome Breen</strong>:</p>
<p>It’s a pleasure to introduce Patrick Sullivan. Pat was the founding chair of the Psychiatric Genetics Consortium (he still is the chair), and does work across a number of different psychiatric disorders. Pat’s track record speaks for itself, but Pat is currently Jurgen Distinguished Professor of Psychiatry in Genetics and Director of the Center for Psychiatric Genomics at UNC. He also is a professor at the Karolinska Institute in Sweden. So, with that, I’d like to hand you over to Pat to introduce PGC future perspectives.</p>
<p><strong>Pat Sullivan</strong>:</p>
<p>Very good, Gerome thank you very much, and thank you to all the other conference organizers for making this happen under short notice. I think this is literally the first time I’ve ever done a professional talk with my knees open to the air. And I’m gonna turn off my video here, because I’m just talking basically there’s nothing to see, and we have dodgy home Wi-Fi sometimes, and this might make it smoother.</p>
<p>Okay, so Gerome has asked me to discuss the PGC from the perspective of the future, and in order for us to understand the future we need to go back quite a wee bit. And so part of the reason, one of the sort of the scholarly motivations for the PGC, was in fact the reproducibility crisis that many of us thought was a real problem. So, for example, in this paper we looked at the historical candidate genes for schizophrenia and few actually have held up over time. This is a paper that Matt Keller drove that did a similar thing for major depression, and again, a lot of things we thought we knew before 2008 actually have not stood the test of time, including, in fact, some some very high-profile papers that that have received a lot of attention over time.</p>
<p>And so that was part of the the impetus for the PGC, from an intellectual perspective, from a scholarly perspective, from a practical perspective I should say, we started the Psychiatric GWAS Consortium in 2007 at the World Congress meeting in New York City, and the initial impetus were the four Foundation of NIH GAIN studies that were funded in 2006, and then we sort of spun those into a more comprehensive effort. I show below the the major funding streams for the PGC: there’s the PGC1, 2, and 3 grants, we’re currently in 3. And since 2012, in fact, we’ve had a lot of pharma being members of the PGC. We have historically been very open to their participation, and they’re related PGC grants for PTSD, anxiety disorders, eating disorders, and a new major depression ECT grant.</p>
<p>I’ll start with what our impacts have been over time, and I think one of the key things, is that we actually have sort of assembled a global workforce. This is showing the global location of authors on PGC papers, and the size of each of the yellow balls refers to the number of authors. So, as you can see, there’s been, you know, it’s a substantial number of authors from all over the place. Second, another thing about the PGC is that we’re data-driven. That’s in terms of the genetic architecture, as well as the phenotype. So here, we show the different aspects of the PGC. So right now we’ve got the Coordinating Committee, which I chair, and then there’s working groups for many different psychiatric disorders: ADHD, Alzheimer’s, anxiety disorders, ADS, autism, bipolar disorder, copy number variation, there’s a cross disorder group which has done some really interesting things and I believe Jordans Smoller will be talking about that later in this conference, eating disorders, major depression, OCD, Tourette’s, PTSD, schizophrenia, substance use disorders, and we have a new trans ancestry group, and then of course we have a bunch of other sort of things that are helping.</p>
<p>But this gives us a example of where we are right now for schizophrenia. The risk allele prevalence is on a log10 scale here at the bottom and the log10 of the genotypic relative risk. And these are all the common variants that we found. Currently there’s well over 270 of them. These are all common, a relatively soft effect. In addition, we have multiple copy number variants that have been identified, that are actually rare, or quite rare, but a relatively strong effect. There’s an increasing number of rare variants that are coming out of exome sequencing. This is the SCHEMA study that TJ Singh and Marc Daly are leading. There’s nine genes that pop up as being significantly associated with schizophrenia, <em>SETD1A</em> is the one that’s been out there for a long time, beginning a paper that was a paper that TJ did when he was a grad student at Sanger. And there’s been so much stuff about the missing heritability, and it’s basically probably right here. There’s this great paper from Peter Visscher where they just they recover basically all of the pedigree heritability of height and body mass when you actually get into things that are in this range that aren’t captured well by our GWAS chips. And then this of course is the CNV paper from the PGC. So, we certainly look at common variation, but we also look we’re very open to rare variation of all types, and we also are agnostic as to phenotypes.</p>
<p>We’ve been quite productive. This is a graph of the number of papers from 2008 to 2019: a total of 320 papers by the end of last year. And this is the impact using the NIH eyesight tool. This is the median citation for NIH grants in a particular scientific area, and these are the eyesight percentiles for each of the PGC papers. The median paper pops in right here, around the 85th, 86th percentile, and the top paper, which is the 2014 Nature paper on schizophrenia, that actually is one of the most cited papers of that year, and one of the most cited papers ever by the NIH, NIMH I should say. And in terms of a bit more breakdown, we got 3 Cell papers, 3 Nature papers, 26 Nature Genetics papers etc. If you consider these as a career, the PGC as an h-index of 65, and 59 of our papers are listed as being highly cited in Web of Science.</p>
<p>Some of the highlights are here. Again, the 2014 Nature Paper. Recent autism, depression, anorexia papers, bipolar, 2 Cell papers from last year, the ADHD paper, and then the Nature Neuroscience cover, which I really like this diagram of beer glasses with the bubbles being basically a Manhattan plot. We also have a strong outreach effort. So we’ve, for example, worked with Wikipedia (it’s surprisingly difficult to work with actually), to actually update the genetic sections of the key disorders. A lot of people go here for their first pass; we wanted to make sure they went with current knowledge. We have a YouTube channel for for the PGC, and you can go there and look at recent things: we’ve had a bunch of stuff on on coping and leadership in the COVID era.</p>
<p>We do aggressive outreach. This is a Scientific American piece on the the anorexia nervosa paper. This is the altametrics board, and this I think is the highest number any PGC paper has gotten, it got a huge amount of attention. We have active Twitter feed etc and so we do a lot of the typical outreach activities. But probably the key thing is that the PGC has allowed a wide network of scientists to bring sustained attention onto this problem. And this is looking at the patterns of co-authorship on PGC papers. Each of the light arrows shows one co-authorship, and as you can see, we have a dense collection of people that work together, that talk together, from around the world to try to make this go further. There’s a lot of stuff that’s happening in in East Asia right now, which isn’t on here because I made this graph before some of the key papers started appearing, but that’s part of the conversation as well. Well probably the key thing, going back to the reproducibility crisis, that I think we have is rigor and reproducibility are really, really deeply part of our DNA. We want the stuff that we come up with to stand the test of time, and largely, with few if any exceptions, that’s been the case so far.</p>
<p>So now I’m going to turn toward the future. So the PGC4 is our next iteration of the project: innovation and improvement. And I start with this graph, which shows, again, the world, and what you’re seeing here is the color scale is log10 number of cases. And so there’s a lot of cases from the United States, Canada, and in almost all countries in Europe, especially the UK, Sweden, Germany etc. There’s increasing numbers from China, some from Russia, Australia is doing well, New Zealand even, but as you can see, there’s, you know, if we were to overlay the world population here, it’s very clear that we’re missing huge amounts of information on people from Africa, from South Asia, and certainly from Mexico, Caribbean region, Central America, and South America, and that’s a major problem. And this has been pointed out in a couple of papers. This is a PGC paper. This is by Marc Daly and his colleagues, who are, of course, closely aligned with the PGC.</p>
<p>And so, in terms of the planning process, so in 2019, we had an extensive set of things that we did to get ready for this. We began with an extended brainstorming session among the coordinating committee. We had conversations for their funders. We moved to planning within groups consultations with our PIs. We did a survey of the membership. The opening plenary at World Congress in Anaheim last October was on this. And then we had a worldwide lab meeting on the proposed aims toward the end of last year.</p>
<p>So, the big ideas that emerge from all these discussions were, of course, to continue our core business, but we also needed to increase our sample sizes by getting people of more diverse ancestry, it’s not just European, and also, how we were going to work carefully with large biobanks. One of the things that I think is really important is the PGC can do things that others can’t . Large biobanks have difficulty finding the the most severe people with illnesses like schizophrenia: we can. And then, we can also, of course, frame and ask the important clinical questions, because, we’re you know, this is our field, and this is our academic area, and we I think we know these questions better than many others do. In addition, for precision psychiatry, there’s a bunch of things that we can and that we should be doing right now. Mike Owen and I wrote a paper on this in American Journal of Psychiatry last month, and you can see that for all the details, I won’t discuss it further. We, of course, collaborate with other consortia particularly psychENCODE, which is doing a ton of really fascinating functional genomic studies in brain. We work with whole exome sequencing and whole genome sequencing consortia for psychiatric disorders, as well as the ExAC and gnomAD groups, and, of course, we put a lot of time and effort in developing infrastructure analysis and tools.</p>
<p>So, our application went in in early February: seven sites, twenty-one sub-awards, so a total of 28 different groups that are involved. 14 PIs, 21 co-investigators, and again, we really needed to both continue our core business, but also to adapt and to expand. So on there are five aims. The first aim is basically core business. So we want to increase discovery by doing GWAS meta-analyses for 11 disorders; we’ll continue doing what we’ve been doing. We’ll carefully incorporate biobank study data, and, because some of the phenotypes that come out of biobanks are actually relatively light by our standards, we want to make sure we use the best ones we can, and it’s possible to do that if you have a good dialogue with the people doing it.</p>
<p>Increasing ancestral diversity, of course, is important, and I point to, as an example, the Nature Genetics paper that came from the PGC East Asian European group on schizophrenia. And the fascinating finding there was that the genetic correlation for schizophrenia between East Asian and European samples was basically one. So, in other words, the common variant genetic architecture is, for all practical purposes, identical in East Asia and in Europe. And I thought that was really cool, and that also really supports the worth of what we’re doing.</p>
<p>Now with a bit more detail. This shows where we are right now in terms of the number of cases. This includes pre-publication stuff. The number of cases in our analyses and the number of associations. Schizophrenia in unpublished stuff is up to 270ish, and depression is up to over 100 with a published paper for Andrew MacIntosh’s group. Bipolar is well over 50. But, you can see here, and this is a figure from the grant, that this is the current number of cases, number of non-European, number of loci currently, and these are our conservative projections for 2025, where we have marked increases in the number of cases across all disorder groups, as well as increases in the proportion of people of non-European ancestry. Over all of these, we anticipate 2.5 million cases in 2025. All of these will have GWAS in PGC analyses. So, I think we’re on track to do some pretty amazing things over the next five years. And then basically, what this is meant to show is that there are multiple ongoing projects where we’re attempting to sort of ramp up the number of cases from from Hispanic populations, LatinX populations, both in the US, as well as Mexico, Central, and South America, multiple projects in Africa, some of which are large-scale, multiple projects in South India, and, of course, many, many things going on in in East Asia. And so, we’re hoping that these ongoing efforts will help us get to where we need to be.</p>
<p>The second aim is talking about across architecture. So, how do we actually look at common and rare stuff together? Well, the first thing, obviously, is to continue and to extend the work of the CNV group. They’ve been working super hard on this, with a great platform and pipeline about how to do this. There’s emerging call sets from multiple different groups right now, and those papers are underway. And that, of course, needs to be continued and expanded. There’s been an ongoing effort, led by Michael Gill and then Aidan Corbin in in Dublin, where we actually find really large pedigrees from across the world using our network of clinicians, that are densely affected or strange in some way, and then do whole genome sequencing within an attempt to find rare variants. There’s a number of such papers that are underway right now: some really interesting stuff coming out of that, and we’ll continue that of course, but we’ve been doing this for or five years now and continuing makes sense. And one of the key things that happens right now is when, if people have all these data types, they write three papers: there’s a GWAS paper, and then a CNV paper, and then whole exome sequencing paper, and those were all separate efforts. Increasingly, I don’t think that makes sense. I think we need to find ways to integrate, it should be about, not about the technology we use, which is the way these papers are currently written, but rather, about the problem. And I think this is something that we’re going to come up with ways to to try to integrate these three different types of information in order to really do much much better job at understanding the architecture and even pinpointing specific genes.</p>
<p>The third is that I think it’s it’s quite clear from the the Brainstorm paper that was in Science a few years ago, led by Ben Neale and colleagues, that psychiatric disorders are not writ cleanly from a genetic perspective. And I think we need to do several things to to push that along. We’re going to develop phenotyping instruments that will be used widely by as many samples as we can, so we can actually do transdiagnostic studies across the world. We’re also going to take genetics, and use it to answer what are currently unanswered questions, particularly patient stratification: can we find subsets of people that differ in important ways? What about the genetics of therapeutic outcomes? Increasingly, we’ve got data on that. Are there few versus many genetic factors underlying clinical presentations? What about varying genetic effects over the lifespan? And one of the things many of us are are pivoting toward is to basically see what happens when human beings are exposed to a universal, unusual stress (COVID-19 pandemic obviously). And one of the things that we can do, that I think few other people can, is we can actually use our global network for new studies of highly severe and/or treatment resistant cases. This is a way to enrich for causative alleles, increase power, and also to increase the translational potential.</p>
<p>So, a couple of examples of things that I think we can do that large biobank studies can’t. A bunch of years ago, we determined that postpartum depression was a more heritable form of major depression. And we launched a large effort, and we’re currently doing a GWAS to 17,000 cases, that’s being finalized, and we also had this new thing called MomGenes, where we actually have active sample collections, through the mail, of people who had postpartum depression. I’ve always loved the CLOZUK study by our colleagues in Cardiff. Basically, these are individuals with schizophrenia getting clozapine, and they’ve done a whole range of really interesting stuff on this. We have this study, the Pennsylvania State Hospital study, where we have 500 people with schizophrenia who have ultra treatment-resistant psychosis. These people are actively treated, and have been hospitalized for five plus years, and nothing works. The first person in the study has been essentially actively psychotic, despite everything, for many decades. We just got funded for the Gen-ECT study, and so essentially, these are the cases or individuals getting ECT for depression. We’ve shown that the SNP heritability is much higher and that these people inherit higher depression genetic risk scores. This is an NIH grant with Peters Zandi, at Johns Hopkins, and me, and this is also part of a PGC global consortium run by Bernard Baune, and our goal is to get 25,000 cases genotyped over the next few years. Another example would be Cindy Bulik’s work, and actually identifying people with severe and enduring anorexia. These are individuals who basically have been sick for a long time, no therapy works, and yet they remain quite actively ill.</p>
<p>Novel therapeutic and preventative opportunities: so the first thing is basically we have to go deeper in the biology with experts. And so, there’s a fair overlap between the PGC investigator team sets, and psychENCODE, and we have now a formal relationship with them. There’s a working committee that interfaces in both directions, and one of the the leaders of psychENCODE is actually a PI or a co-PI, co-investigator, on the PGC grant. Mendelian randomization is something that we can use intensively, we have experts in this, and especially, as the studies get larger, we have a whole aim on that. And, of course, you know, how do we use epidemiology with polygenic risk scores to predict important outcomes and to enable clinically meaningful patient stratification? And then finally, we have an outreach/impact thing, and so we basically want to have engagement with medicine, academic, and industry medicine discovery groups. This call, this, the Seoul conference, obviously, will do more outreach by digital media, education resources, you know, we need to get an SAB so we can maximize our alignment with multiple end users, including people from biotech, including people from pharma.</p>
<p>Okay, so let’s talk about pathways to therapeutics. So the first thing that the PGC may offer in this regard is that our work is explicitly pre-competitive. Our idea is to bring the best minds together, and to try to understand what these data are telling us about the nature of these beasts, and the therapeutic opportunities they may represent. In addition, I would argue for the things that we do, we really do have a large cast of characters who are often best in class, and that’s something else the PGC can bring to offer in terms of the complex trait genomics, in terms of phenotype definitions, etc. Many of the people in the PGC are actually the best that there’s ever been in these efforts. And, of course, this is, by its nature, a community effort. Second thing is target identification. So, our results, you know, especially when we get to the point of pinpointing things, you know certainly can lead to identifying specific targets that might be approached with, you know, various ways to understand, you know, molecular modification of that target and what that target does. But you have to bear in mind that targets, in fact, may be specific brain cell types. We’ve had two papers on this: the one just came out a day or so ago in Nature Genetics, where we basically connected GWAS results to cell types in brain, and this was the first paper and so basically do theses connections of schizophrenia, for example, to specific cell types and brain, and that may in fact be the output of the the readout of GWAS. Julian will be talking about this in his talk on Thursday and I should also note that if any of you are hiring, take a look at this guy he’s awesome.</p>
<p>The other question that we have is: what’s the disease? And the the more I think about it, the more I look at these data, that once we get to the severe end of the spectrum, that the most severe cases with schizophrenia, bipolar disorder, and depression, for instance, there’s more similarities than differences. And so, that may actually be the core phenotype that we need to target. And when we look at the milder versions, then you might see things that tend to be a bit more disease-specific. That’s an emerging idea that we’re working on, and we hope to quantify that more precisely soon; that’s something we can do. Patient stratification is often critical to Pharma. Are there subsets, definable subsets, of people that have differential responses to therapeutics? This is a key interest of ours, and this would also be of important interest for indications from medications, for how to how to do clinical trials more efficiently, etc. And then finally, many individuals, in biotech in particular, are trying to develop therapeutics for small subsets of patients. So, if you think about schizophrenia +22 q11, you are literally in orphan disease territory. And actually accruing enough patients with these strange combinations of large CNVs and a disorder can be difficult, but the PGC can often be a very key way to find at least an initial entre into those individuals.</p>
<p>Now the last two slides: editorial comment to start with, about, if I may. It is an absolutely fascinating time to be in this area. That the more I see what various companies are doing; it’s super super interesting novel approaches, a lot of vigor, a lot of smart people in this, funding, new technology, etc. And that’s awesome, and that’s the way it should be. We want our information to be used that way. However, I would also point out that the PGC needs our Pharma colleagues to help. We need more cases in our analyses, we need more information to help drive these, we need more clinically-relevant samples in our in our analyses. And this is something that we’ve not paid as much attention to in the past, but it’s something that we’re going to have conversations about and pay attention to in the future. But we certainly and with respect we’d ask our colleagues to assist as they can.</p>
<p>And then finally, my thanks. So these are the last names of 802 people in the PGC, and it’s been an amazing effort, and I’m fortunate to have been involved in the ways that I have been. It’s been really amazing to see this move, and that’s it for my talk</p>
<p><strong>Gerome Breen</strong>:</p>
<p>Thank you very much Pat that was an awesome overview. So if people could please post questions to the Q&amp;A box. We already have one question here. It’s from an anonymous attendee: How is the PGC planning to increase the ethnic diversity of the participants?</p>
<p><strong>Pat Sullivan</strong>:</p>
<p>Yep, that’s a great question. So there’s a bunch of things that have come together. The first is, I think, the Stanley Center at the Broad Institute saw this coming some years ago, they have active efforts ongoing in China, in Africa, and in Mexico. Where they’re doing a lot of work to sort of upscale efforts there. On top of that, we’re overlaying other efforts. So in East Asia, there’s a lot of really good academic groups that have been collecting and genotyping samples. Hailiang Huang has done a fantastic job with actually bringing them together with the output of the Nature Genetics paper on schizophrenia and East Asia that came out last year. That work is continuing and advancing. In India and in Pakistan there’s multiple efforts ongoing to try to come up with large sample sizes. So, for example, I’m aware of one group that has, you know, that has insane degrees of throughput, where you know, they see 10,000 patients a year, and they do 50 or 60 ECTs every week. And so, these are other examples. We got a diversity supplement on the PGC grant to try to pull together LatinX populations: so Puerto Rico, Mexico, Central America, South America. That’s ongoing. We also need to develop, one of the things I’m hoping to do during this weird time, is to develop basically a seminar series that will be delivered by zoom, where we can actually begin to train people up as they come in. What’s the PGC? How does GWAS work? How did the analysis work? All that kind of stuff. So we’re hoping to get more than that.</p>
<p><strong>Gerome Breen</strong>:</p>
<p>Okay, that’s great. We have another question: How does the PGC combine epidemiological and polygenic modeling?</p>
<p><strong>Pat Sullivan</strong>:</p>
<p>Sure, so this comes down to, there’s a handful of groups in the world that actually have both. And there have been a bunch of papers that have come out on this more singly, and what we’re trying to do is to basically bring it together in a more rigorous way. So, for example, there’s a group of people in the Nordic countries that are doing this systematically, where they’ve got relatively large samples with GWAS, as well as inherited a whole bunch of register-based information that’s systematically collected in the population. That’s one example. It really has to do with, you know, finding the people that actually have the information and pulling it together. Not every study does. Some are just sort of a one-and-done case control approach, but increasingly, there’s large cohort studies that actually have a lot of information that we can bring to bear</p>
<p><strong>Gerome Breen</strong>:</p>
<p>Okay and another question’s come in: What would you be working on if you were just starting out right now?</p>
<p><strong>Pat Sullivan</strong>:</p>
<p>It’s a great question. So I think mentioning diseases of the eye is probably the wrong thing to say here. If I had to do it over again, I’d probably go to the eye. I think the eye is really cool. In psychiatry, I think there’s a couple of things. First, I would become friends with the people that share the groups, and then volunteer. Offer to do things. Be around. Get stuff done. Be selfless. That’s a way to get deeper into that effort. In addition, I think another thing that you can do, and this is laid out in the article that Mike Owen and I had, especially if you’re a clinician, is to start to specialize on the psychiatric manifestations, but particularly longitudinal treatment, of people with large copy number variations. This is something we’re going to see more and more of, and I think this is something that deserves much more attention. We need a work force on it too.</p>
<p><strong>Gerome Breen</strong>:</p>
<p>Okay we have a few more questions coming in. So from Vera: Is PGC investing in within-family study designs for the future? And do you think such an investment will produce, will prove productive, for psychiatric disorders?</p>
<p><strong>Pat Sullivan</strong>:</p>
<p>Yeah, I mean there are some groups that are doing this a fair bit. Especially, you know, for childhood disorders: autism, ADHD, OCD, for instance. I think, typically though, I think for a lot of our information, for a lot of the studies we do, a case-control is probably as good, or case-control with longitudinal data. Exception, of course, is if you’re looking for variants that are, you know, rare variants, and strong effect that are <em>de novo</em>, obviously having parents is important. But I think increasingly we see things moving toward, you know, case-control collections, cohort collections over time. If, obviously, if one can get family data, if one can get family data, then one should. But I think typically we’re moving in other directions than that, at least for adult disorders.</p>
<p><strong>Gerome Breen</strong>:</p>
<p>Okay, so we have just couple minutes left, so I would just pick another question: Mohammad Suleman asks: Is PGC focusing on using consanguinal family models to enhance genetic predictions?</p>
<p><strong>Pat Sullivan</strong>:</p>
<p>Yes, in part. So, Jim Knowles I believe has, there’s a big project in Pakistan. I don’t know everyone involved. I know Jim Knowles is involved with it, I’m not sure who’s the lead, it may well be Jim. But basically, they’re doing a large collection in Pakistan right now for exactly that purpose. I think this is something a model that, if one knows how to do it, I think this would be certainly very interesting to look at.</p>
<p>So I can answer a couple of questions. So the missing heritability question. Read Peter Visscher’s paper on BioRxiv. It’s about height, BMI, etc. It should be coming out somewhere big soon. That would be, basically, you have to use whole genome sequencing to get there.</p>
<p>Epigenetics. Yes, so there’s a lot of work that we’re doing with epigenetic marks. A lot of it’s done in the context of psychENCODE, but others of us have done studies of that as well, so yes, definitely.</p>
<p>Open Science framework, yeah. So we’ve got a big effort on reproducibility. One of the things is, and we’ve always made our summary statistics available freely, the problem is that getting access to individual data is complicated, mainly because the EU, for example, defines it as personal data. And we have to - it’s possible - but it takes work to get there.</p>
<p>Nature nurture in play, absolutely. I mean that’s part of the whole gene environment or that the modeling stuff we mentioned. You have environmental risk factors with genetic risk scores, and we’ve had a couple of papers on this already, and I think it’s certainly something to to work on.</p>
<hr>
</section>
<section id="sec-adhd" class="level1">
<h1>ADHD</h1>
<p><strong>Title</strong>: Insights into Attention-Deficit/Hyperactivity Disorder from Genetic Studies</p>
<p><strong>Presenter(s)</strong>: Joanna Martin, Christie Burton, Isabell Brikell, Nina Roth Mota</p>
<p><strong>Joanna Martin</strong>:</p>
<p>Hello, my name is Joanna Martin and I’m a research fellow at Cardiff University. This talk is about “Insights into ADHD from recent genetic studies” and is based on a review article that I’ve recently co-authored with Christie Burton, Isabell Brikell, and Nina Roth Mota. I’ll first start with some definitions and epidemiology related to ADHD, and then the bulk of the talk will focus on looking at converging findings from family and twin research, as well as molecular genetic studies. And then I’ll conclude with some summary implications and look at future research directions.</p>
<p>ADHD is a common childhood-onset neurodevelopmental disorder. Typically, it is diagnosed before the age of 12 years or symptoms are present before the age of 12 years. It is characterized by developmentally-inappropriate and impairing levels of symptoms of inattention and hyperactivity and impulsivity. And these symptoms need to be present across settings. So typically in children they’ll be present at home and in the school environment. ADHD is diagnosed using either the DSM-5 or the ICD-10. ADHD is common across the globe and across the lifespan. In children, the prevalence is about 5.3 percent, and boys are about four times more likely to be diagnosed with ADHD than girls. Many children with ADHD continue to experience symptoms that persist into adulthood, and the prevalence of ADHD in adults is about 2.5 percent. There is a similar ratio of ADHD in men and women with ADHD. Comorbid conditions are very common in ADHD. These include neurodevelopmental, psychiatric, and physical health conditions. On the bottom left plot, you can see that a variety of neurodevelopment and psychiatric conditions are increased in children with ADHD, and there are some sex differences in this. Similarly in adults, psychiatric conditions, as well as some somatic conditions like type 2 diabetes and hypertension, are increased in adults with ADHD compared to adults without ADHD. And the impairment of ADHD, along with these co-occurring conditions, does have a huge impact on the lives of individuals with ADHD and their families. So understanding the etiology of this is essential in order to be able to help individuals affected with ADHD.</p>
<p>One of the key emerging findings from genetic studies is that ADHD is highly heritable and polygenic. Several decades of twin studies, summarized in this graph here, suggest that the relative proportion of genetic factors far outweighs the contribution from environmental factors to ADHD risk. The summary of the heritability estimate from twin studies is between 70 to 80 percent, suggesting that these genetic factors are really important. Recent molecular genetic studies have begun to identify specific genetic risk factors associated with ADHD. A genome-wide association study has identified 12 risk loci, summarized here using the top single nucleotide polymorphism at each locus, which are spread across the genome. We also know from copy number variant studies, that large, rare, structural deletions and duplications of whole sections of DNA, known as CNVs, are also associated with risk of ADHD. But these genetic risk loci are really just the tip of the iceberg. If we look at this manhattan plot from the recent genome-wide association study, we can see that although there are these 12 risk loci which we have statistically significant evidence for, there are many risk loci that we do not have the statistical power yet to detect, but will likely be very important. And collectively, these common variants have a SNP-based heritability of about 22 percent. There is this gap you can see between the twin heritability and the SNP-based heritability, suggesting that there will be other genetic risk factors, both common and rare, that are yet to be discovered for ADHD.</p>
<p>But we can already gain many insights into ADHD based on these emerging genetic findings. First of all, we can tell that ADHD is indeed best classified as a brain-based neurodevelopmental disorder. When we partition the heritability from the common genetic variant analyses, we can see that there is evidence of enrichment in the central nervous system tissues, particularly various brain tissues. Although twin studies do not directly speak to the biological underpinnings of ADHD, we can see that there’s a significant genetic correlation between ADHD and various neurophysiological measures of brain based activity, such as EEG measures. Common genetic variants also show a genetic correlation with total intracranial or brain volume. Although this is a modest negative genetic correlation, it does suggest that brain is obviously very important with regards to ADHD pathophysiology. Another line of evidence supporting ADHD as a neurodevelopmental disorder is the fact that it shares quite a large proportion of genetic risk with other neurodevelopmental disorders, including autism spectrum disorder and developmental delay, which I’ll talk about a little bit further.</p>
<p>Another key finding emerging from genetic studies is that clinically diagnosed ADHD shares genetic risk with population traits of ADHD. It’s been long considered that ADHD is the extreme end of an underlying distribution of continuous traits in the population. We use this binary cut off to indicate who has a diagnosis, and who needs treatment, but really if we drew this line elsewhere we would find that ADHD is similarly heritable across different potential definitions of ADHD, supporting this underlying distribution of genetic risk. The estimate of genetic correlation between diagnosed ADHD and sub-threshold symptoms of ADHD is moderate at about 0.56, and genetic correlation analyses suggest that this is even higher for these common variants, close to 0.97, for ADHD diagnosis and symptoms of ADHD in the general population.</p>
<p>ADHD also shares genetic risks with many of those comorbid conditions that it co-occurs with. A recent systematic review and meta-analysis has found that there are moderate genetic correlations between ADHD and various psychiatric symptoms, both in childhood and adulthood, and, using various assessment methods to measure the symptoms, these genetic correlations extend across neurodevelopmental, psychiatric, and somatic phenotypes according to a whole host of emerging studies in the last decade. Some of the strongest genetic correlations are between ADHD and major depressive disorder and autism spectrum disorder, and if we look at the recent genome association study results, the genetic correlations between ADHD and other phenotypes are vast. They include measures of schooling, like educational attainment, various psychiatric measures, but also measures beyond psychiatric and brain-based traits to things like obesity, cancer, insomnia, rheumatoid arthritis, many somatic conditions that often do co-occur. And these shared genetic risks potentially speak to why ADHD is so commonly co-occurring with these conditions.</p>
<p>Another key finding that is emerging is that we cannot necessarily consider environmental risk factors as causal for ADHD without exploring potential shared genetic risks. Studies have long observed that smoking during pregnancy is associated with risk of ADHD in a child. However, maternal smoking during pregnancy is not much more strongly associated than paternal smoking during pregnancy, indicating that there might not be intrauterine effects at work here. A clever <em>in vitro</em> fertilization design examined the risk of ADHD in the offspring of mothers who were related to them, and were smoking during pregnancy, versus mothers who were unrelated to their offspring through the IVF design, and were smoking during pregnancy. And this study found that the association between smoking during pregnancy and offspring ADHD was only present in the related mothers, and not in the unrelated mothers, suggesting that there’s genetic confounding. And many other designs have also been used: for example, comparing siblings, to identify that there is genetic confounding at play. This is also further supported by the recent GWAS, where smoking-based traits are genetically correlated with ADHD, suggesting that smoking during pregnancy may not be a causal risk factor for ADHD in general, but it could just be that there are shared genetic effects that increase risk of ADHD and smoking, either during pregnancy or not, and that these genetic effects are passed on by mothers who smoke during pregnancy. And this suggests caution in trying to interpret potential environmental risk factors for ADHD. This isn’t to say that there are no environmental risk factors for ADHD, just that the weight of evidence needs to be sufficient to deem a factor causal.</p>
<p>Another key finding emerging from the recent studies of genetics of ADHD is that genetic factors are very important across development. Twin studies suggest that there is stable heritability across childhood, adolescence, and early adulthood, and that a large proportion of these genetic effects are actually stable across time as well. The recent genetic correlation analyses from GWAS suggest that ADHD in children and ADHD in adults shows a high genetic correlation. However, there’s some evidence that the persistent trajectory of symptoms of ADHD in children in the general population could be linked to a higher genetic burden compared to symptoms that are low, intermediate, or just limited to childhood. And further work is needed to fully understand how persistence of ADHD links shared genetic risks.</p>
<p>Finally, the last finding I’ll talk about is the fact that there are shared genetic risks between ADHD in males and in females. Early twin and family studies have suggested that there could be a differential burden of genetic risk in males and females. Because girls are less likely to be diagnosed with ADHD, they might require a higher genetic burden to have the diagnosis of ADHD, and several twin and family studies have identified that the relatives of girls who are affected with ADHD are at higher risk than the relatives of affected boys with ADHD. However, quantitative and qualitative twin-based analyses have not found any strong evidence of sex-specific genetic effects, so there are no differences in the heritability of ADHD in males and females. Common genetic variant analyses suggest that the genetic correlation between male and female ADHD is very high, not distinguishable from one, and this supports the idea that common genetic variants are the same common genetic variants are linked to ADHD in males and females. And polygenic risk score and copy number variant studies suggest that there’s no difference in genetic burden in ADHD boys and girls, and this does not support the previous family studies. However, there’s emerging evidence to suggest that the differential genetic risk in family studies may be linked to other factors, for example diagnostic and ascertainment biases.</p>
<p>So to summarize, emerging genetic studies suggest that ADHD is very highly heritable and polygenic. It is best categorized as a brain-based neurodevelopmental disorder, and the diagnosis appears to share genetic risks with traits of ADHD as well as various comorbid conditions. It also seems to share genetic risks with some factors that could be considered to be environmental risk factors. And we know that genetic factors are important for ADHD across development, and are shared between males and females. These genetic studies give us the potential to leverage these large data sets of people with ADHD and population samples to really understand more about how ADHD symptoms link in with co-morbid and related conditions and symptoms. And this gives us potential to increase the sample size for genetic discovery and downstream analyses. The broad shared genetic risks across disorders and with environmental risks have also got implications for designing studies and interventions, and just a note of caution about ascribing causality in these associations. Although genetic studies do not support binary cutoff between presence or absence of ADHD, of course symptoms of ADHD are insufficient for a diagnosis, there are other clinical features, including impairment from symptoms and pervasiveness across settings, which need to be taken into consideration. So considering ADHD symptoms for research purposes and recent genetic research studies is very well supported, but of course the clinical cutoff is still important for treatment decisions.</p>
<p>There are many unanswered questions, and I won’t go through all of these, so feel free to pause the recording to read through them, but the one thing I’ll say is that the emerging large genetic studies of ADHD and childhood and adulthood populations really provide us with the tools that we need to be able to address some of these questions, and begin to understand even more about ADHD and how it relates to other comorbid phenotypes. Thank you very much for listening and I’d just also like to thank again my co-authors Christie, Isabell, and Nora, and also Barbara and Ben who are the co-chairs of the Psychiatric Genomics Consortium ADHD working group. Thank you.</p>
<hr>
</section>
<section id="sec-anxiety" class="level1">
<h1>Anxiety disorders</h1>
<p><strong>Title</strong>: Genetic contributions to anxiety disorders: Where we are and where we are heading</p>
<p><strong>Presenter(s)</strong>: Helga Ask, Rosa Cheesman, Daniel Levey, Kristin Purves, Heike Weber</p>
<p><strong>Kirsten Purves</strong>:</p>
<p>Hi! We’re a group of six early-career researchers on anxiety, who have come together to write a review on the current state of play for anxiety genetics. One of our numbers, Eisha(?), isn’t available for this recording, but the other five of us are going to take you through the main points.</p>
<p>So anxiety is characterized by feelings of unease, tension, and worry, alongside physiological arousal and anticipation of threat or ambiguity. Anxiety is interesting in that it’s a universally experienced thing, and it can even be adaptive, but when it’s experienced disproportionately, intensely, or for a prolonged period of time, it can be really harmful maladaptive. This is particularly troubling because anxiety disorders are amongst the most common class of mental health disorders, with a global lifetime prevalence of 16%. They have a particularly early age of onset, and a chronic course across the lifetime.</p>
<p>In additional to dimensional experiences of anxiety, the DSM-5 describes seven anxiety disorders. Two of these, selective mutism and separation anxiety, are specific to childhood. The remaining five, generalized anxiety disorder, panic disorder, specific phobia, social phobia, and agoraphobia, can be experienced throughout the lifetime. All of these, with the exception of social phobias, are more commonly experienced in women than men. Now, we know that anxiety is complex. It’s influenced by a combination of biological and environmental factors, with genetic influences being amongst the best substantiated risk factors. The leading etiologic model for anxiety to date is the diathesis stress hypothesis. This suggests that genes and environmental stressors both independently and in combination increase an individual’s liability to developing disorders. Now, environmental factors that have been associated with risk for anxiety include low socioeconomic status, parental conflict, and childhood maltreatment. It’s really important to remember that environmental factors are themselves somewhat heritable, and genetic expression might be shaped by the environment and by life experiences via gene environment interplay. Thus, associations between environmental factors and anxiety might reflect effects of genetic and environmental confounding rather than underlying causal relationships.</p>
<p><strong>Daniel Levey</strong>:</p>
<p>As Kirsten just mentioned, anxiety is the most common mental illness, even having been treated as a discrete diagnosis only relatively recently in the DSM-3. The volume of these diagnoses contributes to the costs on healthcare systems, which is further exacerbated by very high persistence across the lifespan, as well as an early onset of diagnosis. Together with depression, which frequently presents comorbid with anxiety, anxiety disorders cost an estimated 90 billion dollars in personal health care costs within the United States alone. But there’s also substantial costs in human health. Anxieties that present frequently co-morbid with depression, as i just mentioned, about fifty percent of the time that there’s an anxiety disorder with a depression diagnosis as well. There’s substantial overlap also with stress related disorders, such as post-traumatic stress disorder, which is formerly considered an anxiety disorder into the DSM-4, and obsessive compulsive disorders as well. Substance use disorders are also often comorbid with anxiety diagnoses. They also often present an overlap with somatic conditions such as epilepsy and irritable bowel syndrome.</p>
<p>There can also be a substantial impact on mortality. You’ll often see a comorbid diagnosis with heart disease and with cancer, and when these co-occur, particularly also when anxiety and depression co-occur with a cancer diagnosis or heart disease, you’ll see poor outcomes in disease. And it can be complicated to diagnose anxiety disorder in someone suffering from a cancer diagnosis or heart disease, as someone with a life-threatening condition might have a normative response with anxiety or depression. There are treatments available in the form of psychotherapeutics, pharmacotherapies, and there’s cognitive behavioral therapy, which has shown some effectiveness in anxiety disorders, but they’re also some concerns about treatment responses. Only half of adults, and one-third children, will respond to existing treatments, and few ever remain in remission. There’s also many with anxiety disorders that don’t receive treatment. Untreated anxiety disorders have particularly bad outcomes, with impaired general functioning, social relationships, and employment, that can last for decades. Improving both access and treatment efficacy is essential for anxiety disorders.</p>
<p><strong>Rosa Cheesman</strong>:</p>
<p>Twin studies provide an elegant natural experiment to investigate the relative influences of nature and nurture on anxiety disorders. By comparing identical and non-identical twins, we can disentangle the proportion of variation in anxiety disorders that’s due to genetics, shared environment, and non-shared environment. Shared environmental influences are those that increase the resemblance among family members, whereas non-shared environmental influences are individual-specific and those that make family members different from one another. For anxiety disorders, genetic influences are moderate; heritability estimates range between 20 to 60. Environmental influences are primarily non-shared, although shared environmental effects are often detected in child anxiety. So, in addition to this core finding of the heritability of anxiety, twin studies have shed light on at least three other fascinating topics related to anxiety. First of all, they’ve shed light on the genetics underlying the comorbidities that anxiety shares, so the genetic correlation with depression is nearly 100, which suggests that anxiety and depression are different manifestations of very similar genetic risk. Secondly, twin studies have informed us about how anxiety manifests across the life course and why, in fact, anxiety disorders tend to have their onset in childhood. Twin studies show that continuity in anxiety from childhood through to adulthood is predominantly driven by a core stable set of genetic factors. Thirdly, twin studies have revealed interesting examples of interplay between genetic risk for anxiety and the social environment. For example, a well-replicated finding is that genes involved in anxiety in young people tend to evoke elevated levels of maternal control. Genomic tools provide new exciting opportunities to extend twin study findings, including about gene-environment interplay.</p>
<p>Heritability of anxiety disorders is mainly based on a large number of genetic variations with only small effect sizes. Such variations are often not restricted to a specific anxiety disorder subtype; they more often influence a variety of different anxiety disorders, also explaining the high comorbidity rates between them.</p>
<p>Aiming to uncover the complex genetic background of pathological anxiety, several genomic association studies have been performed on different anxiety disorders. However, due to small sample sizes and the high genetic heterogeneity of diagnosed anxiety phenotypes, the maturity of results has been negative or not reproducible.</p>
<p>To overcome diagnostic boundaries and sample size limitations, recent genome-wide association studies started analyzing the different anxiety disorder subtypes together or used dimensional anxiety symptoms. In doing so, four large and well-powered genome-wide association studies have been performed, including the AXE Consortium, the Danish National Register Study, iPSYCH, the 1 Million Veteran Program, and the UK Biobank. These studies reached sample sizes ranging from 18,000 up to more than 190,000 individuals. Together, these studies led to the identification of 15 genome-wide significant associated loci, of which 12 mapped to different gene regions.</p>
<p>However, to further uncover the genetic makeup of anxiety disorders, more non-white association studies in large, well-powered sample sizes, and on well-defined anxiety phenotypes are necessary. The first GWAS from the Anxiety Work Group in PGC is currently underway and will be presented at this conference in the future. Large-scale consortia collaborations and methodological developments in molecular genetics hold great promise for the future of anxiety genetics research.</p>
<p>Guided by findings from quantitative genetics, we expect novel findings on the genetics across the lifespan and on possible gene-environment interplay. Core features of anxiety disorders will likely take the field of anxiety genetics in a different direction from that of other psychiatric disorders. The early age of onset and high chronicity highlight the need for studies among young people and across development.</p>
<p>Additionally, the phenotypic complexity of anxiety and its subtypes, and the high level of shared etiology with other disorders and traits, speak to a need for genetic investigations on underlying or intermediate phenotypes as well as on related phenotypes.</p>
<p>The results of anxiety GWAS opened the door for a range of new possibilities for research. One flexible approach is the use of polygenic scores. The richness of longitudinal cohort studies like ELSBack and MOBA provides a unique possibility to map how genetic risk manifests across development, to study the gene-environment interplay, and to estimate indirect genetic effects across generations.</p>
<p>Genetic loci and polygenic scores can also be used to investigate causal relationships using Mendelian randomization. Genetic variation predates any behavioral or even neural variation. Understanding the underlying genetic architecture of anxiety and associated phenotypes will pave the way to a better understanding of the complex downstream relationship between genes, brain behaviors, and environments.</p>
<p>The field of anxiety genetics is at a pivotal point of this understanding. We hope that future work will build on existing research to bridge the gap between disorder prevention and treatment. Thank you so much for listening.</p>
<hr>
</section>
<section id="sec-asd" class="level1">
<h1>Autism Spectrum Disorder</h1>
<p><strong>Title</strong>: Genetic contributions to Autism Spectrum Disorder</p>
<p><strong>Presenter(s)</strong>: Alexandra Havdahl</p>
<p><strong>Alexandra Havdahl</strong>:</p>
<p>This presentation is a summary of a review paper of the most recent developments in human genetics research in autism spectrum disorder. We are a team of six researchers who came together to write the review and made equal contributions to the work. Three of us will present the main points of our review: myself, Alexandra Havdahl, Maria Niarchou, and Anna Starnawska.</p>
<p>[Leo] Kanner defined autism in 1943, with detailed case descriptions of children showing social aloofness, communication impairments, and stereotype behaviours and interests, often accompanied by intellectual disability. A year later, [Hans] Asperger independently published an article on children presenting marked difficulties in social communication and repetitive behaviour patterns, despite having advanced intellectual and language skills. It was only three decades later that [Lorna] Wing and [Judith] Gould united Asperger and Kanner's descriptions and conceptualised a spectrum of autistic conditions. It's worth highlighting that a woman named Grunya Sukhareva published a paper describing autism in 1925, two decades before Kanner and Asperger, but seems to have been forgotten.</p>
<p>The core defining features of autism remain early childhood onset impairments in communication and social interaction, alongside restricted and repetitive behaviours and interests. There's a wide variability in core symptoms, language level, intellectual functioning, and co-occurring difficulties and conditions.</p>
<p>Prevalence estimates of autism have steadily increased from less than 0.4% in the 1970s to current estimates of 1 to 2%. The increase is largely explained by broadening diagnostic criteria to individuals without intellectual disability and milder impairments, and not environmental factors. There are marked sex and gender differences in autism. The male-to-female ratio is approximately 4:1 in clinical and health registry cohorts, but closer to 3:1 in general population studies with active case-finding. The mechanisms underlying the sex difference are mostly unknown and hypotheses include a female protective effect, prenatal steroid hormone exposure, and social factors, like underdiagnosis and misdiagnosis in women.</p>
<p>In 1977, the first twin heritability study was published. A recent meta-analysis of seven primary twin studies reported that the heritability estimates range from 64 to 93%. Family studies have found that the relative risk of a child having autism relates to the proportion of shared genome with affected relatives. As you can see here in the graph, the relative risk increases as relatedness increases from half first cousins to full siblings.</p>
<p>Early GWAS [genome-wide association studies] of autism were underpowered, partly due to overestimating potential effect sizes. Grove and colleagues conducted a large GWAS of autism combining data from 18,000 autistic individuals and 27,000 non-autistic controls recently. They identified five independent GWAS loci. Another recent study identified the further novel locus by meta-analysing results from Grove et al.&nbsp;with the SPARK [Simons Foundation Powering Autism Research] cohort. The sample sizes are still relatively small compared to other psychiatric conditions, such as schizophrenia or depression, and ongoing work aims to double the sample size.</p>
<p>Using genetic correlations and polygenic score analysis, studies have identified shared genetics between autism and different definitions of autistic traits in the general population. These methods have also identified polygenic associations between autism and other neurodevelopmental and mental conditions, such as schizophrenia, ADHD [attention-deficit hyperactivity disorder], major depression disorder, and related traits, like neuroticism, tiredness, and self-harm, as well as risk of exposure to childhood maltreatment and other stressful life events. Notably, autism is genetically correlated with higher intelligence and educational attainment, unlike many other psychiatric disorders.</p>
<p>Polygenic transmission disequilibrium tests, or pTDT, have identified an overtransmission of polygenic scores from parents to autistic children. pTDT is a family design which is robust to population stratification and several other factors which may bias genetic correlations. It involves comparing an affected child's polygenic score to their unaffected parents' average polygenic score, called the mid-parent PRS. The pTDT deviation indicates the average standard deviation difference in PRS in children compared to their parents.</p>
<p>Here is an example from Warrier and Baron-Cohen's study where they found a higher PRS for self-harm ideation and behaviour in autistic children compared to their parents, shown here in red. Whereas this pTDT deviation was not observed for unaffected siblings, here shown in blue. Weiner et al.&nbsp;previously reported evidence of overtransmission of PRS for autism, schizophrenia, and higher educational attainment to autistic children but not to their unaffected siblings.</p>
<p>Maria Niarchou: Thank you, Alex, I'll take it from here. Rare genetic variants confer significant risk into the complex aetiology of autism. They're typically non-Mendelian, with substantial effect sizes and low population attributable risk. It is estimated that about 10% of individuals with autism have been diagnosed with an identifiable rare genetic syndrome and, as you can see in the graph, the prevalence of autism in associated syndromes varies widely. On the x-axis are the rare genetic syndromes, and on the y-axis is the percentage of patients with a syndrome that are diagnosed with autism. The largest whole exome sequencing analysis to date was conducted by the Autism Sequencing Consortium, published in <em>Cell</em> this year. They [researchers] identified 102 autism-associated genes which in our review we have mapped to the autosomal chromosomes, here in red. In blue are the five SNP names identified in the largest genome-wide association study by Grove and colleagues. Notably, <em>KMT2E</em> was implicated in both the largest GWAS and exome sequencing analysis. It is hypothesised that common genetic variation in or near the genes associated with autism influences autism risk although current sample sizes lack the power to detect the convergence of the two. It's important to note that, similar to high genetic correlations found for common polygenic risk for autism with other neurodevelopmental and neurological traits, autism-associated rare variants are also associated with risk for other conditions, including intellectual disability, schizophrenia, ADHD, and epilepsy.</p>
<p>Damaging single nucleotide variants, or SNVs, include protein-truncating and missense variants, very substantial case enrichment of de novo protein-truncated variants and missense variants, and, in total, all exon de novo SNVs explain about 2% of the variance of autism liability. Rare inherited SNVs have a smaller average effect size and reduced penetrance compared to de novo pathogenic mutations and there is little difference in the overall rate compared with unaffected siblings.</p>
<p>A lot of rare variant research in autism has focused on copy number variants, or CNVs. They can impact one or multiple genes and can occur at common or rare frequencies in populations. All CNVs associated with autism have been rare today. Approximately 4 to 10% of individuals with autism have de novo deletions or duplications, frequently mapped to established risk loci. A higher global frequency of de novo CNVS is observed in idiopathic autism cases from simplex families compared to multiplex families and controls. Recurrent or inherited CNVs are among the most convincing rare inherited variations for autism and are found in 3% of affected individuals. However, inherited CNVs can be present in unaffected siblings and parents, suggesting a model of incomplete penetrance dependent on the dosage sensitivity and function of the genes they [CNVs] affect.</p>
<p>Common and rare genetic variants associated with autism are related to heterogeneity in intellectual functioning and, while higher SNP heritability is observed in individuals with autism without intellectual disability, de novo protein-truncated variants in constrained genes are enriched in individuals with autism and intellectual disability. However, the genetic architecture of autism is complex and diverse. For example, common genetic variants contribute significantly to risk in individuals with autism and intellectual disability and in individuals carrying known large effect de novo variants in constrained genes. Also, an excess of disruptive de novo variants is also observed in individuals with autism without co-occurring intellectual disability, compared to individuals without autism.</p>
<p>Based on the most recent developments in human genetics research in autism, we have updated a pie chart showing the proportions of variance explained in autism liability by different classes of genetic variants. The narrow-sense heritability in the different shades of green sums up to 83% being estimated using familial recurrence data. 12% has been identified as common inherited variants, and 3% as rare inherited [variants]. 17% of the variance in autism liability is left to be explained by shared and unique environmental estimates which include non-additive and non-inherited factors. Identified de novo missense and protein-truncated SNVs and variation in non-genic regions together explain up to 3% of the variance. Additionally, non-additive variation accounts for about 4% of the total variance.</p>
<p>Anna Starnawska: Thank you, Maria. As mentioned by my colleagues, both genetic and environmental factors contribute to the risk of autism. DNA methylation, currently the best studied and well-understood epigenetic modification, allows for both genetic and environmental factors to modulate a phenotype. DNA methylation affects gene expression, regulatory elements, chromatin structure, and alters neuronal development, functioning, as well as neuronal survival. The largest EWAS [epigenome-wide association study] of autism was performed in blood in almost 3,000 individuals. It did not identify any genome-wide significant differentially methylated sites. However, elevated autism PRS was associated with differentially methylated positions in <em>FAM167A</em> gene and <em>RP1L1</em> gene. The EWAS of autism performed in postmortem brain tissues reported autism-related co-methylation modules to be significantly enriched in synaptic, neuronal, and immune dysfunction genes. However, these studies were commonly performed on relatively small sample sizes and further replication studies are warranted.</p>
<p>As opposed to the epigenome-wide association approach, several studies utilised methylation quantitative trait loci [mQTL] approach to explore epigenetic contributions to autism. These studies confirmed that common risk variants of autism, identified in work of Grove and co-authors, can act as mQTLs, also across tissues like blood, foetal brain tissue, and adult brains. Through mQTL analysis, several new genes were associated with autism risk, not identified before by the GWAS approach. Bayesian colocalisation analysis confirmed that genetic risk variants of autism are associated with both autism diagnosis and variation in DNA methylation, therefore supporting the hypothesis that autism genetic risk variants can act through DNA methylation to mediate the risk of the disorder.</p>
<p>As for the transcriptomic studies, gene expression measures play a key role in determining the functional consequences of genes and identifying [the] genetic network underlying the condition. Whole-genome transcriptomic studies have identified several pathways, on which different genes associated with autism seem to converge. Three pathways have been consistently reported, the synaptic connectivity and neurotransmitter, chromatin remodelling, and neuronal projection pathways, as visualised on this slide.</p>
<p>Recent large-scale and internationally collaborative investigations have led to a better understanding of the genetic contributions to autism and ongoing work is likely to lead to significant advances in the coming years. Findings show that the genetic architecture of autism is complex, diverse, and context-dependent, highlighting a need to study the interplay between different types of genetic variants, identify genetic and non-genetic factors influencing their penetrance, and better map the genetic variants to phenotypic heterogeneity within autism. Immense collaborative efforts are needed to identify converging and distinct biological mechanisms for autism and subgroups within autism, which can in turn inform treatment. It is crucial to invest in multidimensional and longitudinal measurements of both core defining traits and associated traits, such as language, intellectual, emotional, and behavioural functioning, and also, to establish large omics databases, including genomics, epigenomics, transcriptomics, proteomics, and brain connectomics. Already, large-scale multi-omics investigations are becoming possible in the context of population-based family cohorts, with rich prospective measures of environmental exposures and multidimensional developmental trajectories, from birth to adulthood. Finally, methods, such as Mendelian randomisation, can help investigate causal molecular pathways between genetic variants and autism.</p>
<p>We would like to thank you for listening and thank the Psychiatric Genomics Consortium, Psychiatric Genomics Consortium Working Group for Autism, and chairs Elise Robinson and Anders Børglum for the opportunity to write this review and for their support and advice. Thank you.</p>
<hr>
</section>
<section id="sec-bipolar" class="level1">
<h1>Bipolar Disorder</h1>
<p><strong>Title</strong>: Genetics of Bipolar Disorder</p>
<p><strong>Presenter(s)</strong>: Brandon Coombes, Kevin O’Connell</p>
<p>Hi everyone, I’m Brandon Coombes and today I’ll be talking about the genetics of bipolar disorder, and this is work that I’ve done with Kevin O’Connell and myself for the early career researcher reviews papers.</p>
<p>So, to start this off the definition of bipolar disorder: bipolar disorder is marked by shifts in mood energy and activity levels. You can see here on the right is a plot of what this might look like for someone who has bipolar type one in which this person started off having a severe depression episode and then that slowly transitioned into something that’s more normal and then they transitioned to mania and then eventually went back down and that happened over a certain amount of time maybe months maybe years and that’s what a bipolar disorder could look like.</p>
<p>Bipolar disorder has high clinical heterogeneity and that might mean that people with bipolar disorder can look very different based on their clinical presentation. So, people might have BD (bipolar disorder) type 1 or type 2 – BD type 1 will reach the full manic state, type 2 doesn’t quite reach that full manic state and might only come to hypomania. Type 2 is also more likely to experience the depressive symptoms.</p>
<p>People can have rapid cycling and bipolar disorder and some people don’t -- rapid cycling is talking about this time frame down here below. Some people are cycling back and forth fast, and some aren’t, and rapid cycling is defined as having three or more episodes in a year.</p>
<p>Some people with bipolar disorder have melancholic features, atypical features, there is psychosis in bipolar disorder and that psychosis can either happen in depression or it can happen in mania. There’s also catatonia, peripartum onset, and seasonality to the disorder. So, some people might experience depression during winter um so there’s a certain seasonality aspect. In general, there’s a sort of transition here of what these different mood states could look like as you move along the range here (range referring to severe depression to mania mood states)</p>
<p><strong>Epidemiology</strong></p>
<p>So, speaking of epidemiology here -- there’s a lifetime prevalence of one percent for each type, bipolar type 1 or type 2, and bipolar disorder is ranked 17th in global burden disease. It has a mean age onset of 20 years. And typically, early age of onset - which we define as usually in your teenage years of developing bipolar disorder - that’s associated with poor prognosis and increased comorbidity.</p>
<p>Bipolar disorder also has a potential for misdiagnosis in early onset. So once people are first developing bipolar disorder, depressive symptoms can look like MDD (major depressive disorder) at first if they haven’t experienced mania. Also, if their first time coming in is during a psychotic episode they could be misdiagnosed as having schizophrenia. There are also cases that are misdiagnosed with ADHD, or reverse that ADHD is misdiagnosed as bipolar disorder. So, there’s a lot of misdiagnoses, and as I mentioned before there’s a high comorbidity of bipolar disorder -- at least 90% of people with bipolar disorder have at least one lifetime comorbid disorder and 70% of people bipolar disorder have three or more. Some of these comorbid comorbidities include anxiety disorders, which make up a majority of what the comorbidities are, substance use disorders, ADHD, personality disorders, and eating disorders.</p>
<p><strong>Genetics</strong></p>
<p>So, moving on to the genetics, we’re talking about the genetics of bipolar disorder. Bipolar disorder is highly heritable. So, in fact it’s 60% to 90% heritability, and here on the right you can see bipolar disorder is lined up with all the other different psychiatric disorders and it tends to be towards the highest psych heritability among all the psychiatric disorders. Here you see either heritability measured from twins or heritability measured from the SNPs. BD also has a high genetic correlation with other psychiatric disorders. In this paper that was published last year, it was shown that bipolar disorder has the largest genetic overlap with schizophrenia, and it also has a pretty large genetic overlap with depression, but it also has overlap with all of these other different traits such as anorexia, OCD, ADHD, and autism.</p>
<p><strong>GWAS</strong></p>
<p>The biggest breakthroughs for bipolar disorder have really come through the through GWAS (genome-wide association studies). When GWAS first started for bipolar disorder in 2007 through the Wellcome Trust there were only two thousand cases, three thousand controls, and zero genome-wide significant loci were found in that original GWAS. And that’s where we saw that we’re going to need a much larger sample size to find anything that’s associated with bipolar disorder, even though this is such a highly heritable trait.</p>
<p>And now after the latest GWAS was just posted on MedRxiv, the latest GWAS has over 40,000 cases and over 350,000 controls and now we have 64 genome-wide significant loci - so we’ve made quite a bit of improvements as we’ve moved along through the years.</p>
<p>But still, only 15% to 18% of the variance is explained in this latest sample. But fortunately, we’ve now reached an inflection point in bipolar disorder. So here on this graph on the right we can see here bipolar disorder is here with our recruitment and it’s expected that as we move along, we will now start to increase how much variance we can explain as we add more and more people. We’ll need to add not as many as we have had to before.</p>
<p>So, we actually are missing a lot of the very heritability here - only 15% to 18% of the variance is explained, but bipolar disorder has a high heritability - so that’s what we deem as the “missing heritability problem”. And in the next few slides I’m going to go over the different hypotheses of missing heritability and one of the common ones is that we’re actually leaving out genetic interactions.</p>
<p><strong>Genetic Interactions</strong></p>
<p>Genetic interactions being gene-environment or gene-gene interactions, but so far in bipolar disorder these have been understudied as you might imagine, and they’re not well replicated. So, unlike the main effects for GWAS, gene-environment interactions or gene-gene interactions usually take around four times the sample size to get the same sort of significance that you would in GWAS. So, we’re going to need much, much larger sample sizes and on top of that you’re going to have to measure for environments, you’re going to have to measure the same environments in all those samples. So, it’s very tricky getting gene-environment reactions measured for bipolar disorder, which is why it’s been so understudied. For gene-environment interactions, just to list off a few, there’s been some with toxoplasmosis, child abuse, stressful life events, cannabis. And there’s also been a couple gene-gene interactions, but all of these have had not very much replication.</p>
<p><strong>Missing heritability</strong></p>
<p>There’s also the hypothesis for missing heritability that rare variation is what could be driving some of the heritability. And with that [rare variants] its [requires] whole genome (WGS) and whole exome sequencing (WES) [but] because of the prohibitive cost, it’s usually been limited to small studies. And I’ll talk about in the future directions that that could be changing soon. But right now, it’s only been limited to small studies. These studies usually just have large pedigrees so that you have large families that can concentrate the amount of rare variants to increase the power to detect rare variants.</p>
<p>There has been evidence that there’s rare burden in bipolar disorder, but it’s nothing to the degree to which schizophrenia has [shown]. So, it seems like rare variants might have a role in bipolar disorder, but it’s not going to be as large of a role as we’ve found in schizophrenia. And the same is true for CNVs (copy number variants). So CNVs - it does appear that they do have a higher frequency of CNVs in bipolar disorder than controls, but it’s less than that than it observed in schizophrenia and also neurodevelopmental disorders and there’s really only been one replicated CNV associated with bipolar disorder.</p>
<p><strong>Pharmacogenomics</strong></p>
<p>Moving this along to clinical impact of all these genetic findings, is moving into pharmacogenomics. Bipolar disorder is typically treated with mood stabilizers, antidepressants, or antipsychotics. Pharmaceutical pharmacogenomic studies have been limited in bipolar disorder, but they are growing in size. The largest GWAS of lithium genetics - which lithium is one of the most popular mood stabilizers for bipolar disorder - this study had over 2500 patients treated with lithium but only found one genome wide significant loci associated with treatment response to lithium. So just like bipolar disorder, it seems like we’ll need [a larger] sample size in order to see what is predicting treatment response, at least to lithium.</p>
<p>Beyond the GWAS, polygenic risk scores (PRS) have also been used in in pharmacogenomics. It’s been shown that, for instance, schizophrenia risk, the genetic risk, and depression genetic risk can predict worse response to lithium. And something I have talked about now at WCPG, is that ADHD genetic risk also predicts worse response to lithium. There’s also been a study, a very small study, of other mood stabilizers in bipolar disorder. And this was an extremely small study with 199 [BD patient], but even with this small study we did find two genome-wide significant hits in genes that made sense - so that’s an interesting study to include here.</p>
<p>But as is the case with a lot of different pharmacogenomics, it’s been harder to figure out what predicts treatment response. But we’ve had a lot more success in figuring out what predicts serious adverse reactions, so with that we can genotype a patient and then tell them whether they shouldn’t take a medication, rather than what they should take. And in bipolar disorder, it turns out that there’s HLA haplotypes that predict serious adverse reactions related to carbamazepine, lamotrigine, and other [mood stabilizers].</p>
<p><strong>PRS Prediction</strong></p>
<p>So PRS prediction is the last thing I’ll talk about here - and that’s one way that you can have clinical utility, or clinical implications, for the genetic findings. But unfortunately, in the latest paper we show that the best PRS derived from bipolar disorder GWAS explains less than five percent of the variation in case control phenotypes. So right now, as is the case with any other psychiatric disorder, PRSs are really not clinically useful at this point, at least [not] for predicting case control status.</p>
<p>But they [PRSs] can still be useful in understanding the heterogeneity within bipolar disorder. The heterogeneity is bipolar disorder getting dissected through polygenic risk scores, we’ve seen that higher schizophrenia risk is associated with type 1 bipolar disorder, which aligns well with clinical observations and schizophrenia risk is associated with psychosis in bipolar disorder, typically during mania. We also see that depression genetic risk is associated with suicide and ADHD genetic risk is associated with rapid cycling. So all of these things are sort of lining up well with the clinical outcomes and these sort of findings can lead us to have better diagnostics and potentially better therapeutics.</p>
<p><strong>Future Directions</strong></p>
<p>So, with that, future directions for bipolar disorder and the genetics of bipolar disorder: One of the big pushes from the latest PGC4 is that we’ll have increased ancestral diversity. With that if you look at the latest GWAS from the PGC-BD group, it only contains European ancestries in that GWAS, and so we’ve actually committed to expanding this beyond that to other ancestries in the next wave. And Kevin O’Connell will actually talk about how we included 23andme, which has a large diverse sample, into the current GWAS that is in the MedRxiv paper.</p>
<p>As I mentioned before, larger sequencing efforts will be needed and something that has already been ongoing is the bipolar sequencing consortium. This now includes 4,500 cases and 9,000 controls, and so they have to publish their results, but it’s also noted that we’ll likely need over 25,000 cases before we get really get a good sense of what rare variant contributions are being made to bipolar disorder.</p>
<p>On top of that we’ll need increased deep phenotyping. So, as I mentioned, there’s high clinical heterogeneity and a lot of this heterogeneity in the disorder can reduce the variance explained. Because bipolar disorder might have different [manifestations] and so understanding this heterogeneity would really inform the nosology and drug development for bipolar disorder. So it’s going to be a next effort, and its ongoing efforts, with the PGC-BD group to collect more samples with all this information about rapid cycling, psychosis, suicidality, ADHD - all these different subtypes.</p>
<p>And on top of wanting to get deeper phenotyping, as I mentioned, we are now at the inflection point for our GWAS. So, in order to really get more gains in the variance explained in the trait, we will need larger GWAS samples. And to do that we will need to look at different ways to ascertain [individuals with] bipolar disorder, in addition to including 23andme, we will be reaching out to other diverse populations, and we’ll have to continue recruiting across the world as we move along to try to get a larger GWAS sample. So, with that, I’ll just thank you for your attention and I hope that you enjoyed the genetics of bipolar disorder. Thanks.</p>
<hr>
</section>
<section id="sec-dementia" class="level1">
<h1>Dementia (Alzheimer’s, Parkinson’s)</h1>
<p><strong>Title</strong>: Genetic Risk for Dementia</p>
<p><strong>Presenter(s)</strong>: Malia Rumbaugh</p>
<p>Okay, okay, all right. So, I would like to introduce—I’m actually so happy to introduce—Malia Rumbaugh, who has expertise that we haven’t really had in our series before. So we’re very excited about that. She’s a genetic counselor with 20 years of research experience in the genetics of neurodegenerative disorders. She earned her MS in genetic counseling from the University of California, Irvine, and currently works at Indiana University and researches the genetics of Alzheimer’s and Parkinson’s disease. She provides genetic counseling to study participants, is the lead genetic counselor for two large multi-site studies of Alzheimer’s disease, and co-lead of the asymptomatic subcommittee of The Advisory Group on Risk, Evidence, Education for Dementia, which was convened by the NIA to focus on communication of dementia risk. She is also part of an NIH-convened work group on biomarker disclosure at Alzheimer’s disease research centers.</p>
<p>Her recent publications address interpretation of ApoE results in the primary care setting and the impact of new Alzheimer’s disease medications on biomarker disclosure. Malia is also a co-author of guidelines for genetic testing in Alzheimer’s disease, and we’re very fortunate to have her here today to talk to us about genetic risk for dementia. Welcome, Malia.</p>
<p><strong>Malia Rumbaugh</strong>:</p>
<p>Thank you so much, Barb. I am thrilled to be here, um, and I’d like to start by thanking the organizers of this wonderful series for an invitation to talk about something that is near and dear to my heart, which is working toward a better understanding of how our genes affect our risk of dementia and how to communicate that effectively to our patients</p>
<p>By way of disclosures - Oh my goodness, let’s get this out of the way first, right? Um, by way of disclosures, my work is generously supported by grants from the Alzheimer’s Association, the Michael J. Fox Foundation, the Parkinson’s Foundation, and the National Institutes on Aging.</p>
<p>Today we’re going to start with a short review of basic genetic concepts, just to make sure we’re all on the same page, we will then move on to the genetics of Alzheimer’s disease, frontotemporal dementia, and Lewy Body dementia. And then we’ll talk about some common questions that I get, that probably you get, on the subject of genetics of dementia. Things like, is Alzheimer’s disease genetic? My parent had Alzheimer’s disease, so does that mean that I will get it myself? Should I get tested for APOE? And one of my favorites, “Here’s my APOE results, can you tell me what this means?”</p>
<p>I will stop at the end of each of these sections to see if there are any questions coming in on the chat because I really hope that you’ll have questions for me as we go through this. Okay, so let’s start with a short review of basic genetics.</p>
<p><em>Concepts</em></p>
<p>What is a gene? A gene is a segment of DNA. It contains the information we need to construct our bodies and carry out daily functions. We have two copies of almost all of our genes, one inherited from our mother and one inherited from our father. So, our DNA is packaged into chromosomes, and each of our cells contains a complete set of our genetics. That’s why we can take a blood sample or a cheek swab and test genes that might affect how our nerves work, how our brain functions, and many other aspects of our health.</p>
<p>A gene codes for protein. In this little cartoon, you’ll see a segment here of maybe three nucleotides. Genes are actually hundreds or thousands of nucleotides long, and our bodies take that information and turn it into protein. Protein is what carries out most of the function in our bodies, whether that be contracting our muscles, our neurons firing in our brain, or how we build our bones. If there’s a change in this DNA, also called a variant, that can result in a changed protein.</p>
<p>Now, there are different types of variants, different types of changes in our DNA. Some of these are benign; they’re a natural part of human variation, and they have no known impact on human disease. Some of these are pathogenic, causing disease. These can be fully penetrant or causative; they always cause disease. Or, they can be a risk variant, meaning they increase the chance that a person would have the disease, but it’s not a hundred percent thing, um, and there are some people who carry these risk variants who will not develop the disease. This distinction between causative genes and risk factor genes is going to be very important for the genetics of dementia because we see both types, um, and sometimes we don’t have enough data to tell whether a variant is benign or pathogenic, and those get classified as a VUS or a variant of unknown significance.</p>
<p>Now, what happens if we find a new variant, and we need to decide if it’s gene-causing or not? This is a complex process, and one of the steps in that process is to check a database like gnomAD. This is a collection of sequencing data from hundreds or thousands of individuals, and we can search this database to see if the variant is common and we see it in many people without disease, or maybe it’s never been seen before. The organizers of gnomAD (and I have to say this is a fantastic resource) try to make the database as diverse as possible, but they do suffer from a common problem in genetics, which is that our data doesn’t necessarily represent our diversity. We know that some genetic variants can be risk factors in one population but benign in another. So, if we look at a description of one of the resources available on gnomAD (this is a database of over a hundred and twenty thousand individuals, which is fantastic), but let’s say our patient is of Middle Eastern descent or Native American descent, and as you can see, those populations are not well represented in this database. Therefore, we need to be careful about the conclusions we reach when analyzing their data. This is an acknowledged weakness in medical genetics and one that we’re working hard to fix, but we’re not there yet.</p>
<p>Um, before we move on to the next section, any questions so far?</p>
<p><strong>Barbara</strong>: I don’t see any in the chat.</p>
<p><strong>Malia</strong>: We will keep moving then. Um, and if you out there are ruminating or wondering,</p>
<p><strong>Barbara</strong>: oh, here I got one, okay. Um, how does gnomAD get their genetic information? Do companies like 23andMe contribute automatically?</p>
<p><strong>Malia</strong>: They don’t contribute automatically. So 23andMe, so I don’t work for 23andMe, but my understanding of the process is that you can decide whether or not you want your genetic data in that database, and if so, it may very well be added into gnomAD. There are also other studies that, things like the All of Us study, which is a federally funded study, that add their data into gnomAD. So it comes from a variety of sources. But just like with any genetic research, we do ask people to make sure it’s okay.</p>
<p><strong>Barbara</strong>: I’m sorry I mispronounced that. Um, Lorraine said, “No cultural diversity, amazing.” Or she said, “I should say racial diversity, it was amazing.”</p>
<p><strong>Malia</strong>: Yeah, trying. I mean, it’s better than it used to be, but it’s, it is tough. It is honestly tough. So yeah, working on it. Anything else? Don’t see anything else at this point.</p>
<section id="alzheimers-disease" class="level2">
<h2 class="anchored" data-anchor-id="alzheimers-disease">Alzheimer’s Disease</h2>
<p>Let’s move on. Let’s talk about the genetics of Alzheimer’s disease. Alzheimer’s disease, when you look at it as a whole, um, about 75 percent is going to be late-onset, non-familial. So symptoms begin after the age of 60-65, depending on your definition, and these individuals do not have a strong family history of the disease. About 20 percent will be late-onset familial, onset after 60-65, and there is a strong history of the disease in the family. Depending on what study you look at or your definitions, about five to ten percent of Alzheimer’s diseases are early-onset, and about one percent is genetic onset. And by genetic, I mean full one of those causative genes for Alzheimer’s disease.</p>
<p>Those genes are <em>PSEN1</em>, <em>PSEN2</em>, and <em>APP</em>. They are all involved in the production of amyloid, and amyloid is one of the hallmarks of Alzheimer’s disease. In this cartoon here, the vertical structure you can see here is the cell membrane. Cytosol is the inside of the cell, and here’s the outside of the cell. The amyloid precursor protein, which is the protein coded for by <em>APP</em>, falls across the cell membrane, and it is cleaved or cut by two different enzymes: beta-secretase and gamma-secretase. PSEN1 and PSEN2 interact with gamma-secretase, and if we have a mutation in one of these genes, that can affect where gamma-secretase cleaves this amyloid precursor protein. And that results in different forms of amyloid, and some forms of amyloid are more likely to go and form these amyloid plaques, which are the hallmark of Alzheimer’s disease.</p>
<p>Um, they tend to have different ages of onset, uh, <em>PSEN1</em> being the earliest. Most sources will say 30s at the earliest. I have to say I knew somebody who had a <em>PSEN1</em> pathogenic variant and had onset in his 20s, so we can see that, thankfully, very rare. And you can see onset into the 70s with <em>PSEN2</em> or sometimes, um, I know one case of somebody who had a pathogenic variant in <em>PSEN2</em> and did not develop the clinical signs of Alzheimer’s disease. That is very rare, unfortunately.</p>
<p>This is a typical family tree that we would see in one of these families. Let’s say this is our patient, a man 50 years old, onset of cognitive decline at 48. His father had onset at 42, um, his paternal aunt had onset at 51, and his grandmother on his father’s side had onset at 45. So, when we talk about these purely genetic early-onset Alzheimer’s families, this is the sort of thing that we see.</p>
<p>Now, what about the other 99% of Alzheimer’s disease that is not purely genetic? Here we enter the realm of risk factor genes, and the best studied and most common of these is <em>APOE</em>. It’s the major lipid and cholesterol carrier in the central nervous system. There are three polymorphisms, or forms, flavors if you will, of <em>APOE</em>. E2 is associated with a lower risk of Alzheimer’s disease, E3 we consider average risk of Alzheimer’s disease, and that’s the most common form, and E4 is associated with a higher risk of Alzheimer’s disease. E4 has been linked to all different aspects of Alzheimer’s pathology, so it has been linked to neuroinflammation, synaptic dysfunction, mitochondrial or metabolism dysfunction, Tau aggregation (which forms those tangles, also a hallmark of Alzheimer’s), and amyloid aggregation and reduced clearance. So, this is a fairly well-studied gene and has been linked to a number of different things.</p>
<p>How does <em>APOE</em> affect our risk of dementia? Well, the specific risk for Alzheimer’s related to <em>APOE</em> varies between studies, and you can see different numbers quoted in different sources, but the numbers I’m showing here are compiled by one of the most careful epidemiologists I know, uh, Deborah Blacker at Harvard. And it shows the lifetime risk, or age through age 85, for people who might carry two copies of the E4 <em>APOE</em> genotype have a lifetime risk of between 30 and 55 percent. So it’s definitely higher than the general population, but not everybody who carries any four-four will develop Alzheimer’s disease. If you have just one copy of E4, you also have an increased risk, about 20 to 25 percent. And three-three, which we considered normal, is about 10 to 15 percent.</p>
<p>Now, there are other genes related to Alzheimer’s disease, and I apologize, this is a very busy and complex graph, but I think it captures the complexity of what we know so far about the genetics of Alzheimer’s really well. So here, on the vertical y-axis, we start at the bottom with genes that are associated with a relatively low risk of Alzheimer’s disease and moving up to a near-certain chance of developing Alzheimer’s. And then here, population frequency, we go from very, very rare at zero percent or close to zero percent to relatively more common. And up here, we can see our old friends <em>APP</em>, <em>PSEN1</em>, <em>PSEN2</em>, being very rare in the population but having a near-certain chance of developing Alzheimer’s disease. <em>APOE4</em> is relatively rare and has a higher chance of getting Alzheimer’s. And then we move into some of the more common genes that have a smaller effect on our chance of getting Alzheimer’s disease. There are many, many more genes than this, actually, and here the colors relate to some of the cell functions that are related to these genes, whether it be amyloid metabolism, APP metabolism, cholesterol, immune response, endocytosis, cytoskeleton. There are researchers working to develop what they call polygenic risk scores, where we can combine all the data from testing these multiple and interacting genes, and give somebody more of a accurate, hopefully closer to accurate, assessments of their chance of getting Alzheimer’s. So not there yet, but we may get there. Um, and before I move on to the genetics of frontotemporal dementia, Barb, any questions at this point?</p>
<p><strong>Barbara</strong>: No questions have come across. Do any of you have questions at this point?</p>
<p><strong>Malia</strong>: And it’s fine if you don’t. We can go ahead.</p>
<p><strong>Barbara</strong>: Everyone, just put your questions as they occur to you for this lecture in the chat box, and I’ll track them to the next time um we pause for questions.</p>
<p><strong>Malia</strong>: And I really don’t mean to put you on the spot.</p>
</section>
<section id="genetics-of-frontal-temporal-dementia" class="level2">
<h2 class="anchored" data-anchor-id="genetics-of-frontal-temporal-dementia">Genetics of Frontal Temporal Dementia</h2>
<p>Now, this disorder is less likely to be sporadic than Alzheimer’s disease. We saw about 75 percent of people with Alzheimer’s didn’t have a strong family history of the disease. Frontotemporal dementia, we’re looking at about 40 percent. Also, about 40 percent have a positive family history of dementia, so that includes also psychiatric disease and motor symptoms. And it’s thought, perhaps in this 40 percent, we’re looking at a similar combination of risk factor genes as we saw in Alzheimer’s disease. These risk factors, risk factor genes, for frontal temporal dementia are not as well understood as Alzheimer’s disease, and then 20 percent of people who have frontal temporal dementia have a pathogenic variant in a single gene.</p>
<p>And those genes are <em>C9orf72</em>, granulin or <em>GRN</em>, and <em>MAPT</em>. Now, each of these are more common in the behavioral variant of FTD versus the semantic variant of FTD. They each have their own flavor. <em>C9orf72</em> is the most common cause of genetic frontotemporal dementia. It’s also the most common cause of ALS or Lou Gehrig’s Disease, and we do see sometimes in families that have a pathogenic mutation or variance in <em>C9orf72</em>. You can see people who have both frontotemporal dementia and ALS. You can see people in the same family who have one or the other. Um, <em>C9orf72</em> can also look like Alzheimer’s disease, and so we do see that as well, where someone will have a family history of Alzheimer’s disease, we do the testing and find out it was likely also <em>C9orf72</em>. Granulin and <em>MAPT</em> can cause parkinsonian features, um, also Progressive Supranuclear Palsy or corticobasal degeneration, so we also see that in these families. There are also rare genes for frontotemporal dementia listed here. I will say a lot of these are extremely rare, but they are also fully penetrant for frontal temporal dementia.</p>
<p>And you might be wondering, um, with all of these genes, how do you pick? How do you know which gene to test? And the, uh, quick answer is we don’t. Um, with current technology, it’s often the same price to test multiple genes. And so if you know it’s frontotemporal dementia versus Alzheimer’s disease, maybe you will just test for the frontotemporal genes. But, um, often if a patient hasn’t had an amyloid scan or any sort of positive verification that it is one disorder versus another, the clinician might just order a dementia panel and include all of the causative genes for dementia.</p>
</section>
<section id="lewy-body-dementiaparkinsons-disease" class="level2">
<h2 class="anchored" data-anchor-id="lewy-body-dementiaparkinsons-disease">Lewy Body Dementia/Parkinson’s Disease</h2>
<p>So, I will move on to the genetics of Lewy Body dementia. Say, this one, under this one, I’m including both Parkinson’s disease dementia, which, as you might know, is the onset of dementia more than one year after the start of motor symptoms or parkinsonian symptoms, and also dementia with Lewy bodies, which is the onset of dementia before or within one year of motor symptoms.</p>
<p>The genetics of Lewy Body dementia are not well understood, I have to say, but it seems to be midway between Alzheimer’s and Parkinson’s disease. So, genes recently identified for Lewy Body dementia include, here, um, some of the, uh, Parkinson’s um genes that you see in black, especially <em>GBA</em>, which is one of our more common risk factors for Parkinson’s Disease, has also a higher chance of dementia associated with it. And so sometimes you can see somebody with Lewy Body dementia who has a pathogenic variant in <em>GBA</em>.</p>
<p><em>SNCA</em>, which codes for synuclein, which is the protein that is characteristic for Parkinson’s Disease, that’s the protein that builds up in Lewy bodies. Pathogenic mutations in <em>SNCA</em> are more rare, but they are more likely to also involve cognitive issues and dementia. We also see some of the dementia genes in Lewy Body dementia, <em>APOE</em> we’ve already talked about, granulin and <em>MAPT</em> are actually both tied to frontal temporal dementia, but they also come up in Lewy Body dementia.</p>
<p>It’s a complex system, and I think the best quote I saw in the papers that I’ve been reading about Lewy Body dementia is that the genetic basis of it is not well understood. So I’m not able to give you a nice schematic on how many people have familial versus non-familial and how likely it is to be inherited in the family. I don’t think we’re quite there yet with Lewy Body dementia.</p>
<p>Um, I will say that if you know a patient who has Parkinson’s Dementia or a family history or has Parkinson’s disease, I do happen to know a study that’s doing um genetic testing for Parkinson’s Disease. So that’s PD Gene from the Parkinson’s Foundation. You can just Google PD Gene Parkinson’s Foundation. It’s no-cost genetic testing for Parkinson’s, which includes Parkinson’s dementia, in case that’s helpful.</p>
<p>Um, and Barbara, any other questions have come in before we move on?</p>
<p><strong>Barbara</strong>: Yeah, there was a question about whether or not insurance pays for any of these tests you just mentioned. The study that covers the cost, what about insurance covering the cost of any of these?</p>
<p><strong>Malia</strong>: It varies. Um, for early-onset dementia, where doing genetic testing can be diagnostic for a person’s symptoms, I’ve seen insurance cover the testing. For <em>APOE</em>, because we have these new anti-amyloid therapies, and in a person’s (and here I’m talking about people who have been diagnosed with dementia and are considering tests or treatment for one of the anti-amyloid therapies), having an <em>APOE4</em> allele or variant increases the chance of side effects from those medications. So, I would expect insurance to cover those costs. In other cases, they don’t, and it is highly variable, so, yeah, I’ve seen it go both ways. Any other questions, Barbara? There was a question about typing out.</p>
<p><strong>Barbara</strong>: There was a question about typing out the PDG contact info, and I’m trying to find it right now. Um, I’m seeing it under Michael J. Fox organization, would that be right? Oh, somebody else got it before I did, never mind.</p>
<p><strong>Malia</strong>: Yep, sorry to put you on the spot with that. Um, it’s a full disclosure, I work on that study, and we have had a fantastic response to it, which means there might be some waitlists now. Um, but it is, you know, no-cost genetic testing for the seven most common genes for Parkinson’s, so it is a great service that the Parkinson’s Foundation is doing. And people can do it remotely; they can sign up through the website and be sent a kit, or they can sign up through a site at a local University Medical Center.</p>
<p><strong>Barbara</strong>: Yeah, the link, that’s great, thank you, Lisa.</p>
<p><strong>Malia</strong>: Yeah, it’s a great program, thank you. Anything else, Barb? We’ll say that’s a no,</p>
<p><strong>Barbara</strong>: not yet, sorry, no, not at all.</p>
<p><strong>Malia</strong>: Let’s move on to common questions. Um, so one of the most common questions that I get, maybe you get, is “Alzheimer’s genetic?” and this is what I say: there are rare cases of Alzheimer’s disease that are purely genetic. People in those families get Alzheimer’s early, like in their 30s or 50s, and there are several people with early-onset Alzheimer’s disease in the same family. So yes, it does happen, it is rare. Most cases of Alzheimer’s disease are caused by a combination of genetic and environmental factors, and I should say, speaking as someone who works in genetics, I consider everything that’s not genetic, environmental. So here we’re talking about, right, hypertension, hyperlipidemia, diabetes, sedentary lifestyle, what have you.</p>
<p>One way to think about this is that everyone has an Alzheimer’s jar, and this same analogy could be used for just about any common disease, cardiovascular disease, what have you. Everyone has an Alzheimer’s jar, and this jar can be filled with environmental risk factors that I just described; it can also be filled with genetic risk factors, and some of these, you know, might be <em>APOE</em>, might be <em>TREM2</em>, might be <em>BIN1</em>, any of those genes that we already looked at.</p>
<p>We start with a certain number of genetic risk factors, and over time we accumulate environmental risk factors, and if our jar fills to the top, we develop Alzheimer’s disease. Different people have different combinations of genetic and environmental risk factors, so some people are going to start with a relatively large number of genetic risk factors, and they will accumulate environmental risk factors just like the rest of us, but they don’t need to accumulate quite so many before they develop Alzheimer’s disease. Other people start with a relatively low number of genetic risk factors for Alzheimer’s, but if they accumulate enough environmental risk factors, they will still develop Alzheimer’s disease.</p>
<p>There are protective factors that can make the jar taller, and those can be social activities, mental activities, physical activity, healthy diets, good sleep habits, all of the things we know that decrease our chance of getting Alzheimer’s Disease. By doing these, we can sort of add rings to the top of the jar, make the jar taller, and give us more time before we develop Alzheimer’s disease. And I find, when talking to people, it’s helpful to steer the conversation and end on things that they can do, steps that they can take to lower their chance of getting Alzheimer’s disease.</p>
<p>Here’s another question that I get: “My parent had Alzheimer’s, does that mean I’ll get it too?” And I think it’s helpful to put that in context. We all have a chance of developing Alzheimer’s; it’s unfortunately not a rare disease. But it depends on what study you look at. People who have a parent with Alzheimer’s have about a 20% chance of getting it themselves. That’s compared to about 10% in the general population. Another way to look at it: 20% chance of getting Alzheimer’s is an 80% chance of not getting Alzheimer’s. There’s not a surefire way to prevent Alzheimer’s disease, but you can improve these odds through lifestyle changes.</p>
<p>Maybe I will pause there. Thank you, Barb, for monitoring the chat because I am awful at reading and talking at the same time. But it looks like maybe a couple of questions have come in.</p>
<p><strong>Barbara</strong>: Yeah, well, Joanne just mentioned that these visuals are really great for helping to translate some of this information to people. Lorraine is wondering if by environmental factors, are you referring to lifestyles or perhaps pollution and work-related exposures, or maybe all of the above?</p>
<p><strong>Malia</strong>: All of the above. I have a complete bias in this area, working in genetics. In my world view, there’s genetics and there’s everything else. So we should probably find another name for it, but yes, it could be medical conditions like diabetes, cardiovascular disease, could be air pollution, which has been linked to a slightly higher chance of getting Alzheimer’s. What have you - Anything that’s not genetic, I would call environmental. Anything else?</p>
<p><strong>Barbara</strong>: That’s it for now.</p>
<p><strong>Malia</strong>: Cool. Here’s a great one. Um, should I get tested for <em>APOE</em>? So there are clinical guidelines recommending against <em>APOE</em> testing for people without memory complaints due to its low predictive value. Many people who carry the bad copy of <em>APOE</em>, which is E4, do not develop Alzheimer’s disease, and about half of those with Alzheimer’s disease don’t have an E4 allele. So the catchphrase is, “<em>APOE</em> is neither necessary nor sufficient to develop Alzheimer’s disease.” You can get Alzheimer’s without having an <em>APOE4</em>, the bad copy, and if you have an A4, that doesn’t mean you’re going to get Alzheimer’s disease. A bunch of us got together and recommended against testing for <em>APOE</em> in people who do not have memory complaints.</p>
<p>That said, in the real world, things happen differently, and you might have seen this in the news last November. The actor Chris Hemsworth learned he carries two copies of <em>APOE</em>4 as a part of a National Geographic docu-series, and it’s not clear that he was fully informed before he was tested or that the results were adequately explained. Probably goes without saying, but don’t get tested as part of a television show or without carefully considering what the tests can and cannot tell you, and whether you really want that information.</p>
<p>For people who are thinking about getting tested and do not have memory concerns, there is a really helpful website, GeneTestOrNot.org, and I’ll leave this up for a few minutes so you can write it down if you want. It’s an online decision tool to help people decide whether or not they want to get tested, and it asks people to consider: Do you have a family history of Alzheimer’s disease? Will this genetic testing give you useful information? Is this the right time to get tested, or maybe you should wait? Whether the advantages outweigh the disadvantages. Among the important issues discussed on this website is the Genetic Information Nondiscrimination Act (GINA), which protects against discrimination based on genetic information for health insurance or employment. Important to note that this protection does not cover long-term care insurance or life insurance. And so, once you know something, you can’t unknow it, and it’s better to think things through before getting tested.</p>
<p>Another good question: somebody walks into your clinic and hands you their 23andMe <em>APOE</em> test results, or maybe they got tested through Cardiology or Nephrology, which also do <em>APOE</em> testing, and now they want to know, “Okay, what tell me what does this mean?” It’s a really complex question. It depends on a person’s family history, it depends on their age, depends on whether they’re currently having memory concerns. In order to address or begin to address all of these questions, we wrote a paper, and it’s a series of vignettes of different situations of when patients come in asking about their <em>APOE</em> test results. And here are some ideas and some guidance on what to tell them. I should also say that the table that I included earlier that had the various <em>APOE</em> genotypes (three three, four four, what have you) and the risk of Alzheimer’s disease is included in this paper. So this is a publicly available article through the general family practice. If you have any problems finding it, you’re welcome to contact me, and I can just shoot you a copy of the paper.</p>
<p>And that is actually all I have unless there are other questions or comments.</p>
<p><strong>Barbara</strong>: What um questions might you have or um issues that have come up for you, if you, as you’ve connected with um your patient populations or people in the community?</p>
<p><strong>Malia</strong>: One of the main questions or sort of concerns I get is when we hear the word “genetic,” we think predestination, right? We think absolutely causative. If it’s in your genes, it’s in your genes, and you are going to get whatever it is. And the idea of this complex interplay between our genes and our environments and our lifestyle is a little harder for people to get their heads around. Um, that’s one of the big issues. Um, among the people I see for genetic testing, one of the big concerns is, especially if they have been diagnosed with Alzheimer’s disease, frontotemporal dementia, or Parkinson’s, what are the chances that their kids are going to get it? And almost all of these genes are autosomal dominant, which means that if you carry the gene, the pathogenic variant in this gene, there’s a 50-50 chance that each of your children will also carry this same pathogenic variant. Now, that doesn’t necessarily mean they’re going to get Alzheimer’s or Parkinson’s or frontotemporal dementia, depending on whether this is a causative gene. If it is a causative gene, then that means if that individual lives long enough, they will have a very high likelihood of getting the disease. Risk variants, it just means if they have inherited pathogenic variants in that gene, they have a higher chance of also developing the disease. Um, so those are the most common issues that I hear about from patients and research participants. Um, what does it mean if I carry a mutation, pathogenic variant, in one of these genes and what does it mean for my kids?</p>
<p><strong>Barbara</strong>: So, a question: What lifestyle changes or environmental changes do you see that are most effective at prolonging onset of symptoms of Alzheimer’s or dementia?</p>
<p><strong>Malia</strong>: The Alzheimer’s Association is a fantastic source for that, and I would definitely go to that website. Um, the main thing I talk to people about is the link between heart health and head health, or heart health and brain health. So, all of the things we know we’re supposed to be doing to protect our heart, we should also have implications for our brain health. Managing any high blood pressure, managing your blood cholesterol levels, managing your blood glucose levels, um, exercise is very important, eating a balanced diet—you know, the standard Mediterranean diet, is a very good idea. In addition to all of those things, um, staying socially active and staying mentally active are great things to do for your brain health. So, yeah, what I tell people is it’s all the boring stuff that we know we’re supposed to be doing anyway, and it’s hard to do, but it really does have an impact on our health.</p>
<p><strong>Barbara</strong>: Malia, you have an earlier slide, something about mild cognitive impairment. I’m wondering if you can talk a little bit about the relationship between mild cognitive impairment and genetics and the development of Alzheimer’s. Do you deal with that at all where you’re seeing people come with mild cognitive impairment and then wanting genetic testing done, for example?</p>
<p><strong>Malia</strong>: Yeah, I don’t often see people with mild cognitive impairment because they are not eligible for our Alzheimer’s studies. I do sometimes see people with Parkinson’s disease who are beginning to have cognitive issues. In terms of genetic testing for, let’s say, somebody is concerned about their memory and they want to get <em>APOE</em> testing to see if it’s Alzheimer’s or not. That is an option, but it gives you limited information since, again, <em>APOE</em> testing is not absolutely predictive, and half of people who develop Alzheimer’s disease don’t have an <em>APOE4</em> allele. What I’ll sometimes talk to them about is what might be just as helpful or more helpful is doing baseline cognitive testing and then repeating that a year later to see if there has been decline, and also doing some of those lifestyle interventions in the meantime.</p>
<p>So, it’s a different story, I’ll have to say, if it’s a family with early-onset Alzheimer’s disease or early-onset dementia. In that case, you may want to think about doing testing for those genes, and in one of those families, it’s most helpful to do genetic testing on an individual who has been diagnosed with early-onset dementia, to know that you’re testing the right person in the family. Because if you say have, you know, somebody whose parents and uncle and aunts and grandparents died with early-onset dementia, and you do genetic testing on that individual in the next generation who does not currently have cognitive complaints. Let’s say that testing comes back negative, which is great. But you don’t know if you’ve tested for the right thing. I mean, maybe it’s come back negative because there’s a gene we haven’t found yet in the family, and so we are in our testing for the currently known genes, we’re not going to pick that up, and we may be giving false reassurance to that individual. So general concept in genetics, start with somebody in the family who has the disorder, make sure it’s a gene that you can identify in the family, and then you can offer predictive testing to other individuals as appropriate if they want the information.</p>
<p><strong>Barbara</strong>: I also hope you all don’t, it seems like a lot of the genes are focusing on the amyloid hypothesis, and yet there’s still some controversy about amyloid and Alzheimer’s, for example, and I’m wondering if you have some information about how that’s playing into the research that’s going on right now and what we might see in the future.</p>
<p><strong>Malia</strong>: Yeah, let’s, uh, flip back. Sorry for all the changes here. Let’s flip back to that table, um, that graph of all of these. Um, so there was, I mean, there’s been a huge debate for years, right? Um, is it Tau or is it amyloid? It used to be called the Baptist versus the taoists in Alzheimer’s research, and a lot of the genetics is pointing toward amyloid. Um, but there are genes like we can see here, <em>BIN1</em>, which has the red circle around it, is linked with Tau, and it’s entirely possible there are other genes linked with Tau as well. That, um, in just concentrating on amyloid and not Tau at all, we are missing part of the picture. We don’t know yet, but I think as we have seen many of these anti-amyloid therapies in the early stages fail and fail and fail, and now we have some anti-amyloid therapies which are somewhat effective, um, they are not a cure, they hopefully slow down the disease. Yeah, you do wonder if it’s an oversimplification to only go after amyloid. Um, so say we don’t know at this point, um, but it’s certainly seems that amyloid is very important in Alzheimer’s disease. Is it the only pathogenic mechanism? No, and is it the only thing we should be targeting in our therapies? Potentially not, but we will see.</p>
<p><strong>Barbara</strong>: So, Lorraine mentioned that, um, she thinks that since we don’t have treatment or specific improvement, it would be a big conundrum to decide to test or not. She said, “I can see this if you’re interested in research and helping for the future, you might test.” So that makes me think about other studies I’ve been involved with where you really puzzle about what kind of recruitment message you want to convey. And I’m wondering how you do that for the studies that you’re involved with.</p>
<p><strong>Malia</strong>: Yeah, it is really difficult. Um, I would say in our early onset families, our early onset genetic families where our testing is predictive, more than half of people decide they don’t want to be tested because it’s not information they can act on, um, and it’s not something they think will make their life better. And one of the exercises we go through in genetic counseling often is having somebody imagine getting their test results and finding out they do have the family variants, that they do have a strong genetic predisposition to Alzheimer’s or frontotemporal dementia. And, um, how that sits with them and whether this is something that they are going to regret learning or not. Some people, for some people, the uncertainty is worse than knowing, and they just want to know that if they forget their car keys, if they lose their car keys, you know, does that mean it’s the start of something, um, or not? And, of course, individuals in these families have, even if they don’t carry pathogenic mutation in the gene, they have the same risk for late-onset Alzheimer’s disease as the rest of us. Many use the information. Those who decide to get tested, many use the information to join studies, and there’s a study called DIAN (Dominantly Inherited Alzheimer’s Network), um, which has done great work in understanding the pathology of Alzheimer’s disease, understanding the very early signs of Alzheimer’s disease because in these individuals who carry one of these variants, um, you can look at the family and make a fairly good prediction of the age at which they will develop symptoms. And so those individuals are carefully followed in the five to ten years before average symptom onset to see what are the changes in, you know, on brain scans, on lumbar puncture, on blood tests, what are the very early signs we can pick up, um, as Alzheimer’s starts to develop in the brain but the person isn’t showing symptoms yet? And once we do have very effective therapies, it’s going to be really helpful, I think, to be able to identify people in those very early pre-clinical stages and start the therapies at that point.</p>
<p>So that’s all the causative genes, <em>APOE</em>. Um, some people, again, they just want to know whether or not they have it. Um, frankly, a lot of people I think get, you know, 23andMe testing. People I’ve talked to may not think through it that thoroughly, and then they find out that they do have an <em>APOE4</em>, and then they’re really wondering what it means.</p>
<p><strong>Barbara</strong>: So this is my stupidity Malia, I’ve never done 23andMe. What preparation is involved in that?</p>
<p><strong>Malia</strong>: So 23andMe tests for Parkinson’s, and that’s part of their standard um, panel for your health, uh, health screening. For <em>APOE</em>, they do ask you to opt in, they do ask you to check a box and say, “Yes, I want this information,” and they tell you a bit about what the information involves. So they do ask people to read through a description about what the test can tell you and what it can’t.</p>
<p>Um, and I don’t think I’ve seen any numbers from 23andMe on how many they’ve tested, but it’s at least in the thousands. So that is a common way that people find out their <em>APOE</em> results.</p>
<p><strong>Barbara</strong>: Can you please repeat what DIAN stands for?</p>
<p><strong>Malia</strong>: Dominantly Inherited Alzheimer’s Network.</p>
<p><strong>Barbara</strong>: Thank you. And if a patient joins a study, in case, say, they want life insurance or long-term care insurance, so if they join a study and there’s testing, is that protected information?</p>
<p><strong>Malia</strong>: It is, so, well, it depends on the site, depends on the hospital. Um, the Mayo Clinic requires that any test done in a research study be added to the clinical record, and so that is something that, for our sites at Mayo, both, you know, Rochester and Jacksonville and they have many sites, um, that those individuals need to understand because their research results are not protected from their clinical record. That’s the only site I know of that does that. For everybody else, it is not included in their clinical record.</p>
<p>Often, clinicians really want it in the clinical record because that’s useful information for a person’s medical care, and so participants are given the choice whether or not they want it in there. Um, now, if somebody, you know, decides that they don’t want it in their clinical record and they are asked by long-term care insurance, you know, “Have you ever gotten genetic testing for this or that for Alzheimer’s disease?” Um, they could say no. I don’t think there would be a way for that insurance company to find out, but it is insurance fraud, so it is something not to be taken lightly.</p>
<p>Um, when we are especially talking about some of these more predictive testing, one causative genes, um, for especially Alzheimer’s disease, frontotemporal dementia, for someone who has not developed symptoms of that disorder, we say, you know, “Are you comfortable with your long-term care insurance? Um, are you comfortable with your life insurance?” That might be something you want to review before getting this testing because it is not protected by federal regulations.</p>
<p><strong>Barbara</strong>: And other than 23andMe, is it likely that anyone getting genetic testing would see a genetic counselor before the testing occurs?</p>
<p><strong>Malia</strong>: Um, it depends. So if you, uh, are seen in a genetics clinic, you’re certainly likely to see somebody in a genetic counselor for something like Huntington’s disease or some of the genetic forms of other neurological diseases, Charcot-Marie-Tooth or what have you. Um, especially for predictive testing, again, if someone doesn’t currently have symptoms, they’ll probably talk with a genetic counselor. For other, like, for our Parkinson’s studies, uh, people don’t meet with a genetic counselor prior to testing. We have a video that goes over the most important issues in genetic testing. It’s an issue in healthcare because, frankly, there aren’t that many genetic counselors and, um, genetic testing is getting more and more common. And so we’re trying to find ways to make sure that people are fully informed about the ramifications of this testing, um, while taking into consideration the availability of people to really sit down one-on-one and go over those issues. So, it depends, sometimes yes, sometimes no. It’s helpful.</p>
<p><strong>Barbara</strong>: Any other questions, folks? Well, I, for one, have learned an incredible amount from this. I really appreciate your expertise, Malia, and all the information that you’ve shared. Um, we’re getting comments about it being a wonderful presentation and just the information, the way you’ve presented it, makes it easier for us to convey to people out in the community. So, I appreciate that.</p>
<p><strong>Malia</strong>: Absolutely. And again, if you have any questions in the future, if you’d like a copy of that um, uh, paper, that’s my email address. Shoot me a note. And all right, oh goodness, all right.</p>
<p><strong>Barbara</strong>: thank you so much, and I everyone, again, this is the last day of the last session for this series. The next one will start up on March 28th. You should be getting an email with all of the information for registering. But what a great session to finish up with because, um, it’s there’s not a lot of people with your expertise, Malia, and so I really appreciate it.</p>
<p><strong>Malia</strong>: Yeah, absolutely.</p>
<p><strong>Barbara</strong>: See everyone in the spring, hopefully we’ll be done with the rain and the snow by then. Bye, everyone. Bye, Barb. Bye-bye.</p>
<hr>
</section>
</section>
<section id="sec-eatingdisorders" class="level1">
<h1>Eating Disorders</h1>
<p><strong>Title</strong>: Genetics of Eating Disorders</p>
<p><strong>Presenter(s)</strong>: Hunna Watson, Alish Palmos, Avina Hunjan, Jessica Baker, Zeynep Yilmaz, Helena Davies</p>
<p><strong>Helena Davies</strong>:</p>
<p>Hi everyone, thanks for coming to watch our talk. Uh, so just to introduce myself, I’m Helena Davies, a PhD student at King’s College in London. And today, I, along with Alish and Zeynep, will be presenting our new review on the genetics of eating disorders, and this is on behalf of all the authors that worked on it.</p>
<p>So, just to start, this is a timeline of advances in eating disorder genetics, and I apologize for the small font. I, uh, I’m not expecting you to be able to read everything that it says. Um, but it’s mainly there just to demonstrate how far we’ve come. So, from the introduction of epidemiological twin studies in 1991, all the way to finding 18 genome-wide significant hits for anorexia in 2019, and now efforts are continuing, and we’re moving even further towards unlocking the biological basis of eating disorders. And today, we’ll go into, uh, some of these findings in a little bit more detail. So this is what I’ll be going over before passing you on to Alish.</p>
<p>So, I’ll discuss twin studies, genome-wide association studies, SNP-based genetic correlations, and genetic risk scores. But first, just to briefly go over the main features of each eating disorder that we discuss in our review:</p>
<p>Anorexia is the least common but most deadly of all the eating disorders and is defined by significantly low body weight, restrictive eating behavior, and body image disturbance. In contrast, bulimia nervosa is characterized by binge eating and then engaging in compensatory behaviors. So, this includes things like self-induced vomiting and laxative use. And finally, those with binge eating disorder also binge eat, however, they don’t engage in compensatory behaviors. So, these are the eating disorders that we discuss in the review. However, other eating and feeding disorders do exist, but there’s a lack of genetic and epidemiological research into them.</p>
<p>So, just to start moving into twin studies, these were the first studies to suggest a genetic component of eating disorders, and the heritability estimates were pretty substantial. So, they range from 16 to 74 percent for anorexia, 28 to 82 percent for bulimia, and 39 to 45 percent for binge eating disorder. And as you can see, these vary quite a bit, and this depends on whether we use threshold or relaxed DSM criteria. So, for example, the heritability of anorexia increases when we expand the definition of anorexia to include subsyndromal cases.</p>
<p>Twin studies can also tell us about the genetic overlap between eating disorders. They’ve told us that around 60% of the genetic effects of anorexia and bulimia may be shared. And they also can tell us about the genetic overlap between eating disorders and other psychiatric disorders. There’s evidence to suggest that eating disorders share genetic effects with alcohol and substance use disorder, obsessive-compulsive disorder, and major depressive disorder.</p>
<p>However, while twin studies are great in showing us the relative contribution of genetics and environment on human traits, they don’t tell us anything about the biological and molecular mechanisms involved in risk. And this is where molecular genetic approaches come in.</p>
<p>So, after candidate gene studies and linkage studies had little success and they also failed to replicate, genome-wide association studies or GWAS became the dominant approach to exploring genetic variation across complex disorders. Genome-wide association studies are hypothesis-free, and what they do is they compare the genomes of those with a trait or illness to those without that trait or illness to discover differences in the genetics associated with the phenotype of interest.</p>
<p>So far, all GWAS for eating disorders have been conducted on anorexia. This is because there’s a lack of genetic data on other eating disorders, and the PGC (Psychiatric Genomics Consortium) has conducted the two most recent GWAS, with the latest one in 2019 finding 18 genome-wide significant hits and a SNP heritability estimate of 11 to 17 percent. But what’s really interesting is that if we take all these GWAS results altogether, it tells us that, firstly, it supports the polygenic nature of anorexia and also tells us that with increasing sample size, we’re more likely to be able to detect novel risk variants associated with risk.</p>
<p>This is the Manhattan plot of the latest PGC GWAS, and the clearest evidence was for the single gene loci that intersected with the four genes in red here. The authors concluded that these genes are likely to have a role in the etiology of anorexia, and this is really exciting. However, it’s only the beginning because we expect hundreds of genes to be associated with anorexia.</p>
<p>So now moving on to SNP-based genetic correlations, these provide insight into the overlapping genetics of traits and give further clues to their biological basis, as they indicate where some of the same genes are operative. And what we’ve got here is all the significant genetic correlations from the latest PGC anorexia GWAS. Just to break it down for you, in the red brackets are the positive genetic correlations with psychiatric disorders or traits, and this confirms previously reported patterns of comorbidity with OCD, depression, schizophrenia, and anxiety. Next, here are some positive genetic correlations with educational attainment, so this includes things like years of education and college completion. And here is a newly discovered genetic correlation with physical activity, and this is actually really interesting because compulsive exercise is a core feature of individuals with anorexia or can be a core feature of individuals with anorexia, and this has traditionally been explained away as a drive to lose weight. But what this genetic correlation shows is that this may also have a genetic origin. So some of the same genes that influence risk for anorexia are also associated with high physical activity, and this could help to shift our understanding of anorexia away from a purely drive for thinness explanation.</p>
<p>Next are some newly discovered genetic correlations with metabolic activity, and these are metabolic traits that are typically considered healthy. So these are things like a lower risk of type 2 diabetes and a positive genetic correlation with HDL cholesterol, which is a good type of cholesterol. However, these so-called healthy traits may actually have effects that are undesirable, such as causing individuals with anorexia to store less of their food intake as fat. This might explain why anorexia can be so challenging to treat. Finally, here are some negative genetic correlations with anthropometric traits, which are related to measurements and proportions of the human body. For example, there’s a negative genetic correlation with BMI and body fat percentage with anorexia. Again, this could help to explain why people with anorexia have such a hard time maintaining higher weight and may help us move away from explaining low BMI as solely a result of a drive for thinness or body dissatisfaction.</p>
<p>Interestingly, another study found a positive genetic correlation between anorexia with binge eating and cannabis initiation, whereas they found a negative genetic correlation between anorexia without binge eating and smoking phenotypes. This is evidence for potential differences in the genetic architecture of anorexia subtypes.</p>
<p>So now moving on to genetic risk scores, the GWAS results for anorexia hinted at the presence of thousands of unidentified genetic variants, each of which has a small effect on the phenotype. Genetic risk for anorexia can be represented with a genetic risk score, which is the sum of risk alleles weighted by the effect size from the GWAS. However, genetic risk scores for eating disorders are in their infancy because we need larger, more well-powered GWAS to be able to take them further.</p>
<p>What is interesting is that there’s an association between genetic risk scores for psychiatric traits (such as bipolar disorder and schizophrenia) and anthropometric traits with eating disorder diagnosis and symptom phenotypes. Excitingly, there are many future uses of genetic risk scores in eating disorder research, and there are just some listed here. For example, they can be used to evaluate whether GWAS findings generalize to multi-ethnic populations for which we might not have GWAS findings yet.</p>
<p><strong>Alish Palmos</strong>:</p>
<p>Hi, my name is Alish Palmos, and I’m going to talk to you about the molecular genetics of eating disorders, covering the following points:</p>
<ol type="1">
<li><p>Psychiatric disorders are highly co-morbid with one another, and at times, differentiation of a diagnosis based on symptoms may be complex, suggesting shared risk.</p></li>
<li><p>Cross-disorder GWAS (Genome-Wide Association Studies) have begun to investigate common genetic pathways in the etiology of psychiatric disorders.</p></li>
<li><p>One such effort, including anorexia and seven other psychiatric disorders listed on the slide, identified 109 independently significant loci associated with the 18q 21.2 region, showing pleiotropic association with all eight disorders.</p></li>
<li><p>Anorexia, OCD, and, to a smaller extent, Tourette’s syndrome clustered together at the genetic level.</p></li>
</ol>
<p>Across the cross-disorder GWAS of anorexia and OCD, it was found that the genetic correlation of the anorexia-OCD cross-disorder phenotype resembled the genetic correlation patterns of both disorders. When the unique contribution by anorexia and OCD was examined, the metabolic and anthropometric correlations observed were driven primarily by anorexia and not OCD. Also, a gene-centric enrichment analysis using anorexia and OCD datasets revealed an overlap in many common features of brain regions and developmental stages for anorexia and OCD, suggesting a role for future sequencing efforts alongside GWAS to better understand the biological nature of the shared risk between anorexia and OCD.</p>
<p>Mendelian randomization analysis uses linkage equilibrium independent genome-wide significant SNPs identified in GWAS as instrumental variables for a given exposure and measures the degree to which the exposure is causally associated with the outcome. In the most recent PGC GWAS of anorexia, MR analysis revealed a causal bidirectional relationship between anorexia and BMI, whereby genetic risk variants for anorexia led to lower phenotypic BMI, and genetic variants for lower BMI led to phenotypic anorexia in general population samples. These results point towards a positive causal association between high BMI and eating disorder behaviors and symptoms.</p>
<p>Adiponectin is a fat-derived hormone that plays a key role in energy homeostasis and appetite regulation. Altered adiponectin levels have been observed in patients with anorexia and bulimia. An MR study found that higher blood adiponectin was causally associated with eating behavior disinhibition. These studies point towards innate biological drivers that may lead towards symptoms of eating disorders.</p>
<p>MR for eating disorder research is in its infancy since the strength of genetic instruments in MR is determined by well-powered GWAS. Recent MR approaches allow for non-linear associations, which may significantly advance the application of MR in disorder research in the future.</p>
<p>Genetic studies have started to explore the role of rare and structural variants in eating disorders. Regarding studies of copy number variants, one study found no evidence that anorexia cases had a significantly higher burden of CNVs than controls. However, they identified a recurrent 13q12 deletion in two cases and CNVs disrupting the <em>CMTN6</em> and <em>CNTN4</em> region in several other cases. Another study found another anorexic case with the deletion in the 13q12 region, and the authors observed two instances of CNVs with at least 50% reciprocal overlap with regions associated with psychiatric and neurodevelopmental disorders. In addition, mixed results have been found for mitral duplications at 15q11.2 region.</p>
<p>Whole exome and whole genome analyses have also provided evidence for an enrichment of rare variants in anorexia. A whole exome analysis in two independent families with males with anorexia found variants in the <em>EN1</em> gene, which is involved in brain development. Another study combined exome sequencing, whole genome sequencing, and linkage analysis to examine two families for the recurrence of anorexia. They found a missense variant co-segregating with the affected family members in the Estrogen-related receptor alpha and a potentially damaging mutation in the Histone DSLs4 in the second pedigree. These genes are linked to the estrogen system.</p>
<p>A whole genome sequencing analysis in six individuals, two maternally linked cousins with severe anorexia, and their parents found that, of the approximately 5.3 million variants per individual analyzed, almost 500,000 variants were shared identical by descent by the cousin pair. They identified novel variants in seven genes. These findings suggest that there may be utility in whole genome sequencing of families with affected individuals to detect rare variants that may influence anorexia. Despite strong evidence for the heritability risk of anorexia, rare variant contributions of large effects have not yet been identified. Early studies show promise, and larger-scale studies with well-matched control groups and replication studies will be necessary for determining whether rare and structural variants contribute to eating disorders.</p>
<p>Gene expression offers insight into the genes and molecular mechanisms that influence phenotypes. One study, which investigated the brain regions’ enrichment for gene expression to understand the molecular neuroanatomy of anorexia, combined the gene lists from two common variant studies, a rare variant study, and a stem cell study. They used genetic and transgenic resources spanning human fetal and adult, as well as mouse gene expression data. Genes associated with anorexia resulted in subcortical feeding and reward circuits, and furthermore, they implicated the microbial genes and genes responding to fasting in mice’s hypothalamus. Likewise, the PGC’s GWAS of anorexia found an enrichment of gene expression in the CNS brain tissues and striatal hippocampal neurons linked to feeding and reward. Another study applied transcriptome expression profiling in anorexia from inpatient admission to discharge of the top differentially expressed genes. Findings revealed that the <em>CPA3</em> and <em>G82</em> expression were positively associated with levels of leptin, a hormone linked to nutritional status and the immune response.</p>
<p>Another study examined gene expression in anorexia before and after weight restoration. Among the top 20 genes reported, down-regulation of the cholesterol side chain cleavage enzyme and up-regulation of genes related to protein secretion, protein signaling, defense response to bacterial regulation, and olfactory receptor regulation were observed.</p>
<p>Another study modeled anorexia using induced blood protein stem cells, with the transcriptomic analysis revealing a novel gene, <em>TARC1</em>, that may contribute to anorexia pathophysiology. The <em>TARC1</em> gene encodes a neurokinin-1 receptor, which is involved in a range of biological processes, interacts with several neurotransmitters, and has previously been associated with anxiety disorders, bipolar disorder, and ADHD, suggesting a novel system that might contribute to anorexia symptoms. Although several studies on gene expression in eating disorders exist, there are not many, and most have small samples, limiting the conclusions that can be drawn.</p>
<p>In epigenetic research, global methylation results have been mixed, with largely inconsistent findings and opposite effects. EWAS, however, identified <em>TNXB</em> hypermethylation, although the significance level was nominal, and therefore, a false-positive finding cannot be ruled out. <em>TNXB</em> plays a role in maintaining muscles, joints, organs, and skin, and regulates the production of collagen. Further studies would need to replicate this finding using larger samples, but these early findings could indicate epigenetic changes in people with eating disorders.</p>
<p>Notably, future studies need to be well-designed in order to disentangle epigenetic differences in eating disorder patients by disorder type, tissue type, cell type, and take into account large numbers of environmental factors, such as diet, binge eating, and purging behaviors, as well as medication. Gene-environment interaction studies have predominantly focused on candidate genes relating to behavior, emotions, and the stress response, such as serotonin and glucocorticoid genes. However, these candidate studies are subject to false positive results as well.</p>
<p>Another avenue to study gene-environment interaction is via the use of genetic risk scores to capture the genetic effect. Recent studies have begun modeling genetic risk scores caused by environmental interactions in various psychiatric disorders, with some very interesting findings. However, this methodology is still in its infancy and has not yet been applied to eating disorders due to a lack of well-powered GWAS needed to calculate the genetic risk scores themselves. In the future, we are likely to see great advancement in this area of study. Thank you very much for listening.</p>
<p><strong>Zeynep Yilmaz:</strong></p>
<p>The second area is the expansion of eating disorder phenotypes. Currently, anorexia nervosa is the only eating disorder for which we have GWAS results, and efforts are underway to include other eating disorders, such as bulimia nervosa and binge eating disorder, in future GWAS. Taking this one step further, we also need to gain a better understanding of how phenotype measurements, more specifically with diagnoses obtained through clinical interviews serving as gold standard, how register data, electronic health records, questionnaire-based algorithms, and self-report may affect genomic findings. Finally, future GWAS efforts should also include intermediate phenotypes that go beyond an eating disorder diagnosis, as well as symptom-level investigations.</p>
<p>The third area is functional genomics. As we continue to discover common and rare variants associated with eating disorder risk, functional genomic analyses will be needed in order to convert these insights into an understanding of the underlying biological mechanisms. As sample sizes increase and we have better statistical power, we will be able to utilize an array of computational methods, such as statistical fine mapping, eQTL analysis, TWAS, gene set enrichment analysis, as well as molecular biology-based approaches, such as RNA-seq, ChIP-seq, Hi-C chromatin accessibility assays, and knock-out animal models, for future functional follow-up.</p>
<p>To summarize the state of the eating disorder’s genetic research as per our review, over the last few years, we have gained tremendous insights into the genetic architecture of anorexia nervosa, and we will continue to do so as our sample sizes increase. As of now, several disease-associated loci and over 130 genes have been linked to anorexia nervosa, and we have uncovered genetic correlations with psychiatric, cognitive, metabolic, and anthropometric traits. Now, efforts are needed to elaborate on the functional context of these discoveries, and a peek over the horizon into clinical management suggests that patient screening, care, and outcomes may improve from advances in molecular genetics.</p>
<p>And as discussed earlier, genomic discovery depends on very large sample sizes and large-scale global collaborations. There are numerous international projects taking place to boost the sample sizes for anorexia nervosa and to add other eating disorder phenotypes to future GWAS. For instance, in the next few years, large-scale studies such as the Eating Disorders Genetics Initiative (EDGI) and Binge Eating Genetics Initiative (BEGIN) will be important to watch for advances in progress. As they, among other ongoing studies, will make significant contributions to the work carried out by the Eating Disorders Working Group of the Psychiatric Genomics Consortium.</p>
<p>And with that, on behalf of my co-presenters, I would like to acknowledge and thank our co-authors on this paper. We would also like to acknowledge and thank PGCD co-chairs Cynthia Bulik and Gerome Breen, the International Society of Psychiatric Genetics for this platform, Psychiatric Genomics Consortium for this opportunity, our funding sources, and we would also like to thank you for your time and your attention.</p>
<hr>
</section>
<section id="sec-mdd" class="level1">
<h1>Major Depression</h1>
<p><strong>Title</strong>: Major Depressive Disorder: Introduction and General Epidemiology</p>
<p><strong>Presenter(s)</strong>: Kim Kendall, Evelien Van Assche, Till Andlauer, Karmel Choi, Jurgen Luykx, Eva Schulte, Yi Lu</p>
<p><strong>Kim Kendall</strong>:</p>
<p>Hello everyone and welcome to this review of major depressive disorder by early career researchers of the major depressive disorder work group of the PGC. My name is Kim Kendall and I’m a psychiatrist from the MRC Center for Neuropsychiatric Genetics and Genomics at Cardiff University and I’m going to talk to you about the phenotype of depression and its epidemiology.</p>
<p>Depression is a common mood disorder which has the potential to substantially impact on an individual’s morbidity and mortality. It’s characterized by the core features of depressed mood, which may manifest as sadness but also feelings of irritability or emptiness and anhedonia, which is a decreased ability to feel pleasure. Individuals with depression will tend to also experience a wide range of symptoms, which can be broadly categorized into emotional, neuro-vegetative, and cognitive symptoms.</p>
<p>Depression is a very phenotypically heterogeneous disorder, and a study of individuals with with the disorder from the star D trial found over a thousand unique symptom profiles and the commonest was only endorsed by 1.8 of the sample. The term depression is also used inconsistently both in research and in clinical practice, and a diagnosis can be reached in multiple different ways using different criteria, so this clearly has implications for translating research findings to the clinic. Some have suggested that the disorder might be better conceptualized as an umbrella term for multiple related disorders or disorder subtypes which have different risk factors.</p>
<p>The 12-month prevalence of depression is thought to be around 6% and the lifetime prevalence between around 15 and 18%. The peak of onset extends from mid to late adolescence to the early 40s, with the median onset being in the mid-20s.</p>
<p>Established epidemiological risk factors for depression include age, sex, marital status, and socioeconomic status. There are elevated rates of co-morbidity with other psychiatric disorders and also physical health disorders. In some situations these associations can move in either direction, but these this is difficult to untangle and unpick with the existing data.</p>
<p>So to summarize, depression is a common, phenotypically heterogeneous disorder, which has multiple risk factors crossing the biological, psychological, and social domains. There are high rates of co-morbidity with other disorders and some untangling some of these associations can be quite difficult.</p>
<p>I’ll now hand you over to Lu Yi for her section of the review.</p>
<p><strong>Lu Yi</strong>:</p>
<p>Hi everyone, this is Lu Yi. Today I’m going to talk about the genetic epidemiology of MDD. It is really clear that MDD is a complex disorder so neither gene or environment plays a role alone. Early evidence suggests that MDD runs in families, with increased risk in the first degree relatives of MDD patients compared to those in controls. But how much is attributed to genes and the environment respectively? The answer to which will guide our current gene mapping effort.</p>
<p>Heritability is a key concept in genetics. It is the proportion of phenotypic variance attributable to additive genetics. It is typically estimated from twin studies based on the expected genetics and environmental sharing within the twin pairs. There is a long history of twin studies. Here, I will just go through a few key papers. The first one is a meta-analysis from two decades ago and yet this estimate of 37% is still the one most cited for MDD heritability to date. The next one is the single largest twin study and the third one is the largest meta-analysis of all twin studies in the past 50 years. The last one is a more recent study; it has a higher higher estimate potentially due to higher severity in the treated cases. Overall the heritability is made from the twin studies are fairly similar.</p>
<p>But twin studies rely on some key assumptions especially the common environmental assumption between the two twin types. This was relevant in the missing heritability debate, because some suspected that disease twin heritability might have been overestimated. What are other lines of evidence? There is a growing literature of heritability estimation based on population-based register or electronic health record data. These studies reconstruct the extended pedigree, based on either recorded or inferred familial relationships. The estimates here are fairly similar to those from twin studies, so they seem to be very limited evidence for overestimation.</p>
<p>Now is there a sex difference in MDD heritability? The motivation there is to see if the higher female prevalence can be explained by a higher genetic component. Now as shown in the figure, the heritability does seem to be higher in females than in males, but only meta-analysis failed to detect a significant difference, whereas the other study showed a genetic correlation between sexes much lower than one but with wide confidence interval. The more recent study, based on the siblings, were able to provide a much more precise estimate of about 0.9, so despite a higher MDD heritability in females, the majority of genetic risks are still shared across sexes.</p>
<p>Now given the low heritability of MDD, it will be really useful to identify clinical subtypes that are more heritable. One successful example is a converged consortium where they ascertained a recurrent MDD cases in women to enhance this GWAS power. Early evidence was only able to establish recurrent mdd as a more heritable form, but recent studies using the large population-based data were able to extend the evidence to suggest that the subtypes of early age of onset comorbid anxiety disorder, higher severity, and postpartum depression as more heritable forms.</p>
<p>Now to summarize MDD heritability is within the range of the 30 to 50 percent, but it is important to note that heritability is population- and time-specific. There are six differences and subtypes that are more heritable. The combination of this high prevalence and low heritability makes it really challenging for gene mapping, which you will hear more from Eva next, thank you.</p>
<p><strong>Eva Schulte</strong>:</p>
<p>Good morning, good afternoon, and good evening. My name is Eva Schulte and I work at the department of psychiatry and the Institute of Psychiatric Phenomics and Genomics at the University of Munich. i would like to tell you a little bit about candidate genes and rare variation in MDD genetics.</p>
<p>Ever since people discovered that there was this underlying heritable component to MDD, they set out to try to identify that genetic component. So over the course of 40 years, more than 1 500 individual studies and more than 200 genes were assessed. Candidate genes belong to pathways considered either biologically possible or implicated in drugs used in the clinical practice to treat MDD. Most of these studies were pretty small, and even in large meta-analyses of the data that is out there only seven common variants showed significant association with the MDD phenotype, and that is really not much more than would be expected by chance. So overall these results were very inconsistent, and the studies that have been performed only had limited power, and also did not correct for population stratification.</p>
<p>The candidate genes that have been suggested all have minor allele frequencies and odds ratios that would make them detectable in the current GWAS that we have. However the current GWAS have not lent support to either the classical candidate genes or the pathways that they’re implicated in. Also systematic analyses of the most commonly studied polymorphisms in 18 candidate genes in the UK Biobank data and the PGC MDD data found little evidence of association with several different definitions of depression. So overall there is no sound evidence for genetic variation in candidate genes in the context of depression, and many funding bodies, and also many researchers, have suggested that we stop looking for it.</p>
<p>So genome-wide association studies have been very successful in identifying genetic factors implicated in MDD. So why are we even interested in looking at additional genetic variants if we already have these common variants? The reason is that common variants only harbor very small effect sizes, and therefore the identification of larger rare alleles which are expected to harbor larger effect sizes would be very beneficial because it is much better able to inform the underlying biology of the disease and, to me, at least that is the main reason why I’m interested in studying the genetics of MDD.</p>
<p>Rare variants are currently mostly analyzed using whole exome sequencing data, however costs are still prohibitive and sample sizes that are needed are so large that, at least to my knowledge, there is no effort currently to compare just MDD cases to controls. And many people have resorted to assessing some phenotypes such as sex differences, suicide completion, treatment resistance, family studies, and population isolates, and early onset MDD. Well of course it is very difficult to summarize all these studies into just one sentence, i think that it is fair to say that overall there is an enrichment in singular rare and low frequency variants in specific cases, but most of these studies fail to reach both statistical significance due to limited power and they also lack replication.</p>
<p>Next to the rare single nucleotide variants, copy number variants may present another form of genetic variation that may be implicated in the genetic architecture of MDD. However, here too one of the problems is that very few high-powered studies have been performed, although there are ongoing efforts by the PGC to rectify that. One large-scale study showed that there was an increase in genome-wide burden of rare short deletions in MDD which map mostly to intergenic regions and to enhancer regions, suggesting that they may alter RNA expression levels. Also Kim Kendall, using a large-scale data from the UK Biobank, showed that there were 53 CNVs of known association with neurodevelopmental disorders, which are also associated with self-reported depression. Taken together these data suggests that there may be a risk increase which is carried by CNVs, however there is no evidence for large effect multi-genic CNVs such as those that we see in schizophrenia or autism spectrum disorders in the genetic architecture of MDD.</p>
<p>To summarize, genetic variation in candidate genes does not seem to contribute to the genetic architecture of MDD. Rare variants and rare CNVs may influence MDD risk, however no final conclusion can be drawn at this point with regard to the amount of heritability that they carry. Still it is useful to look for rare variants because of the larger effect sizes and therefore the better ability to inform underlying biology that they carry compared to common variants. With that I would like to thank you for your attention and I would like to hand over to Till Andlauer who will tell you a lot more about the role of common variants and the genetic architecture of MDD.</p>
<p><strong>Till Andlauer</strong>:</p>
<p>Hi, I’m Till and like Eva I’m also from Munich and I’m going to talk about genome-wide association studies on MDD. If you don’t know what a GWAS is at all then you could also watch my other video called “What is a GWAS?” that introduces you to the topic.</p>
<p>Unlike the rare variants that you’ve just heard about, GWAS are typically conducted on SNPs with a minor allele frequency of at least one percent or higher. Until 2015, GWAS in MDD identified either no genome-wide significant loci, or very few, up to two in the Chinese study by the converged consortium. This changed from 2016 on, when the sample sizes increased and more and more genomic significant loci were identified. These increased sample sizes were mainly made possible by the efforts of the PGC, but also by 23andme and UK Biobank that also contributed large samples.</p>
<p>I’m going to introduce you to the biology of MDD as discovered by genome-wide association studies and for this I’m going to use the 2018 study by the PGC. This study analyzed about 135,000 MDD cases, and they were mainly from cohorts from the PGC, but they also included several other cohorts, notably a large one from 23andme, so among these 135 000 cases were many with a self-reported diagnosis of depression.</p>
<p>This study identified genome-wide significant variants at 44 independent loci, and as is always the case in genome-wide association studies, it’s difficult to tell which genes these variants actually influence. But when using positional overlap of genes at these identified loci, one can see that many of the putative genes at these associated loci have functions in neurite outgrowth, in synaptic function and plasticity, but also in immunity and inflammation. This study also conducted MAGMA-based gene level and gene set analysis, and in these gene level analyses many genes were found that code for pre- and post-synaptic proteins, and especially receptor subunits, for example voltage-gated calcium channels, the dopamine receptor DRD2, and many different glutamate receptor subunits.</p>
<p>Gene set analysis found pathways that are important for neuronal differentiation and protection for the immune response and mainly for synapse function. And here it really appears to be synaptic plasticity, and especially post-synaptic plasticity, that plays in a very important role in the development of depression and other common psychiatric disorders like, for example, schizophrenia, with all of its players from pre-synaptic picollo, to voltage-gated calcium channels in the post-synapse, to RNA-binding proteins like FMRP that regulate local translation and regulate how glutamate receptor subunits are either incorporated or removed from the synaptic membrane. All of this appears to be really important in the development of psychiatric disorders, at least from the genetic point of view.</p>
<p>New GWAS on MDD are now being published basically every few months. The last published one is from last year, and it identified 102 independent variants already and to achieve this it combined cases from the PGC, from 23andme, and from UK Biobank, so it’s a combination of cases with different phenotype definitions. In this GWAS, the variance explained by depression polygenic risk scores remain low, so the pseudo-r square ranged from 1.5 to 3 percent, and that’s not too good yet. Recently, a preprint of an even larger GWAS on depression has been uploaded. It analyzed 340,000 cases, and, in addition to the cases already analyzed before, it now includes also the Million Veteran Program. Importantly, it also includes trans-ancestry meta-analysis with African-American samples.</p>
<p>Well, to sum up so far, dozens, or better, hundreds of independent variants associated with depression have been identified, but each of them has very small individual effect sizes, so odds ratios of up to 1.05. The identified genes indicate that pathophysiological mechanisms of neuron development, synaptic plasticity, and also inflammation might play an important role, and many of the identified genes overlap with findings for other psychiatric disorders, as for example schizophrenia. In general, one can say that maybe the identification of depression risk variants has been a bit more challenging than for other psychiatric disorders like schizophrenia, and that’s likely because MDD has a higher prevalence, a lower heritability, and possibly the patients are more heterogeneous. It is important to point out that phenotype definitions matter for depression, so it makes a difference, you get different genetic results on whether the cases were defined by the diagnostic interviews, or self-reports, or ICD codes or whatever. So phenotype-induced heterogeneity might be an important focus of future work like trans-ancestry analysis. And now Jurgen Luykx will tell you more about genetic correlations.</p>
<p><strong>Jurgen Luykx</strong>:</p>
<p>I’ll briefly discuss the genetic correlations that have been found for MDD over the past decade or so, maybe less. First to start, what did I do to get to these genetic correlations? First, like a brief introduction to what exactly genetic correlation is. It’s a statistical measure of correlation between genetic underpinnings of one trait, and genetic underpinnings of the other, and it can range from 1, being perfect positive genetic correlation, to -1, which is perfect negative. So i extracted the genetic correlation results from recent genome-wide association studies, where I focused on the largest GWAS for MDD, but also some other studies that specifically delved into other kinds of phenotypes, to see whether there’s genetic correlations.</p>
<p>So first this may be a graph you’re familiar with; it’s derived from the recent Cell cross-disorder paper that came out last year, where basically what you can appreciate from the graph is that MDD is almost right in the middle, grouping with bipolar disorder and schizophrenia on the one hand, and also with autism spectrum disorders and ADHD, and it’s associated with 48, so almost half, of the 109 plyotrophic loci that have been found across these eight main psychiatric disorders.</p>
<p>So i think first to show a graph from the Brainstorm paper where psychiatric and neurological disorders were compared and genetic correlation analyses were run. Basically there’s very few genetic correlations between neurological and psychiatric traits, except for MDD and migraine and ADHD and migraine. For the rest, there’s very little overlap across neurological traits and also between MDD on one hand and other neurological traits.</p>
<p>So I think this is an important graph to show the like the main genetic correlations that have been found in the two largest GWAS studies that have also been discussed by Till. On the left you see the Wray et. al Nature Genetics 2018 paper, where on the top you see that depressive symptoms, not surprisingly, list very high with a genetic correlation of almost 1 (0.9). On the right panel, you see a graph from the Howard, et. al Nature Neuroscience 2019 GWAS, where basically depressive symptoms were also ranking highest. Not surprisingly, I think, for psychiatrists, neuroticism also ranks really high, and when you look at genetic negative correlations on the right you see that subjective well-being, college completion, are negatively associated with MDD.</p>
<p>Another study that I found of interest is a study relating different kinds of mood disorders, such as bipolar 2 disorder and bipolar disorder type 1, and different kinds of major depressive disorders, so recurrent single, and sub-threshold depression, and what you can appreciate from this graph, where the genetic correlations are also depicted in these numbers, is that recurrent major depressive disorder has a way higher genetic correlation, more than double, with bipolar disorder type 2 than with bipolar disorder type one. I think, as a clinician, I think you can you know, you can basically understand, because it’s also apparent in the clinic, where sometimes bipolar 2 disorder has less clear manic episodes, hypomania, whereas bipolar one disorder has clear manic episodes of being kind of a distinct phenotype. I think this is kind of confirming what we also see in clinical practice a lot. And obviously recurrent major depressive disorder single episode, major depressive disorder, and sub-threshold disorder depression have very high genetic correlations if you compare them to other mood disorders.</p>
<p>And then atypical symptoms: So these are symptoms that are not commonly seen in most depressive episodes, maybe they’re seen they’re observed in about 10 to 20 percent of major depressive episodes, so these are increased appetites, and increased weight, and so if you look at the graph on the right this is a genetic correlation graph with BMI. What is apparent is that this subtype of MDD with atypical symptoms, so increased appetite and weight, is strongly genetically correlated with BMI, whereas other subtypes of MDD (decreased appetite for example) are not associated with BMI.</p>
<p>So in conclusion, genetic correlations for MDD are strongest for mood and personality-related traits, being social well-being depressive symptoms and neuroticism, and with that i would like to conclude and thank you so much for your attention.</p>
<p><strong>Karmel Choi</strong>:</p>
<p>Hi my name is Karmel Choi, and I’m a clinical and research fellow at Mass General Hospital Harvard Medical School and the Harvard School of Public Health. My research focuses on data-driven methods to inform our understandings of psychological resilience, protective factors, and depression prevention. As my colleagues have now mentioned, genetic factors play an important role in influencing depression risk. However, environmental factors still explain a large proportion of variation in depression outcomes, and as such scientists have been very interested in how genes might interact with these environmental factors to shape depression risk.</p>
<p>The earliest efforts involved candidate gene-by-environment interaction, and a global summary of these efforts is basically: no robust, large-scale evidence for interactions involving candidate genes like the serotonin transporter gene or otherwise, in combination with environmental exposures, when it comes to depression.</p>
<p>So moving from candidate genes to empirical evidence that has drawn on genome-wide data, this early study from Roseanne Peterson, Ken Kendler and the CONVERGE team, found that genetic influences on depression seemed to vary by environmental exposure. and they identified GWAS hits for depression only in individuals who were unexposed to adversity. And though the differences were not significant, they observed slightly higher SNP-based heritability in the unexposed group as well.</p>
<p>On the other hand, recent findings from Joni Coleman, Gerome Breen, and team in the UK Biobank, have suggested somewhat of the opposite: where heritability of depression was actually higher among individuals who reported lifetime trauma exposure compared to those who did not report such exposure. And so these studies drew on quite different populations and samples, and suggests that the genetics of depression may vary depending on the characteristics and the type of environment under study.</p>
<p>There have been other studies now that have used polygenic scores to aggregate genome-wide effects and examine how these scores interact with life adversities. A study from NESDA in the Netherlands found a significant interaction where polygenic risk was more strongly associated with depression in individuals who were exposed to childhood trauma. On the other hand, a study led by Niamh Mullins in the UK found the reverse, where polygenic influences were stronger among individuals who were unexposed to childhood trauma. And then a later combined effort, that included these prior studies as well as some others, found not much evidence for polygenic interactions with childhood trauma. Despite this being, you know, a large pooled effort, the combined sample size was still quite small as you can see.</p>
<p>And so in the UK Biobank, which is one of the largest single databases available right now, groups have found significant interactions between polygenic risk for depression and traumatic life events. including Joni’s earlier study that found an additive interaction, and Shen from Andrew McIntosh’s group in Edinburgh, reporting a similar finding where polygenic effects for depression were stronger among individuals exposed to childhood trauma specifically though not necessarily later life events.</p>
<p>And so by far the most common gene environment interaction studies have focused on stress, trauma, and life adversity, but other potential exposures have also been examined. Our group, as well as colleagues in the Netherlands, have looked at social support as another kind of environmental factor. And what both studies found, in quite different samples, are main effects, where polygenic risk is associated with more depression as expected, social support is associated with protective effects on depression, but these effects were relatively independent of each other. Similarly we recently looked at physical activity as another kind of exposure, and again we found independent non-interactive effects with polygenic risk. And this suggests that even if you are at higher genetic risk for depression, higher levels of physical activity may still help to reduce the risk of depression. And this may apply for other environmental factors as well.</p>
<p>Finally another genetically informed approach to mention is to look at environmental effects on depression using mendelian randomization as a way to triangulate potential causal effects. We recently provided an example of this by looking at the relationship between physical activity and depression using two-sample MR. And we recently extended this to look at a wider range of other modifiable exposures that were screened in the UK Biobank. And so there are a lot of exciting directions for gene environment work. There are new methods to study whole genome by environment-wide interactions, we need to rule out gene environment correlations, and also address the heterogeneity of measurement of environments. And we will continue to need large and well-powered analyses in diverse samples to be able to test and detect these interactions. So with that i will pass this talk along to Evelien who will wrap things up for us. Thank you so much.</p>
<p><strong>Evelien Van Assche</strong>:</p>
<p>Hello everyone my name is Evelien Van Assche. I’m a psychiatrist currently working in Germany, and I have the pleasure to conclude this presentation with a short summary.</p>
<p>So we’ve been discussing depression, which is a highly prevalent common disorder, very debilitating for the patients. We have discussed that it’s phenotypically very heterogeneous and it has a heritability of around 30 to 50 percent. Studies have shown that there is a strong polygenicity, with over 100 loci that contribute to disorder risk, which is related to neuronal growth, synaptic function, and inflammation. It is possible there are some subtypes that are more heritable, including early onset depression, and, for example postpartum depression, but also the recurrence and severity of depression can be considered as potential subtypes.</p>
<p>Furthermore we see an important role for environmental factors, including stressors such as childhood trauma, socioeconomic adversity, but also protective aspects such as social support and physical activity. In short we can state that an individual’s genetic makeup likely impacts the disease course from prevention, to diagnosis, treatment, and prognosis. This gives us potential applications for genetics and depression, for example the development of pharmacological treatments can be genetically informed and genetically informed pharmacological modes of action can be targeted.</p>
<p>Furthermore we see a role for genetics in the discussion regarding disorder risk and treatment choice. When we are talking about risk stratification in prevention and interventions and personalized medicine. And also we see a risk for genetic counselling for worried parents from one side, but on the other side also for the patient itself it comes about a shared risk of depression genes, but also physical health phenotypes.</p>
<p>We still have a to-do list to finish as researchers in genetics and depression, for example the exploration of stratification properties of depression where more homogeneous disorder subtypes would be welcome. This could also help us increase the accuracy of the depression polygenic risk score, within new opportunities for risk stratification at the population level. Furthermore, this would also increase our understanding when it comes to the etiology, but also prevention and treatment opportunities that can benefit the patient.</p>
<p>So in conclusion understanding the genetics of depression helps us to improve our insight into the environmental and genetic factors in depression which in turn help us to improve clinical care, and let’s hope it holds a bright future for the field. I would like to thank you for your attention.</p>
<hr>
</section>
<section id="sec-ocd" class="level1">
<h1>Obsessive-Compulsive Disorder</h1>
<p><strong>Title</strong>: Genetics of Obsessive-Compulsive Disorder: What we know in 2020</p>
<p><strong>Presenter(s)</strong>: Christie Burton</p>
<p>Hi, my name is Christy Burton and I’m a research associate working at the Hospital for Sick Children and I’m going to talk to you today about what we know about the genetics of obsessive-compulsive disorder [OCD]. This material comes from a review that’s forthcoming in a PGC issue of Psychological Medicine. So, I’m presenting today on behalf of my co-authors who are listed below [Behrang Mahjani, Katharina Bey, Julia Boberg, Christie Burton].</p>
<p><strong>Introduction to Obsessive Compulsive Disorder [OCD]</strong></p>
<p>OCD is a psychiatric disorder that’s characterized by obsessions which are intrusive and wonted distressing thoughts that cause a lot of anxiety and that often precipitate compulsions which are repeated or ritualized behaviors. Now, the nature of these obsessions and compulsions can vary a lot from person to person so the presentation of OCD is quite heterogeneous. But we do know that symptoms tend to fall into clear factors and the most common factor structure identified in meta-analysis is hoarding, symmetry, forbidden thoughts, and cleaning. OCD is also considered to be a common disorder. The lifetime prevalence is between one and three percent. it’s also thought to be a brain-based disorder with a lot of converging evidence coming from neuroimaging, genetics, and pre-clinical work - with the corticosteroid thalamic circuit being one of the main pathophysiological mechanisms that’s been implicated.</p>
<p>In terms of the epidemiology of OCD, when we look at age of onset there does seem to be two peaks in the distribution: one in childhood and the other in early adulthood. And when we look at children with OCD two-thirds of cases tend to be boys, but the prevalence rates between males and females is about equal in adulthood. OCD also co-occurs with several psychiatric disorders; the most common being major depressive disorder, but also various anxiety disorders like generalized anxiety disorder, Tourette’s, as well as anorexia and obsessive-compulsive personality disorder. There are several risk factors that have been associated with OCD - obviously we’re going to spend a lot of time talking about genetics today, but there’s also environmental factors like stressful life events, perinatal complication, childhood trauma, as well as infection in the case of Pandas or Pans</p>
<p><strong>Key Findings from OCD Genetic Research to Date</strong></p>
<p>Now we’re going to go over some of the key themes or messages that have come out of OCD genetic research to date. So first, is that OCD is familial and that’s clearly illustrated in these pedigrees where we look at the green symbols which represent OCD phenotype - so a lot of clustering within families and across generations. And when we look at the recurrence risk among first-degree relatives, some estimates are as high as 50 percent. But a lot seem to cluster within the 10 to 20 percent range which is considerable when we think of the prevalence of OCD in the general population [1-3% lifetime prevalence].</p>
<p>We also know that genetics play an important role in OCD. So, the concordance rate in monozygotic twins is consistently twice that of dizygotic, or fraternal, twins. And the heritability estimates from twin studies range between 27 and 65 percent with slightly greater estimates found for childhood-onset rather than adult-onset OCD. Now recent evidence from family studies provides further evidence for the additive genetic effect in OCD. And the effects coming from mothers tend to be genetic rather than environmental. So, there’s a lot of converging evidence coming from twin family studies that implicate genetics as an important factor in the etiology of OCD.</p>
<p><strong>OCD is Highly Polygenic</strong></p>
<p>We also know that OCD is highly polygenic - so there is no one genetic variant for OCD, there’s likely to be hundreds of thousands of genetic variants that are contributing to OCD risk. A lot of this evidence comes from genome-wide association studies [GWAS]. So, the largest published GWAS of OCD to date comes from a meta-analysis of two large consortia, the IOCDF and OCGAS and all those which are now part of the PGC-OCD sample. And although the study didn’t identify any genome significant variants, it did identify variants that are in, or close to, genes that have previously been associated with OCD or involved in brain function. SNP [single-nucleotide polymorphism] heritability estimates from the study were around 25 percent, which is consistent with previous studies. There is a very large meta-analysis of OCD GWAS that’s forthcoming from the PGC-OCD group with some very promising results that hopefully will be published soon.</p>
<p>There have also been two genome wide association studies [GWAS] of obsessive-compulsive traits and symptoms in the general population. Interestingly, both of them have identified genome-wide significant variants. The most recent, which comes from the group that I work with, showing a genome-wide significant variant in <em>PTPRD</em> - a gene that’s come up a few times in OCD genetic studies. This particular variant was also associated with OCD diagnosis in independent samples, so there’s definitely some converging evidence for this variant. Now what’s interesting, is when we look at the SNP heritability for OCD symptoms and traits, it does seem to be a bit lower than what we see for OCD diagnosis and the reason for that still unclear. Although, it does seem to occur in other disorders like ADHD. And there’s also a very large initiative within the PGC-OCD group to conduct a large genome wide association study of OCD compulsive symptoms and traits and that’s currently ongoing.</p>
<p><strong>Rare Variants in OCD</strong></p>
<p>Now there was some evidence from a few years ago that suggested that a lot of the genetic effects from OCD were coming from common variants, but there is definitely evidence that rare variants also play a role in OCD. So, when we look at rare copy number variants, or CNVs, the very consistent finding is that there’s no increased overall burden in rare CNVs for OCD. This is quite different from what we see for other disorders like autism or schizophrenia. But there does seem to be an increased number of rare CNVs in gene sets that are related to brain function or in genes that have previously been associated with neurodevelopmental disorders. Some of the CNVs that have been identified in CNV cases include <em>PTPRD,</em> which again is coming up in findings, as well as <em>BTBD9</em>, <em>NRXN1</em>, and 16p13.11. There are still a lot of the CNV studies today that have been relatively small and there’s an initiative within the PGC-OCD group to conduct a larger scale CNV study of OCD.</p>
<p>When we look at CNVs and Indels, this research is so very much in its infancy as there are very few studies using next generation sequencing in terms of OCD, but one such recent study that conducted whole exome sequencing in a few hundred trios really focusing on <em>de novo</em> variants showed that <em>de novo</em> damaging variants were more common in OCD cases compared to controls. And that these damaging mutations conferred risk for OCD in about 22 percent of cases. Now we don’t know how much risk they conferred in those cases, and there’s actually just a lot to be learned still in the rare variance space when it comes to OCD, especially as larger samples become available.</p>
<p><strong>OCD Dimensions have Shared and Unique Genetic Risk</strong></p>
<p>Now we also know that OCD dimensions have both shared and unique genetic risks. So, a recent study in a large sample of twins looked at the SNP heritability of compulsive symptoms versus obsessive symptoms, and compulsive symptoms had twice the SNP heritability of obsessive symptoms. Although both [heritability estimates] were relatively low. And only compulsive symptoms were genetically correlated with diagnosed OCD in the PGC sample whereas obsessive symptoms were not [correlated] at all. So there definitely seems to be some differences in the genetic architecture of those two very broad symptom types.</p>
<p>Now a slightly older twin study that looked at symptom dimensions in the more typical factor structure, broke down the genetic variance for each dimension by the shared variance versus the unique variance. And even though [their findings are] supporting this previous twin study, there’s definitely genetic variance that’s unique to most of the symptom dimensions in OCD. There is a large proportion of the variance, the dark gray part of the bars, that is shared across the OCD dimensions. Now one seemingly outlier here seems to be hoarding and there is some growing evidence that hoarding may have slightly different genetic architecture which supports the separation of hoarding into its own disorder in the DSM. But there’s still a lot more to be investigated there. And again, there is an effort within the PGC-OCD group to conduct some large-scale GWASs of hoarding.</p>
<p>We also know that OCD seems to share genetic risk with conditions that it’s comorbid with. These are results from two large studies that are pretty recent with very large samples and very similar results: So, OCD is genetically correlated with disorders that it’s commonly comorbid with, like MDD and anxiety, but it’s most genetically correlated with Anorexia and Tourette’s. It does seem to form this compulsive behavior cluster in terms of genetics. So, these results suggest that some of the phenotypic core correlations that we’re seeing for certain of OCD’s comorbidities are driven by genetics, whereas others may be not as genetically driven.</p>
<p>And finally, we also know that risk for OCD seems to be shared quite a bit between males and females. This is a study conducting a sex stratified analysis using PGC data. As you can see, although there are no genome-wide significant variants, there’s definitely peaks that are distinct between males and females. And in fact, in a gene-based analysis there were two genes that were significant for OCD in females only, this was in grid two and grp-135.</p>
<p>So, although there are some distinctions when you look at the genetic correlations, they’re extremely high. There’s definitely likely to be some sex-specific variants between males and females, but there is a large proportion of the genetic risk that is shared between sexes.</p>
<p><strong>Summary of Findings</strong></p>
<p>So just to summarize some of the key findings:</p>
<ol type="1">
<li>OCD is familial, and genetics plays an important role.</li>
</ol>
<!-- -->
<ol start="2" type="1">
<li>OCD is highly polygenic with contributions from both common and rare variants.</li>
</ol>
<!-- -->
<ol start="3" type="1">
<li>OCD dimensions have both shared and unique genetic risks.</li>
</ol>
<!-- -->
<ol start="4" type="1">
<li>OCD shares genetic risks with some of its comorbid conditions, like anorexia and Tourette’s.</li>
</ol>
<!-- -->
<ol start="5" type="1">
<li>And genetic risk for OCD is shared between the sexes.</li>
</ol>
<p><strong>Implications</strong></p>
<p><u>Research</u></p>
<p>So, what are some of the implications of these findings? I think first and foremost, there has been a lot of discovery, but we definitely need larger samples for GWAS and next generation sequencing to get more replicable findings and to better understand the genetic risks underlying OCD. It does seem important to look not only OCD as a diagnosis, but also as a continuum. And we should be taking into account factors that we know are wrapped up in the genetics of OCD, like age of onset, sex, comorbidity, and symptom dimensions and that OCD comorbidity is driven at least in part by genetics.</p>
<p><u>Clinical</u></p>
<p>Now in terms of clinical implications, it’s a little bit “early days” for that. So, I know there’s hope that genetic information will be helpful as part of a diagnostic assessment at some point, although the extent of which that’s going to be helpful is still and clear. But it’s certainly much too early for that given the current results - much larger samples are going to be required if we’re going to get to that point.</p>
<p>And hopefully, emerging pharmacogenetics work, which I wasn’t able to talk about today, will help to identify some novel therapeutics or people who are more likely to have therapeutics that are successful for them. There’s some evidence from glutamate work that has prompted the investigation of glutamate-based therapeutics for OCD.</p>
<p><strong>Future Directions</strong></p>
<p>And I think there’s still this is a really exciting time to be in OCD [research] with growing samples and more momentum there’s going to be the opportunity to ask a lot of important questions that are still unanswered. And these are just a couple [of questions] where we can start:</p>
<ol type="1">
<li>Whether genetic risks are similar across ancestral groups because right now there isn’t a whole lot of diversity, and this is something that the PGC is actively trying to rectify.</li>
</ol>
<!-- -->
<ol start="2" type="1">
<li>How do genetic factors affect treatment response? - This is something that the PGC-OCD group is currently organizing itself.</li>
</ol>
<!-- -->
<ol start="3" type="1">
<li>Looking at which risk factors for OCD are causal. And hopefully, when we get to big enough samples, we’ll be able to use mendelian randomization to answer some of those questions.</li>
</ol>
<!-- -->
<ol start="4" type="1">
<li>How do genetic and environmental factors interact to affect risk?</li>
</ol>
<!-- -->
<ol start="5" type="1">
<li>What role do epigenetics play? And there has been research in this area, but I didn’t have time to talk about it today. But [we are] getting larger and larger samples [to] have enough power to really understand the role of that particular type of genetic effect.</li>
</ol>
<!-- -->
<ol start="6" type="1">
<li>Also, starting to probe into that interesting relationship between sex and age of onset and how much genetics may or may not play a role in those patterns.</li>
</ol>
<!-- -->
<ol start="7" type="1">
<li>Can we predict symptom presentation with genetics?</li>
</ol>
<!-- -->
<ol start="8" type="1">
<li>And finally how does shared genetics lead to comorbidity? What are the underlying common biological mechanisms there?</li>
</ol>
<p>So, I think we’ve learned a lot in the last 10 years in terms of OCD genetics and I think we’re going to learn a lot more in the next 10. There’s a lot of exciting discoveries to come so I just want to end by acknowledging the TS-OCD working group and the PGC more broadly for the opportunity, thank you.</p>
<hr>
</section>
<section id="sec-ptsd" class="level1">
<h1>Post-Traumatic Stress Disorder</h1>
<p><strong>Title</strong>: Posttraumatic Stress Disorder: From Gene Discovery to Disease Biology</p>
<p><strong>Presenter(s)</strong>: Frank Wendt</p>
<p>Hi everyone, my name is Frank Wendt. I am a postdoctoral fellow with Renato Polimanti at Yale School of Medicine. Today we’ll be talking about post-traumatic stress disorder and some of the historical studies that have led us from gene discovery to understanding some disease biology in relation to PTSD. I don’t have any conflict of interest to disclose. So PTSD is ultimately uh sort of the end product of a series of events including pre-trauma risk factors and traumatic events but in order for us to really understand what’s going on with PTSD, we have to have some understanding of what’s happening prior to uh diagnosis of PTSD including those traumatic events and the pre-trauma risk factors.</p>
<p>So, looking at PTSD in the DSM-5, I’ve listed here the criteria that are used for diagnosis, the description of each of those um diagnostic criteria and some example qualifiers for each of those criteria. We see here some of our sort of common themes that we think about with PTSD including re-experiencing, hyper-arousal, and avoidance symptoms. We also see some qualifiers in terms of the length of time. These disturbances are happening. This is required for at least one month and we also see some qualifiers here for um what the disturbances cannot be attributed to. So in many ways this qualifier is more of an exclusionary qualifier where we have to make sure that the disturbance is not due to an illicit substance use, some other medication use, or another medical disorder.</p>
<p>Looking at some of those traumatic events, we’re showing here the lifetime prevalence of those traumas. And those are listed across the x-axis and they are broken down into a general category of trauma. We see things like accidents and some of these other traumas of a loved one are fairly prevalent in the general population. Overall, across the lifetime um there’s approximately a 70% prevalence of any trauma and uh we know that a relatively small proportion of those individuals something like six or seven percent go on to actually develop PTSD. So even though most of the population does experience some traumatic event throughout their lifetime, relatively few individuals actually end up with a PTSD diagnosis.</p>
<p>We know that many of those traumatic events have some pre-trauma risk factors. Some of the major ones include gender and sex, age of trauma, education, socioeconomic status and other psychiatric comorbidities. We do know that probably um one of the most prominent of these pre-trauma risk factors is gender. We know that women are four times more likely to develop PTSD when compared to men, when we account for exposure to a traumatic event. And we also know that many of these traumatic events, um you know, produce the same PTSD risk in both men and women, including accidents, natural disasters, and sudden death of a loved one. However, things like a sexually assaultive trauma um have a much different uh PTSD um incidence between sex, where we see that uh even though women are more likely to experience sexual trauma, we see that men are more likely to develop PTSD following those sexual traumas.</p>
<p>So to summarize here, we have three time frames that we’re primarily interested with PTSD. We have a pre-trauma, a peri trauma, and a post-trauma time frame and we’ll talk about all of these sort of in concert as we move through the next couple of slides.</p>
<p>So we’ll be walking through quite a bit of material today and sort of touching on each of these topics individually. I’ve provided here sort of a one-liner for people who are less familiar with this type of study to sort of acclimate themselves and get prepared for what will be shown on each of the slides. If you’re familiar with all of these things, you know feel free to skip over this and move on to some of the actual meat. If any of these topics are unfamiliar to you, please feel free to reference the one liner provided here.</p>
<p>We’ll start with twin studies. On the left side of the screen here we see a really nice summary of some of the early twin studies, showing that as the female percentage in the twin study increases, the PTSD heritability estimate also increases. We’ll see this trend kind of following through into our uh GWAS studies of PTSD as well. On the right side of the screen, we see a twin study that attempted to sort of separate out the additive genetic component, the shared environmental component, and the non-shared environmental component between some of those traumatic events including in this case an assaultive trauma category and a non-assaultive trauma category. We also see that many of these twin studies have identified some comorbid conditions in relation to PTSD, in particular those sort of highlight some other psychiatric diagnoses. We see insomnia pops up quite readily, and some other sleep-related disturbance phenotypes that pop up quite readily.</p>
<p>In candidate gene studies, one of the first of which was looking at DRD2. That’s the dopamine receptor D2. Comparing the A1 and A2 alleles at DRD2, and there was a significant relationship between the hyper reactivity or hyperarousal PTSD subcluster and DRD2 locus. I’ve also listed some additional candidate genes on the bottom left side of the screen. You’ll notice some of the sort of favorite psychiatric disorder players here including COMT, which is the catechol-O-methyltransferase locus. And then we see on the right side of the screen, some of the history of those candidate gene studies and where those p-values lie in some of those additional studies.</p>
<p>So moving into genome-wide association studies, these are uh this slide in the next slide is pretty much just an introduction to GWAS before we get into what the PTSD data look like. Um genome-wide association studies are essentially a brute force experiment that is uh by nature hypothesis generating uh and essentially we are looking at the differences in allele frequency for single nucleotide polymorphisms (SNP) or single base changes in the DNA between two groups of individuals. In the case for PTSD, we might be looking at a group of individuals who are diagnosed with PTSD versus a group of individuals who are not diagnosed with PTSD.</p>
<p>Because we only have two outcomes, this is considered a logistic regression, and this regression model is typically covaried for a number of confounders in these disease association studies, including age, sex, and some principal components of ancestry, essentially the proportion of ancestry assigned to an individual. We also typically co-vary for a technical artifact. In this case, I’ve listed batch number here, and the example I’m showing here is a logistic regression with two outcomes, although we could also be looking at a continuous or quantitative outcome. And if we’re looking at PTSD, this might be something like the total symptom count from the PTSD checklist.</p>
<p>I’m showing here a number of different Manhattan plots which is the visual representation of GWAS results. We see a number of different patterns here but all of them have the same structure. So on the x-axis we have all of the human autosomes, those are all of the non-sex chromosomes in humans. On the y-axis, we see the minus log 10 P value for the association between each data point and our phenotype of interest. Each of those data points represents one of those single nucleotide polymorphisms that I discussed on the previous slide. What’s important to note here is that depending on how we define our phenotype, we typically will observe a slightly different pattern in our Manhattan plot. So on the left side of the screen, we’re looking at a quantitative versus a binary definition of PTSD. On the right side of the screen, we’re looking at stratifying our PTSD data by males and females in separate studies and we’ll see that each of those patterns of our Manhattan plots look slightly different, depending on how we’ve defined the phenotype.</p>
<p>On this slide we’re showing how that whole pattern of SNPs, or that whole pattern of genetic architecture overlaps with other phenotypes. On the left side of the screen, we see that PTSD shares genetic risk with anthropometric traits, some behavioral phenotypes, some cognitive phenotypes, and certainly some psychiatric disorder phenotypes. On the right side of the screen, we’re not necessarily looking any more at what PTSD is and is not associated with on a genetic perspective, but now we’re looking at how does PTSD compare to some other disorders. So PTSD for example is associated with a number of anthropometric phenotypes but when we look at the major depression category on the far right side of the screen, we see that major depression has some genetic correlates that are not observed when we look at the PTSD category and those typically uh align with the anthropometric traits, at least in this figure.</p>
<p>So one of the major uh pitfalls to the current status of genetic studies is that there is this overwhelming representation of European ancestry in our genetic studies and this means that most of our findings are generally only applicable to other individuals of European ancestry. And as you can imagine this uh puts a considerable hindrance on applying any of this information to a clinical practice. So to sort of mitigate that downfall or that pitfall of the field, the PTSD genetic research community has really made a push towards understanding how genetic risk for PTSD exists in individuals of admixed ancestry. And this is done by using um a method that essentially paints your chromosomes, and we can see that on the bottom left side of the screen where we have red and blue that indicates a different part of the chromosomes that are of different ancestries. And then we can perform our genome-wide association studies using this information in our model.</p>
<p>So next we’ll be talking about gene by environment interactions because PTSD relies so heavily on the presence of some indexed trauma, it’s extremely important to consider how different environments interact with risk for PTSD. However no large study to date actually looks at PTSD as the outcome in terms of gene by environment interaction. There are a number of different studies however that look at PTSD or related traumas and pre-trauma risk factors as environments. Um and one of these will be presented by me on Tuesday, so please feel free um to tune in for that between 2:15 and 3:45.</p>
<p>Next we’ll look at epigenetics and transcriptomics. This is really important because we can actually get relatively strong effect estimates using uh slightly smaller sample sizes. This is because these epigenetic and transcriptomic changes can be directly related to either causal mechanisms or downstream consequences of the phenotype of interest.</p>
<p>Looking at epigenetics, we see that PTSD is associated with a number of genes that share risk with some other psychiatric disorders, and that’s shown in the top right from Smith et al.&nbsp;We also see that some of those PTSD subdomains, in this case avoidance symptoms are positively correlated with accelerated DNA methylation age.</p>
<p>In terms of transcriptomics, we have uh two types of information shown here. On the left side of the screen, we’re showing uh peripheral tissue, in this case it’s blood. And on the right side of the screen, we’re showing some of these gene expression changes using brain tissue. And both of these studies and sort of this transcriptomics of PTSD in general converge on this concept of dysregulated immune inflammatory response in PTSD. And this is really important because it’s much easier to study a minimally invasive tissue like blood than it is to study brain tissue. So if we can find things that overlap between those two tissues, we can make some more translatable findings that are hopefully more meaningful and applicable to individuals with PTSD.</p>
<p>Finally looking at neuroimaging, we see two examples here on the left and right side of the screen, where we identify hippocampal markers of current PTSD and some of its genetic correlates. And this is a really nice study because it breaks down PTSD into civilian cohorts, military cohorts, and then by sex as well. And we know that some of these things are hypervariable between risk factor right, especially the male and female dichotomy and some of these pre-trauma risk factors. On the right side of the screen, we see an association between the putamen volume and PTSD and anxiety disorders. And this is really nice because we’re able to see brain regions that sort of confer risk for not only PTSD, but a number of other correlated phenotypes.</p>
<p>So to conclude, molecular studies of PTSD and its symptom clusters reveal an extremely complex architecture that overlaps with psychopathology and neurodevelopmental disorders, as well as some psychiatric phenotypes. But also, what we’re finding is that there’s a number of peripheral disorders and even non-disease phenotypes that overlap with PTSD. We see this quite readily with anthropometric phenotypes. We also know that sex, ancestry, and trauma type specific risks are abundant across all of these molecular investigations. And we know that this induces some level of heterogeneity on top of the diagnostic complexity that exists for PTSD already. Sample sizes are increasingly uh growing and this is happening quite rapidly. And this will make our studies uh much more robust and in particular allows us to start stratifying the epigenetic transcriptomic and neuroimaging studies to make inferences and conclusions about specific types of changes in those heterogeneous cohorts that make up our PTSD cases.</p>
<p>Finally given the stigma associated with some trauma types, it is imperative that both researchers and clinicians engage with both high-risk communities and general communities to begin reducing stigma and improve PTSD scientific literacy and advocacy.</p>
<p>And with that, I will provide thank yous um to Karestan and Caroline and Gita for their critical feedback, as well as the PGC and the MVP, and the WCPG organizing committee. Um and the contributors of this work are myself, as well as Renato Polimanti and thank you all for tuning in.</p>
<hr>
</section>
<section id="sec-schizophrenia" class="level1">
<h1>Schizophrenia</h1>
<p><strong>Title</strong>: Genetic Architecture of Schizophrenia</p>
<p><strong>Presenter(s)</strong>: Kaarina Kowalec, Niran Okewole, Sophie Legge, Marcos Santoro, Sathish Periyasamy</p>
<p><strong>Kaarina Kowalec</strong>:</p>
<p>Good morning, good afternoon, or good evening, depending on where you’re tuning in to from today!</p>
<p>What I’m going to be covering over today is the genetic architecture of schizophrenia. We’re going to cover over a review of some of the major advancements in this area. And so, I have the pleasure of introducing the topic and then we’ll be passing it off to four other early career researchers who have all of us have come together to perform this review, and so, I’m excited to be able to present this for all of you today. &nbsp;</p>
<p>For those of you that don’t know me, i’m Dr.&nbsp;Kaarina Kowalec. I’m an assistant professor in Canada, as well as affiliated with the Karolinska Institute in Sweden.</p>
<p>Okay, so a bit of an overview: it’s going to be a series of five-minute talks by early-career researchers, and so we present actually from a number of different countries around the world. So like I said, I’m going to do just the introduction today, and I’m from Canada, and then I’ll pass it off to Niran who’s from Nigeria. He’ll cover epidemiology and genetic epidemiology of schizophrenia, and then he’ll pass it off to Sathish who is from Australia, who will review some of the common and rare genetic variations of schizophrenia. Arsalan from Pakistan actually performed the original review for the rare genetic variation section, unfortunately he could not join us today.</p>
<p>From there we’ll cover off the functional annotation section by Marcos, who’s from Brazil, and then lastly the polygenic risk scores and genetic correlations with schizophrenia will be reviewed by Sophie from the UK. And lastly, I’ll provide a few future perspectives in conclusion.</p>
<p>So without further ado, we know that schizophrenia is a highly heritable condition. Our understanding of the genetic architecture of schizophrenia has increased greatly for the past decade, which is fantastic, and a lot of this is due to the fact that we have some major advances in genetics technology such as those that are more feasible now and more affordable.In order to perform this large scale gene typing sequencing we have a lot more research samples of course now with respect largely due to worldwide collaboration.</p>
<p>So for example, from groups such as the PGC, of course, and this has led to identification of a number of different common genetic risk variants as well as a number of rare variants now, and copy number variants as well. &nbsp;And together this has provided some really key insights into the biological basis of schizophrenia that was previously unknown.</p>
<p>And so, schizophrenia itself: this is a severe and often chronic psychiatric disorder. It has - it unfortunately causes substantial personal and societal burden primarily from severe and long-term disability. It’s associated or characterized by positive symptoms and so these are things that occur sort of in addition to like hallucinations and delusions, and as well as negative symptoms such as avolition and anhedonia which is sort of the lack of motivation/inability to feel pleasure for things, as well as disorganized symptoms with respect to someone’s speech and their behavior. It’s also marked by cognitive impairment. And so from there I’m going to pass off to Niran who is going to cover off more general epidemiology and genetic epidemiology.</p>
<p><strong>Niran Okewole</strong>:</p>
<p>Hello viewers around the world, a very good day to you wherever on the planet you’re watching!</p>
<p>I will be talking very briefly about schizophrenia epidemiology. &nbsp;Schizophrenia as we know is a disorder characterized by positive symptoms such as delusions and hallucinations, negative symptoms like avolition and their associated cognitive and also some effective component to it as well. The global age standardized point prevalence of schizophrenia is estimated to be about 0.28 percent. The incidence of schizophrenia is about 15.2 per one hundred thousand persons, with a median male/ female ratio of 1.4.</p>
<p>Schizophrenia contributes approximately 30 million years of life lived with disability. Poor outcomes are thought to be common, and known to be common, and include premature mortality, long-term hospitalization, treatment resistance, and poor quality of life. Approximately one in three people with schizophrenia would attempt suicide during their lifetime, and it’s also been found that there’s a two to threefold increased risk of death in patients with schizophrenia, with a median standardized mortality ratio for all-cause mortality being about 2.6.</p>
<p>In terms of the factors that we associate with schizophrenia: it’s been shown to be an interplay of genetics and environmental factors, which contribute to psychopathology.In terms of environmental factors, it’s possible to categorize these as early development proximal factors and onset factors.</p>
<p>Factors in the early development would include obstetric complications and advanced paternal age. Proximal factors would include social adversity, migration, urban living, and then factors that are known to be present at onset that might trigger episodes would include substance use and trauma.</p>
<p>In terms of the genetics, we can think about the concept within the context of heritability: The phenotypic variance, that’s the variance associated with the phenotype of schizophrenia. It’s possible to talk about the variance that is due to environmental factors, and then variance that is due to genetic factors. The latter can be further broken down into variance due to dominance factors, interactive factors, and then additive factors.</p>
<p>So, talking about broad sense heritability, that would be the variance due to genetic factors over the total phenotypic variance, whereas the more parsimonious term of narrow sense heritability, or h squared, is the variance due to additive factors divided by the total phenotypic variance.</p>
<p>In terms of schizophrenia heritability – heritability (being the variance that is due to genetic factors) estimates based on family studies doing studies have ranged from 41 to 87 percent, but the current estimates, the current consensus, is that it’s approximately 80 percent.</p>
<p>We can talk about this SNP heritability which is a component, a fraction, of the neuroscience heritability, and SNP heritability is a phenotypic variance due to genetic factors as tagged by polymorphisms derived from original typing. It has a tendency to be lower than the total narrow sense heritability that is associated with additive factors.</p>
<p>Most recent schizophrenia GWAS figures, the most recent figures that we have give a sleep-based irritability of approximately 24 percent, which means that the substantial portion of the variance remains unexplained and continues to be studied. Some of the component variants that are associated will be discussed by my colleagues in other presentations.</p>
<p>Thanks for listening.</p>
<p><strong>Sathish Periyasamy</strong>:</p>
<p>Good day everyone! In this part of the talk, I’m going to talk to you about the genetic architecture of schizophrenia, especially focusing on common variants.</p>
<p>This slide gives you a general overview of genome-wide associated studies of schizophrenia. If you look at the figure on the left it shows the genomic associations reported as of July 2019 in GWAS catalog. &nbsp;It represents around 17 trait categories, however, if you look specifically with respect to schizophrenia the GWAS catalog has listed around 2112 genetic associations from 36 schizophrenia and related publications. This includes schizophrenia and schizoaffective disorders, and schizophrenia severity measurements. If you look specifically for schizophrenia there are about 2057 associations published in 23 publications, however, many of which are replications across studies. Currently, there are over 300 unique genome-wide significant variants listed in the g-was catalog.</p>
<p>The initial schizophrenia GWAS genomic association studies were reported in 2007 through 2010. &nbsp;Many did not identify any genome-wide significant variants, this was mainly due to insufficient sample sizes for the relatively small effect sizes that are typical of common variants. The study sample sizes were in the order of one thousand to two thousand, with only five hundred two thousand cases. They were mostly conducted by independent groups. Some of the reported publications are mentioned here.</p>
<p>This figure shows the genetic architecture of Schizophrenia: the x-axis represents the risk scaling frequency it controls, and the y-axis represents the effect sizes in log scale. As you can see, most of the GWAS discovered SNPs have relatively small effect sizes, and they have relatively higher frequencies compared to the rare variants. This is also consistent with the common variant hypotheses. Only a few less-common variants have been discovered.</p>
<p>I’ll be talking to you about the rare variance in my next talk. This video gives you a history of schizophrenia GWAS in key publications. The x-axis represents the pubmed ids and the year of publication. &nbsp;The y-axis represents sample size. AS you can see, with the increase of sample sizes, in this decade we were able to identify many more risk variants.</p>
<p>The first series of publications with genome by significant hits were identified in/reported in 2009. The first PGC 1 schizophrenia study was published in 2011, which reported seven loci. This was followed by a series of small studies, and the second most important paper - the landmark paper by the PGC, was published in 2014, which identified 108 loci.</p>
<p>This was followed by a series of meta-analysis projects with the Chinese as well as across the UK, which here identified even more regions. And more recently, the PGC East Asian group published an article which identified 176 loci which I’ll be discussing in detail later on. More recently the PGC 3, this which is under review, has reported around 270 loci.</p>
<p>As I mentioned in the previous slide, the first reported genome-wide significant loci associations were reported in the year 2009. They were reported by three independent publications: the first GWAS by PGC identified ten independent associations in eight distinct loci in 2011.&nbsp; They were mostly of European ancestry and the most robust common variant association was in the chromosome 6p22.1 locus. &nbsp;The second GWAS by the PGC, if I remember correctly, identified 128 independent associations in 108 distinct loci, this was published in 2014 and they were predominantly of European ancestry. And again, the most robust common variant association was in the chromosome 6p22.1 locus.</p>
<p>The third GWAS by PGC identified 329 independent associations in 270 distinct loci (this manuscript is under review), and this consists of around 80 percent European and twenty percent of East Asian ancestry, and again the most robust common variant association was in the chromosome 6p22.1 locus.</p>
<p>In addition to that, if you look at the East Asian study, the first GWAS by the PGC East Asian schizophrenia group identified 21 independent associations in 19 distinct loci. The top three associations were shared with the European studies, however, the most robust associations in Europeans (the six p22.1 locus) was not observed in East Asian. The subsequent meta-analysis of East Asian and European ancestors reported 208 independent associations in 176 distinct loci. 53 of these loci were novel.</p>
<p>In addition to the large European and East Asian ancestor studies, there have been studies in other populations which includes the African American, Latin American, and Indian, and we expect over the next few years that these studies will increase in sample sizes and discover many more risk variants.</p>
<p>In this part of the talk, I’m going to talk to you about the genetic architecture of schizophrenia, especially focusing on rare variants. To give you a brief overview of rare variants: the key characteristics are they have a minority frequency of less than one percent, and they have a relatively large effect size compared to common variants.</p>
<p>The type of variance we’re looking at: Single nucleotide variants (which are basically alterations of one or a small number of base pairs), and insertion deletions (which can range up to one to a few base pairs), and copy number variations are basically duplications or deletions of thousands to a million or millions of base pairs. The genotyping technologies we use for these studies are microarray (usually used for CNV analysis) and whole exome sequencing, and whole genome sequencing (which are basically used for most of the analyses). They are highly informative variants due to their large effect sizes.</p>
<p>As you can see in this figure, most rare variants have very large effect sizes and relatively low/ very low frequencies, and are consistent with common disease rare variant hypotheses.</p>
<p>Now if you look at SNVs and indels in particular, the early whole exome sequencing studies demonstrated enrichment for rare variants, and one study published by Purcell and the subsequent extension of that study using the whole exome aggregation consortium data set identified an excessive of burden of deceptive and damaging alternative variants in cases. However, no individual gene was found to have a significant excess of damaging variants.</p>
<p>And now, if you look at ultra rare SMVs and indel studies which had significant genes, there was one study published in 2006 which identified a loss of function variant in SETD1A gene. The more recent and the largest whole exome sequencing effort by the SCHEMA consortium (the manuscript is under review) used over 24,000 cases and 97,000 controls and they identified 10 genes, including the 71a gene, which reached genome-wide significance threshold. A further 22 genes reached a suggestive level of significance as defined by the false discovery rate of 0.05.</p>
<p>And if we look at this figure once again, the genes identified by the schema consortium are shown in the top left part of the figure. As you can see, most of the genes have very large exome sizes and very low frequencies.</p>
<p>And now, if you look at DNV variants, they’re usually identified in trio based studies and are usually present in offspring as a result of new mutation events, but are absent in parents. One recent whole exome sequence study, which used over three thousand schizophrenia parents and proband trios, identified loss of function DNVs to be enriched in loss-of-function intolerant genes, however no gene actually achieved exome-wide significance. One of the interesting things was that the burden was higher in genes previously associated with neurodevelopmental disorder. Gene set analysis revealed that DNVs were enriched in evolutionary constrained genes, and genes implicated in multiple neurodevelopmental disorders.</p>
<p>And now if you come to the last slide and look at copy number variants: CNVs are duplications or deletions ranging from 50 base pairs to megabases in the genome, and can span a whole gene or set of genes in a region. CNVs have been consistently implicated in the etiology of schizophrenia. The first risk locus was conferred in chromosome 22q11.2 which had a 20-fold increased risk with 25 percent of the carriers developing schizophrenia.</p>
<p>The largest CNV study to date (comprised of over 21,000 cases and over 20,000 controls) was published by the CNV and schizophrenia working group of the PGC in 2017. They identified eight CNVs; six deletions and two duplications passing the genome-wide significance threshold. However, the CNV penetrance in schizophrenia is relatively lower compared to other neurodevelopmental disorders.</p>
<p><strong>Marcos Leite Santoro</strong>:</p>
<p>Hi guys, in this video I’ll talk briefly about the possibilities of post GWAS analysis in which researchers try to interpret the findings from common and rare variant results.</p>
<p>So, GWAS confirmed that schizophrenia is a highly polygenic disorder, and this has been pretty challenging to understand where and when these many genetic variations and variants play a role in the disorder.</p>
<p>The post GWAS approach aims to gain biological insights by identifying genes, tissues, cells, and biological pathways associated with schizophrenia. This video covers three topics that have been successful in identifying new biological insights for schizophrenia.</p>
<p>So, first, the gene set analysis tests whether sets of genes grouped by a biological pathway are enriched for variants associated with schizophrenia. These analyses have been pretty important to confirm, for example, that schizophrenia is a disorder of neural dysfunction. So, genes highly expressed in the brain (mainly in the cortical inhibitory interneurons and excitatory neurons) are strongly enriched for variants associated with schizophrenia.</p>
<p>Besides, the gene set findings confirmed associations within genes of dopaminergic and glutamatergic pathways like the dopamine receptor 2 d2 which is a target for encyclic drugs. Notably, as the sample sizes increase in genetic studies, enrichment analysis have shown a convergence of rare and common variant findings, pointing to concordant genes and pathways which strongly support their relevance to schizophrenia.</p>
<p>TWAS uses multiple known eQTL variants from a gene in a specific tissue to predict its expression. And, when merged with GWAS summary statistic data, it is possible to infer which genes are up or down regulated in schizophrenia.</p>
<p>A TWAS in 2018 investigated eQTL data from brain blood and adipose tissues and found that 42 out of the 157 genes associated with schizophrenia were in fact involved with chromatin organization. So, highlighting that regions responsible for gene expression regulation can be potential targets.</p>
<p>A more recent TWAS used expression data from dorsolateral prefrontal cortex and identified 89 genes associated with schizophrenia, and also that these genes were enriched in gene-set for abnormal CNS synaptic transmission, and also presentation of peptide antigen via MHC class 1, which is consistent with well-established association between schizophrenia and common variation at the MHC class 1 region.</p>
<p>Finally, although confirming GWAS results and identifying novel hits, TWAS may not extract time-specific information of disease, nor which kind of cell type is involved inside that tissue.</p>
<p>So RNAseq and single cell studies have been pretty important to fill this gap. As an example, transcriptomic data from dorsolateral prefrontal cortex identify an overlap of 20 variants between this data and genome-wide association hits interfering in the gene expression of one or more genes.</p>
<p>Besides, a recent study from PsychENCODE using single cell RNAseq data mapped the associated genomic losses from GWAS to pyramidal cells, cortical interneurons, and other cell types with schizophrenia.</p>
<p><strong>Sophie Legge:</strong></p>
<p>Hi, my name is Dr.&nbsp;Sophie Legge, I’m from Cardiff university and I’m going to talk to you about polygenic risk scores and genetic correlations in relation to schizophrenia.</p>
<p>So, polygenic risk scores have emerged as an informative tool for studying the effects of schizophrenia genetic liability. &nbsp;It’s a single measure of the cumulative effects of SNPs associated with the disorder, with higher genetic scores indicating higher genetic liability. So polygenic scores can currently explain around 7.7 percent of the variance in schizophrenia case control status. This graph here is taken from the PGC 3 preprint. So, this level is currently insufficient for diagnostic purposes, and how schizophrenia polygenic risk or could be applied clinically? It remains unclear, although there have been some promising findings in regards to first episode psychosis samples.</p>
<p>A key challenge in polygenic risk score analysis is the application across major ancestral populations. So, polygenic risk derived from alleles discovered in GWAS of Europeans explains less variance when it applied particularly to African, but also to Asian populations. This is likely due to differences in allele frequencies and linkage disequilibrium structures.</p>
<p>In the recent GWAS by Lam and colleagues, &nbsp;polygenic risk score prediction of schizophrenia was only 45 percent as accurate in East Asians compared to European individuals, despite a broadly shared genetic etiology. And so, greater diversity in GWAS must be prioritized to realize the full potential of polygenic risk scores.</p>
<p>One application of polygenic risk scores has been to disentangle the clinical heterogeneity in schizophrenia by investigating phenotypic markers of genetic liability, and this has been done particularly in relation to treatment outcomes, symptoms severity, and cognitive ability.</p>
<p>So, a higher polygenic risk of schizophrenia has been associated with a more chronic illness course, and this is indexed by the number and the length of hospital admissions. However, results from studies investigating treatment resistance to antipsychotics have been conflicting, and this indicates that specific polygenic risk scores are likely to be required for some outcomes.</p>
<p>Schizophrenia polygenic risk score has been associated with negative and disorganized symptoms, specifically in individuals with schizophrenia, although the reported variance explained by the polygenic risk score is small.</p>
<p>There seems to be no evidence to suggest that the polygenic risk score is associated with the severity of positive symptoms in individuals with schizophrenia, but it has been associated with psychotic symptoms in first episode samples and bipolar disorder.</p>
<p>There’s been inconsistent findings from studies investigating the association with cognitive deficits in schizophrenia, and this may be due to differing measures of cognitive ability and at what stage the illness the assessment was made. However, several recent studies have indicated that the schizophrenia polygenic risk score may be associated specifically with cognitive decline.</p>
<p>A further area of interest has been the relationship with intermediate phenotypes such as neuroimaging measures. There doesn’t currently appear to be any strong evidence for an association with brain structural changes that are relevant to schizophrenia, but a higher polygenic risk score has recently been associated with lower connectivity in control samples.</p>
<p>So, common genetic liability for schizophrenia has been robustly found to have pleiotropic effects on related psychiatric disorders, and this figure details the significant genetic correlations from the Brainstorm Consortium paper. So, the highest correlation by some way is bipolar disorder for schizophrenia and this has correlation of 0.7, this is followed by depression, OCD, ADHD, anorexia, and autism. Schizophrenia also has a positive genetic correlation with neuroticism, and negative correlations with subjective well-being, BMI, and intelligence. These results are also consistent in non-European populations, and in a polygenic risk score analysis of US-based health care records.</p>
<p>These findings indicate that a substantial portion of the common genetic architecture among psychiatric disorders is shared. Well thanks for listening, that’s a wrap from me! Bye.</p>
<p><strong>Kaarina Kowalec</strong>:</p>
<p>Okay, so, thank you very much Sophie for finishing off the polygenic risk scores and genetic correlations, and now I’m just going to give a few future perspectives to tie things off today.</p>
<p>So, we now know that some of the, in the future, we’re hoping that larger sample size GWAS and rare variant studies will help us to identify more genes and refine some of the biological processes that we know or have been implicated in schizophrenia.</p>
<p>Updates to some of the functional annotation consortiums like PsychENCODE, and GTEx, &nbsp;and CommonMind will help to further interpret some of the current GWAS as well as future GWAS findings, and hope that these together would contribute to drug discovery for schizophrenia and psychosis, as well as providing some personalized medicine efforts. And, we also hope that some of the future research will cover off how genetic risk interacts with environmental factors and ultimately cause and develop schizophrenia.</p>
<p>We also know that it’s of paramount importance in the future to improve diversity of schizophrenia genetic studies, and doing so by including other non-European populations. So for example, the latest published GWAS are probably running around 20 percent of samples were non-European (mostly of East Asian), but of course we have still a long ways to go. And by increasing genetic diversity we’ll hope to ensure that findings are applied to all human populations, and this will increase the power for identifying new genetic associations as well as improving fine mapping.</p>
<p>And in conclusion, we know that schizophrenia has provided some of the evidence of the value of GWAS; performing GWAS and investigating common genetic risk for psych disorders. Further functional studies are needed to now better understand how and when schizophrenia vulnerability acts during the different stages of neurodevelopment.</p>
<p>In the future we hope that an increase in sample size, and increase in population diversity, identify new regions in some of the increasing fine mapping, and hopefully in hopes that will decrease the disparities in PRS predictions and hopefully eventually possibly even clinical application.</p>
<p>Thank you so much for your attention today, and please feel free to reach out to any of us if you have any further questions about what we presented here today. Thanks very much!</p>
<hr>
</section>
<section id="sec-sharedgenetic" class="level1">
<h1>Shared Genetic Architecture</h1>
<p><strong>Title</strong>: Shared Genetic Architecture across Psychiatric Disorders</p>
<p><strong>Presenter(s)</strong>: Andrew Grotzinger</p>
<p>Hi everyone, my name is Andrew Grotzinger and we’ll be talking today about the shared genetic architecture across psychiatric disorders. I’m going to break up my talk into five sections starting with the comorbidity problem more generally, and how family studies have been used to understand comorbidity, and then move on to talk about how genomic methods have been used more recently to understand comorbidity: starting with the genome-wide level analysis, followed by the genetic variant level, and ending with mechanistic studies. Then, we’ll wrap up by talking about next steps in cross disorder efforts.</p>
<p>So, beginning with the comorbidity problem and family studies: I’m starting here by showing all individuals with mental disorders in this blue space. Among that lump of people about two-thirds will meet criteria for a second disorder in their lifetime, 51 percent will meet criteria for a third disorder, and 41 percent will meet criteria for a fourth disorder, indicating some pretty large shared pathways across these disorders.</p>
<p>Proband studies or family studies that are often depicted using this schematic over here with the parents on the top and the offspring on the bottom, have identified reciprocal relationships across a number of disorders. And by reciprocal relationships, I mean that the offspring is at risk for not just the disorder present and the parent, but also this alternative disorder. These relationships have been identified for bipolar disorder and schizophrenia, bipolar disorder and major depression, autism, and bipolar and schizophrenia, and a general shared liability across major anxiety disorders. Indicating that these disorders in general do not breed true.</p>
<p>In this 2012 study they looked at these five disorders in the parents on the x-axis and the odds ratio of the child developing a particular disorder on the y-axis, with the null at one in this red dash line. What they find consistent with those proband studies is that the children were at risk for really any disorder, and not just the disorder present in the parent.</p>
<p>Twin studies have been used to follow up on these findings by looking at specific genetic correlations across disorders, which is shown here on the bottom half of this matrix from this smaller review paper in 2019.</p>
<p>One thing I want to draw your attention to is that a lot of these cells are listed as N/A, and that’s because for a twin study to estimate the genetic correlations across these disorders both disorders would have to be present in one of the twins. Because a lot of these disorders are really pretty rare, or even mutually exclusive, this is often not possible. Whereas, for genomic methods (in blue here above the diagonal) you’ll see that genetic correlations are estimated across the psychiatric space and that’s possible because using genome-wide methods such as bivariate LD score regression, introduced in this nature genetics paper in 2015, we’re able to estimate genetic correlations across samples of varying degrees of sample overlap, including mutually exclusive samples such that the samples can themselves be independent. Now you can look at the genetic correlations for these more rare disorders that are unlikely to be measured in the same sample.</p>
<p>Given high rates of genetic correlations across the disorders this motivated us to develop genomic structural equation modeling, which we introduced in this nature human behavior paper, which is a framework for modeling the patterns of genetic associations across different constellations of traits.</p>
<p>I want to go now through the different iterations of factor modeling we’ve done using genomic sound for psychiatric disorders, starting with this initial factor model we fit in that nature human behavior paper for these five major disorders, where we identified this overarching p factor.</p>
<p>We called it p in part just because it was a placeholder given that we only had these five disorders that were sufficiently powered to include in the model at the time, which is something we wanted to improve upon in future iterations, which we were able to do in this 2019 cell paper for the second major cross disorder effort where we examined the multivariate architecture across these eight disorders to identify a compulsive psychotic internal developmental disorders factor.</p>
<p>We’ve updated that model still further to include these additional three disorders shown in red for this med archive preprint in 2020. With the inclusion of these three disorders, we’re now pulling out this fourth internalizing disorders factor, that maps on pretty nicely to a lot of the phenotypic factor modeling work that’s been done for psychiatric disorders. I would highlight too that for a lot of these disorders we have updated sample sizes, but are still pulling out those same compulsive psychotic and neurodevelopmental factors from the cell paper.</p>
<p>There’s some caveats to keep in mind when interpreting the genetic correlations across the corresponding factor models. One being the genetic correlations are only going to be equivalent across genomic and family based methods when the genetic correlations are constant across rare and common variants, and that’s because LD score regression is estimated using relatively common alleles with a minor allele frequency greater than one percent. So if that clustering really changes at the rare variant end of the spectrum, which is certainly possible, then that’s something that’s going to shift those factor models as well, and it’s something that remains to be examined in future work.</p>
<p>It’s also important to keep in mind that genetic correlations can be upwardly biased in two main cases. The first being when GWAS studies utilize supernormal controls, which refers to instances when the controls are not simply screened for the disorder of interest but also for related disorders, which is going to induce a dependency between that screen disorder when estimating genetic correlations.</p>
<p>They’ll also be upwardly biased when misclassification is present, so when a related disorder is accidentally classified as the disorder of interest, which will then also induce an upward bias correlation, although simulations indicate that extremely high rates of misclassification would be needed to explain the current pattern of findings.</p>
<p>So in general, I think given both of these two points we should treat the genetic correlations as a sort of upper bound of what’s likely present in the population, but it’s something that’s really an interpretive limitation, not something that should cause us to completely throw out the results.</p>
<p>I want to talk now about genetic variant level of analyses with respect to GWAS and cross-disorder efforts starting with the first cross-disorder paper in 2013, where they examined five major disorders across over 33,000 cases and 27,000 controls to identify four hits. Which is expanded rapidly in the 2019 cell paper to now include eight disorders and over 230 thousand cases, and almost half a million controls to now identify over 109 polyotropic loci.</p>
<p>Using a fixed effects meta-analysis in genomicSEM we also examined&nbsp; pleiotropic loci across the 11 disorders and identify 184 pleiotropic loci, including 69 of the 109 CDG2 loci which points to the replicability of those findings.</p>
<p>In this Manhattan plot I’m showing hits both with black and red triangles. where black triangles reflect hits that were in LD with the univariate hits, and red triangles indicate novel loci relative to the univariate GWAS. Which highlights the ability of cross disorder efforts not just to unpack the genetic variants underlying comorbidity, but also to leverage shared power across the traits to identify new hits and novel discovery.</p>
<p>We also looked at multivariate GWAS in the context of SNPs acting on those factors: where on the top half of these Miami plots we have the SNP effects on the factor, and on the bottom half what we call a genomic sum qSNP, which indexes heterogeneity across the indicators that load onto that factor. So, the qSNP really identifies disorder-specific effects so for these results we find 132 hits in LD with individual hits, 20 novel loci and nine highly disorder specific hits, which again points to another benefit of crosses over efforts which is to identify not just pleotropic loci, but loci that also underlie phenotypic divergence or cause disorders to seem really dissimilar as they present.</p>
<p>I want to briefly touch on the different types of pleiotropy that we should consider when thinking about what these cross-disorder hits are picking up on. The first being horizontal or true pleiotropy, in which case a single genetic variant or gene affects two disorders. &nbsp;The second being vertical or mediated pleiotropy ,in which a gene affects a disorder, and then that disorder then goes on to affect a second disorder, so it’s part of this sort of causal cascade. &nbsp;</p>
<p>While the general rates of horizontal and vertical pleiotropy remain to be tested within psychiatric disorders, in general we find for human complex traits that there tends to be a mix of both, which likely also holds for the psychiatric space.</p>
<p>One way to examine vertical pleiotropy is using Mendelian randomization which is something we also did with that 11 disorder paper where we used eight instruments, eight SNPs as instruments for alcohol use disorder, to examine this causal cascade of vertical pleiotropy, and indeed find causal effects of alcohol use disorder on bipolar and major depression in this model using those eight SNPs as instruments.</p>
<p>A third type of pleiotropy is spurious pleiotropy, which will often occur when diagnostic misclassification is present, which is highly relevant for psychiatric disorders given rates such as 15 percent &nbsp;misclassification across bipolar disorder and schizophrenia. But, I want to highlight that it can be kind of unclear what misclassification means given really high rates of genetic overlap. &nbsp;So, if we take this theoretical distribution of bipolar disorder genetic risk in red, and schizophrenia risk in blue, it’s unclear what the correct diagnosis would be for someone here in the middle in purple. I think that’s something that we can begin to move past in terms of how we think about this problem using symptom level data which I’ll touch on at the end.</p>
<p>But before that I want to talk about mechanistic studies, or what’s often referred to as functional studies, which in general look to take these hundreds of genetic variants that we’re starting to unpack and start to understand the biological picture that’s being painted by lumping these genetic variants into different functional categories, such as when or where the genes are expressed.</p>
<p>In that cross-disorder paper they have a number of exciting findings including the result from GTEx that pleiotropic loci are generally enriched in the brain, shown here in blue. &nbsp;Using spatial temporal gene expression patterns, they find that pleiotropic loci (in this dark blue line) are generally enriched during that second prenatal trimester here.</p>
<p>Using cell type specific analyses, they find that pleiotropic loci are generally enriched in neurons, but are not as enriched in microglia as indicated by this blue shading.</p>
<p>In that 11 disorders paper we use this new method of stratified genomicSEM that can be used to examine multivariate enrichment to examine enrichment at the level of those psychiatric factors that I talked about. We find that for the psychotic disorders factor, that these that this factor is enriched within excitatory genes. GABAergic genes. protein truncating variant intolerant genes here in the middle. and in particular is enriched at their intersection, which really gives some understanding of the shared processes for the psychotic disorder factor indicators (namely bipolar disorder and schizophrenia). And, would also highlight that in line with the cross disorder two findings we do not find enrichment in these glial cell categories across any of the factors.</p>
<p>So I end now by talking about major next steps for cross disorder efforts. One of the main ones being the consideration of more nuanced phenotypes, which can help how to classify some of those more mixed presentations that I talked about in the case of bipolar disorder and schizophrenia, and polygenic risk score analyses really point towards why this might be important. So for example schizophrenia polygenic scores have shown a stronger association for bipolar disorder characterized by mood-incongruent psychosis as compared to bipolar disorder without psychosis, and with earlier age of onset bipolar, which indicates that certain presentations may be may be more tightly linked at the genetic level across disorders.</p>
<p>There are different methods like BUHMBOX that can be used in conjunction with raw genotype data to detect if certain subgroups within a disorder are driving genetic correlations with other disorders, and while initial findings did not find evidence for these subgroups for major depressive disorder, this largely remains to be tested for other disorders.</p>
<p>Electronic health records are going to be an amazing resource moving forward that is going to give us access to some of that symptom level data that with a number of different consortium already underway with that explicit purpose including eMERGE, PsycheMERGEm and the NIH All of Us program.</p>
<p>Another major consideration for future cross-disorder genomic efforts is inclusion of the environment, namely because the pattern of overlap across disorders may shift across certain contexts, and by splitting GWAS across different environments or different cohorts we can examine the stability of these genetic correlations and whether or not certain correlations are really environmentally specific.</p>
<p>One particular environment that I think of as relevant is the age of onset for that disorder with respect to the internal biological environment that might be going on at that time. &nbsp;So if we take these distributions of age of onset for schizophrenia major depression on the top you can see these two kind of peaks later in life that might really indicate that there’s a really specific sort of biology that’s getting activated for genes expressed later in life, and if we split the GWAS for these two disorders within this particular late 40s age bin we might find that there’s a really unique clustering across disorders within that age group.</p>
<p>So in general, although our disorders may seem dissimilar in some respects, which has caused us to lump them into these discrete categories, their co-occurrence at levels much higher than chance really requires us to understand the underlying causes of this overlap.</p>
<p>I would argue that genomics offers a real opportunity to fill in some major gaps in our understanding of this overlap, in part because we can examine the full psychiatric space by using different methods that allow us to examine overlap for oftentimes independent samples, and it also allows us to examine this convergent at multiple levels of analysis, including at the genome-wide mechanistic and SNP level analysis. And so with that I’ll just thank everyone for their time.</p>
<hr>
</section>
<section id="sec-sud" class="level1">
<h1>Substance Use Disorder</h1>
<p><strong>Title</strong>: The Genetics of Substance Use Disorders: A Review</p>
<p><strong>Presenter(s)</strong>: Joseph Deak, Emma Johnson</p>
<p><strong>Joseph Deak:</strong></p>
<p>Alright, so we’ll go ahead and get started. Today Emma and I will be reviewing some recent progress in the genetics of substance use disorders and much of what we’ll talk about today is based on a recent review paper that we wrote.</p>
<p>Alright so today’s talk will largely cover what substance use disorders are and how we define them, the epidemiology and genetic epidemiology of substance use disorders, recent progress in substance use disorders genome-wide association studies, and conclusions and next steps.</p>
<p>So what are substance use disorders? Substance use disorders are psychiatric disorders that frequently co-occur with many other mental health conditions. These can be assessed when we’re seeing a variety of substance classes including substances that are often described as lciit substances, such as alcohol and nicotine, as well as illicit substances, such as cannabis, opiates, and cocaine. Prominent substance use can result in a variety of negative consequences, including death as a result of consuming the substance itself or what we call overdose, as well as indirect consequences, such as being involved in a traffic-related accident or experiencing health-related issues and disease as a result of long-term use. As we’ll talk more about here shortly, some of these disorders are influenced by both genetic and environmental factors.</p>
<p>So, on the previous slide, we mentioned some adverse health issues related to substance use. Here’s a quick example of what this might look like for one substance category, here being alcohol use. So these are figures from the Global Status Report on Alcohol and Health, released in 2018, that reported an estimated 5.3 percent of all deaths worldwide that are attributable in some fashion to alcohol use. The pie chart below breaks down these two million deaths into different causes, where we can see that nearly 21% of the deaths are the result of the alcohol-related injury and approximately 40% due to digestive disease, cardiovascular disease, and diabetes. And so, we can see that alcohol use and misuse is having a large impact on the population worldwide and is doing so in a variety of ways.</p>
<p>And so moving on from the negative consequences to how we define substance use disorders. So substance use disorders are defined by DSM-V as the presence of at least 2 of 11 criteria within a 12-month period with disorder severity indexed by the number of criteria endorsed: 2 to 3 for mild, 4 to 5 moderate, and 6 or more for severe. Broadly speaking substance disorder criteria correspond to the presence of different substance-related problems and can be assessed relationships with various substance classes, including alcohol, nicotine, cannabis, opiates, and cocaine.</p>
<p>And so as I mentioned a couple slides back, substance use disorders can be influenced by both genetic and environmental factors, and these influences are dynamic in that they interact either increase or decrease one’s risk for developing the use disorder. And these factors can be influential across all stages of substances to develop. For example, environmental factors, such as peer influences, familial environment, socioeconomic status, stressful life events, and co-occurring health conditions, can also raise environmental risk factors, while positive peer networks, supportive family environments, and greater socio-economic advantage can all be protective against development of disorders. Similarly genetic influences can also contribute to elevated risk as well as serve as a protective role. It’s also important to note that substance use disorders are highly polygenic in nature, meaning that no one gene is necessarily or sufficient to result in development by itself.</p>
<p>And so how can we study the genetics of substance use disorders? Early efforts focused on twin and family studies that demonstrated terrible heritability estimates that say genetic influences account for approximately 50% of the risk for developing a substance use disorder and that, in addition to the substance-specific influences, there are also heritable factors that contribute to substance use disorders more broadly. In molecular genetic studies, we are studying specific points of variation, or what are called polymorphisms. Advancements in genomic association studies have allowed us to reliably go across the genome to look at measurements of these specific points of variation and see if these are related to a trait of interest, in our situation being substance use disorders. Now this story has emerged from what was found from early twin and family studies where there appeared to be certain substance-specific variants that emerge for polygenic studies of substance use disorders as well as pre-attribute variants that influence susceptibility for many psychiatric traits.</p>
<p>Alright so now we’ll talk a bit about some progress that has been made for substance us disorders. I’ll be talking about alcohol and nicotine, and then Emma will walk you through some other additional studies.</p>
<p>So alcohol dependence and alcohol use disorder is a great example to demonstrate the progress that we have made in the past few years. So this figure is from one of our first well-powered studies on alcohol dependence. And then to better orient you to it, this is what is called the Manhattan plot, and we’ll see a series of these on the next several slides. So here on the x-axis we have chromosome position across th22 autosomal chromosomes and on the y-axis we have negative log 10 of the p-value. Then these plotted data points correspond to individual variant associations with the trait that we’re interested in. So basically what I’m thinking here is that smaller p-values are going to correspond to these higher points in the plot. Generally looking from 5 x 10<sup>-8</sup> for a variant to be deemed genome-wide significant, but really what we’re looking for are these towers to emerge, which alcohol dependence does a really nice job of demonstrating for us here on chromosome 4. Before, so as I mentioned, this was one of the first well-powered substitute use disorder GWAS for alcohol dependence, and here we see that this lone tower emerges on the <em>ADH1B</em> region of chromosome 4 with a p-value of 9.8 x 10<sup>-13</sup>. So this study had about 11,500 individuals of European ancestry that were diagnosed with alcoholic dependence and so that was the biggest thing.</p>
<p>Let’s take a look at the next slide, so this is currently the largest GWAS study of alcohol use, which was a meta-analysis of problematic alcohol use in approximately 435,000 individuals. And so one thing I point out here is that on the previous slide we’re seeing the most significant variant with the p-value of 9.8 x 10<sup>-13</sup> and that was the only tower that was emerging. But now with more samples, we’re seeing many more peaks, including signals that are now called metabolizing genes on chromosome 4 that have increased in magnitude substantially and we also see starting to see additional associations in emerging genes, such as what we see over here on chromosome 2 in the glucokinase regulator gene which includes the regulatory protein involved with glucose metabolism.</p>
<p>Alright so transitioning to nicotine and tobacco use, here are initial findings from the largest GWAS in nicotine dependence to date, here being assessed by FTND scores, in a study including 58,000 smokers. So for nicotine dependence and in studies of other smoking phenotypes, such as cigarettes per day or regular smoking, there’s a robust association with genetic variants within a cluster of nicotine acetylcholine receptor genes on chromosome 15 that includes the <em>CHRNA5</em> gene, which is also more specifically replicated in the variant rs16969968, which here had a p-value of 1.6 x 10<sup>-39</sup>. So these are the most pronounced results from the largest GWAS to date, but prior studies have found a similar overall pattern of results.</p>
<p>And with that, I’ll go ahead and hand things off to Emma to tell us about what’s been found in other disorders.</p>
<p><strong>Emma Johnson:</strong></p>
<p>Thanks, Joe. Here I’m going to present a Manhattan plot, and this is showing the results of the largest GWAS of cannabis use disorder to date. This GWAS combined data from the GPC, iPSYCH, and deCODE Genetics and included about 20,000 cases and just over 360,000 controls. And this GWAS of cannabis use disorder identified two significant loci. There’s a locus on chromosome 7, where the lead risk variant is in an intron of the <em>FOXP2</em> gene. There’s also a significant locus on chromosome 8, and this locus was previously identified by Demontis et al.&nbsp;in the iPSYCH data and replicated in the deCODE data. The lead risk variant here is an eQTL for <em>CHRNA2</em> in the cerebellum as well as <em>EPHX2</em> in the cerebellum and adipose tissue.</p>
<p>I also want to point out some other results from GWAS of cannabis use disorder, and this was just recently accepted in <em>The Lancet Psychiatry.</em> Here we’re contrasting the genetic correlations between cannabis disorder and a number of relevant traits and cannabis initiation and the same traits. As you can see, there’s some divergence when we look at the genetic correlations of cannabis use disorder versus cannabis initiation or ever use. I’m pointing out three in particular here. For body mass index, we see that cannabis use disorder is significantly positively correlated with body mass index. When we look at age at birth of the first child, we see that cannabis use disorder is significantly correlated with an earlier age at first birth, and it is also significantly negatively correlated with educational attainment. When we look at cannabis initiation we see that there’s actually significant correlations with these three traits but the opposite direction of effect. And actually, for 12 of the 22 traits that we tested, those marked with an asterisk we see significantly different genetic correlations when we look at cannabis initiation versus cannabis use disorder.</p>
<p>Moving on to opioid use disorder, this is the latest and largest GWAS of opioid use disorder, and this combined data from the MVP, Yale-Penn, and Age. This is by Hang Zhou et al., and they identified a significant functional coding variant within the mu opioid receptor gene, or <em>OPRM1</em>.</p>
<p>I also want to point out some interesting results from an earlier GWAS of opioid dependence, and this is by Renato Polimanti et al.&nbsp;He looked at a PRS for risk tolerance, or a polygenic risk score for risk tolerance, and a polygenic risk score for neuroticism, and if we look at the risk tolerance PRS, Renato saw significant associations with two of the contrasts that he tested. So he saw that the risk tolerance PRS was significantly associated when he compared individuals with opioid dependence to unexposed controls and when he compared exposed controls to unexposed controls. If we look at the neuroticism PRS, we see that Renato saw significant associations with the opioid-dependent individuals and unexposed controls contrast and the contrast of opioid-dependent individuals and exposed controls, so this suggests that risk tolerance or externalizing behaviors may be more associated with exposure, while something like neuroticism and negative affect may be more associated with dependence than simply exposure.</p>
<p>Moving onto cocaine use disorder has had much smaller sample sizes than for other substance use disorders and fewer GWAS and little replication to date. But what I’m showing here is one of the largest studies published to date, and this is from Spencer Huggett and Mike Stallings. And here they’re actually taking GWAS summary statistics from an earlier study by Gelernter et al., but they’re conducting a gene-wise association in both the European American and the African American samples. They identified this novel significant gene <em>NDUFB9</em> in the African American samples. Spencer has also done some interesting work comparing human genetic studies to mouse models, and here I’m showing a figure from a recent paper of his looking at overlap of differentially expressed genes and gene networks when they compared human cases of cocaine use disorder to a mouse model of cocaine self-administration. They do see significant genetic overlap, suggesting that some of these animal models might be useful for studying substance use disorders, at least in the case of cocaine dependence.</p>
<p>Joe mentioned earlier that twin and family studies have identified some common genetic factors shared amongst substance use disorders more broadly, and we see this mirrored when we look at genome-wide data. So here, I’m presenting a figure on the right, and this is from a paper by Jang et al.&nbsp;I do want to note that here we’re looking at substance use traits or substance initiation, such as lifetime cannabis use, whether someone has ever reported being a regular smoker, so these aren’t substance use disorders per se, but they are correlated with substance use disorders. We do see genetic correlations between and among substance use disorders as well as substance use traits, starting at around 0.5 and higher, so we do see these genetic correlations mirrored in the molecular genetic literature. We also see that substance use disorders and, in this case, substance use traits are correlated with other psychiatric disorders. So, in the cannabis use disorder GWAS, we saw a significant genetic correlation with post-traumatic stress disorder. Alcohol use disorder and major depression are significantly correlated, and with opioid use disorder, we see a significant genetic correlation with ADHD.</p>
<p>So, what’s next for the substance use genetics field? In our review paper, we outline three key areas that we see as priorities for our field for the next few years going forward. The first of these priorities is increasing the diversity of samples that we include in our GWAS. So most of the GWAS that we presented today and certainly the largest GWASs today for substance use and substance abuse disorders have primarily included individuals of European ancestry. And it’s been shown that when discovery GWAS only include individuals of European descent, polygenic risk scores that are derived from those GWAS have limited utility in target samples of other diverse populations. So we really need to increase the diversity of individuals that we’re including in our discovery GWAS. The second priority that we outline is the incorporation of diverse types of multi-omics data as well as extension to cross-species data. So, in terms of multi-omics data, for example, gTEx is a great resource and one that a lot of us use in conducting follow-up analyses for our GWAS. However, we really need substance specific data so gene expression data sets after substance exposure or substance use disorders. Finally, the third priority that we outline is the refinement of phenotypes and ascertainment strategies. So, as I mentioned earlier in terms of cannabis use versus cannabis use disorder, we do see some divergence for some of the substances that we’ve talked about today in terms of use versus use disorder or problematic use. So, for cannabis, there’s some been some evidence of this. There’s also been some evidence of this for alcohol, so when we compare alcohol use disorder to drinks per week or some other measure of perhaps more typical alcohol consumption. But we really need to do a better job of determining whether these phenotypes really show these diverse genetic backgrounds or if there’s something about the ascertainment strategy that is causing us to see these interesting and sometimes paradoxical genetic correlations when we look at use versus use disorder.</p>
<p>Finally, in our review paper, we tried to address the question: “Can we translate any of these substance use or substance use disorder gross findings to the clinic yet?” And the answer is unfortunately we’re not there yet. Particularly with substance abuse and substance use disorders. we really don’t have the sample sizes or the robust findings and power that we need yet to be able to use something like a polygenic risk score in a clinical setting. So polygenic scores, at the moment, explain such a small amount of variance in target samples that that percent variance explained really isn’t at a clinically meaningful level yet, but we hope to get there one day and there may be other strategies, such as pharmacogenetics or drug repurposing/repositioning strategies, that may help us translate these GWAS findings to the clinic eventually.</p>
<p>Thanks for checking out our video, and we would like to mention that if you’re interested in working with us more to better understand the genetics of substance use disorders, please see our PGC website. There’s information there and you can contact the any of the co-PIs of the substance use disorders working group. We would love to work with you. Thanks.</p>
<hr>
</section>
<section id="sec-suicide" class="level1">
<h1>Suicidal Thoughts and Behaviours</h1>
<p><strong>Title</strong>: Insight into the Genetics of Suicide</p>
<p><strong>Presenter(s)</strong>: Hilary Coon, Anna Docherty, Douglas Ruderfer, Niamh Mullins, Eli Stahl (Host)</p>
<p><strong>Eli Stahl</strong>: OK let’s go ahead and get started. Welcome to the Psychiatric Genomics Consortium World Wide Lab meeting on September 27, 2019. Thank you for joining ahead of the conferences and during grant season. I know that at least two presenters are in total crunch time for the October 5th deadline for NIH, so I appreciate making time for this and for you guys presenting and for all of the attendees. Today, we have insights into the genetics of suicide from a group of collaborators within the PGC, the International Suicide Genetics Consortium, and the psychEMERGE consortium working with EHR data. We have professor Hillary Coon from University of Utah, assistant professor Anna Docherty from Utah and the Virginia Commonwealth University Institute for Psychiatric and Behavioral Genetics, VIPBG, assistant professor Doug Ruderfer from Vanderbilt University Medical Center, and postdoctoral fellow Niamh Mullins from Mount Sinai School of Medicine. Their talk titles are shown on this second slide.</p>
<p>The talk titles are shown on this slide. We’re planning for short talks, 10 to 15 minutes, and we hope that there’s time for a short question and answer period at the end of each talk, and then according to plan we’ll have 5 to 10 minutes at the end for questions or any broader discussion. My plan is to look on the attendees list for people who raise their hand. If you raise your hand to ask a question, then I can unmute you, and you can ask your question directly with the panelists here. So we’ll see how this works. Typically then also you can send questions, you can type your questions into either the Q&amp;A button, after pushing the Q&amp;A button at the bottom, or the chat button, and we’ll read your questions aloud and then answer them. September is Suicide Prevention Month, “is that so?”, you ask. Well, every month should be Suicide Prevention Month, and so we focus on suicide biology and genetics. This is Hillary Coon.</p>
<p><strong>Hilary Coon</strong>: Thanks very much. It’s a pleasure to be able to present to you, and we’re absolutely delighted that suicide is now sort of moving into the mainstream of psychiatric genetics, and this of course hasn’t always been the case, it’s been sort of on the fringe, but this hasn’t been because it isn’t important. We never have any trouble with our impact statements of our grants. So suicide is certainly a huge public health crisis, it’s the tenth leading cause of death in the US. In 2017 over 47,000 people died by suicide, and this number is over 800,000 people worldwide. And attempts are much more common: so for every suicide, there’s 25 to 30 attempts. So a big worldwide problem, a growing problem,</p>
<p>and I’m going to give you a little bit of overview of suicidal behaviors and also then suicide death, and our talks are sort of spanning the spectrum of those two outcomes. So certainly the problem is that attempts are increasing, and our prediction of attempts is pretty poor at this stage, which means prevention is really challenging, and a possible solution to this is really playing in genetics.</p>
<p>So the heritability of suicidal behavior from twin and family studies is up to 55%, which suggests some promise for using this type of data to really help us in prediction and prevention. We have an opportunity, though, because most of the prior genome-wide studies have been within specific psychiatric diagnoses, and actually, most of them really have not been large enough to identify any replicated genome-wide significant loci. So, the time is really right for studies for studies using really large population samples and also the very large psychiatric samples within the Psychiatric Genomics Consortium, and you’ll hear about both of those efforts from our colleagues, Dr.&nbsp;Ruderfer and Dr.&nbsp;Mullins.</p>
<p>At Utah, we actually focus more on suicide deaths, and this is partly because of some resources that I’ll tell you about in a few minutes. I just wanted to highlight some of the differences between suicide death and suicide attempt, which were really focused on trying to figure out some of the potential differences in genetic etiology between these two outcomes. So, just at a very basic level, suicide death is way more rare than attempts (I mentioned this in the previous slide), and then if you look at another very sort of basic etiological variable, which is sex, especially death is very male. In the US, this male to female ratio is almost four to one. For attempts, it’s more difficult to get counts accurately, but they’re far more common in females, and this is especially in youth where the females are about twice as likely to make an attempt. It is also worth noting that I showed you a slide about the incidence of attempt increasingly; incidents of suicide death are also increasing dramatically. It’s increased about 33 percent since 1999.</p>
<p>So one issue that we think about a lot is to what extent is suicide really sort of some dramatic etiological similarity with mental illness. So many of the individuals who die by suicide struggle with mental illness, and this has often been a chronic situation, but we also know that most individuals who have a mental illness don’t die by suicide. So there’s some indication that the suicide has some other etiology over in above just psychiatric vulnerability.</p>
<p>The slide I had here is actually not a Utah extended family; this is actually from Jenna Egeland’s seminal work in the Old Order Amish. This is a suicide family that is one of many families that she studied for major depression and bipolar disorder. And what she found, she had records for suicide expand a hundred years in this cohort, and of those suicides, all of them fell into just four of the families that she was studying, and she made a conclusion (this is in the 1980s) that suicide were significantly familial and that it looked like there were some risks that were independent of psychopathology or at least that spanned multiple psychiatric disorders. There’s some other seminal work that’s been done by Brent and Mann published in 2005; it that also suggests that there potentially risk factors unique to suicide.</p>
<p>I mentioned that suicide attempt is significantly genetic. These are some data from aggregated studies of twin and adoption studies of suicide death. So this outcome also looks like it’s significantly genetic. You can see the identical twins are much more concordant than fraternal twins, and also in the adoption studies, the biological relatives are much more at risk than the adopting relatives. From some meta-analysis that this work, the evidence for suicide death is actually similar to the heritability for attempt; it’s about 50%.</p>
<p>Okay, so why Utah? Well, Utah has the sort of dubious distinction of being in the top fifth or sixth for suicide rate (dipped down to sixth recently, but still, we’re in what’s called the suicide belt of the US). In Utah, suicide is actually the leading cause of deaths for people under 25. We also have some just really great resources that make the logistics of doing this fantastic. So we have a centralized medical examiner’s office, and the medical examiner is really interested in collaborating with us, and we’ve been collaborating for about twenty years. We now have over sixty-two hundred individuals who died by suicide with DNA, and this grows at the rate of about seven hundred a year. So within about the next five years, we’ll have about 10,000 individuals who died by suicide with DNA.</p>
<p>We also can link these cases to a fantastic population resource called the Utah population database. This is a statewide database that includes access to extensive medical records data, demographics, genealogical records, geographical data, which allows us to look at exposures, and is a terrific resource. Now, one of the potentially most unique aspects of this resource is actually the extended families. We have very large high-risk families in the Utah population database through a gift of the genealogical data from the LDS Church, and we have identified forty-three very high-risk families where we then looked for familial sharing, genomic sharing, and people who were then related in this distant way. We’ve actually published this in Molecular Psychiatry and found 30 genome-wide significant regions in ten of these families. We’re following this up with some sequence data, and I listed a couple of these families below that gave us these significant regions. The people that actually have data are circled in red. So, we found that the genomic regions that are familial, these are challenging to follow up because what’s driving this significance is likely regulatory; it’s not necessarily really easy to see a single variant with very high penetrance. So what we’re doing in the meantime is actually using the familial cases in a couple of different ways to address some other hypotheses.</p>
<p>So these families, these people in these families, actually represent cases with enhanced genetic risk. They often, once we get up a couple of generations, are tied to common multiple common founders. So the genetic risk is actually not just part of one single family. We affectionately call this “spaghettigrees” in the Utah population database, and this happens with other conditions as well. So they really represent familial enrichment. This is also genetic enrichment, right, because these cases share very little common environment; they’re very distantly related. So what we thought we’d do is look at these cases and see if they look like they were at significantly elevated risk of psychopathology, and by using polygenic risk scoring (and you’ll hear more about the polygenic risk scores from my colleague, Dr.&nbsp;Docherty), what we did was we looked at these genetically enriched cases (the European subset of them), calculated 60 polygenic scores for a variety of traits, and then we’ve just looked at the difference between these genetically enriched cases and our other suicide deaths, then again adjusting for residual ancestry effects. We also included polygenic scores of suicide death, actually direct for our own cohort, and we used a 10-fold cross-validation to do this since it’s direct in our own cases.</p>
<p>So here’s what we find: the high-risk familial suicides are actually significantly elevated just for risk of suicide death. We did not find any enrichment for any other psychopathology in these cases that are selected for genetic risk. They just look like they’re enriched for suicide death, and we found this to be very interesting. We were kind of expecting them to be enriched for some other genetic risk of psychopathology, but didn’t find this.</p>
<p>So we’re also looking at these cases and beginning to see some associations of gene pathway level between them. Looking at rare structural variants and single nucleotide variants of high impact, these look like they’re starting to cross-match with findings from our GWAS, which is very interesting. If you want to talk to me about that, you’ll have to find me at World Congress; I don’t have time to tell you about that now, but it’s exciting work.</p>
<p>We really believe that studying suicide death is important, of course, studying attempt is very important as well, but by studying suicide death, we can really try to get at genetic discovery for the very, very highest of the high-risk cases, and we can really start to look at the extent to which etiology differs between suicide attempt and death.</p>
<p>I just want to acknowledge a ton of people who are working on this project. It’s a really big resource, a fantastic resource, with lots of folks at the University of Utah and then across different departments within our university and many external collaborators. And this is growing; we are really excited to be part of the international consortium work, and lots of folks who have supported this work.</p>
<p><strong>Eli Stahl</strong>: Thank you, Hillary. I think that it’s worth it to take a brief time for questions. If anyone has a question, please raise your hand or again type your question into the Q&amp;A box or the chat box quickly, please, before we move on to the next presentation. Five, four, three, two, one. Okay, we’ll move on to the next presentation, which is Anna Docherty.</p>
<p><strong>Anna Docherty</strong>: Okay, thank you for attending today, and I am going to briefly discuss the genome-wide association analysis of the suicide death samples here in Utah, from the incredible resources that Hillary Coon has obtained here. This is going to be a brief talk, so I’m obviously happy to provide details to interested people offline as well.</p>
<p>Our genome-wide association study of the first genotyped suicide deaths was a case-control design with 3,413 suicide deaths from Utah, these are non-Finnish northern European suicide deaths, and 14,810 controls. We had some significant challenges to the genome-wide association analysis using Utah suicide data, because we don’t have controls in Utah, and so we had to take multiple steps to filter and minimize false positives in our analyses. Another challenge is that we have only about 20 percent of our sample with significant admixture, and so with this preliminary data set, we didn’t have a sufficient sample to complete a trans-ancestry GWAS, although we do expect in the next three years to have the numbers to do that.</p>
<p>So, I’m going to talk about the steps to minimize false positives in a moment. These are the PCA plots of the suicide deaths and the controls from Generation Scotland and UK10K that we used, as well as the 1000Genomes super populations. On the left, you’ll see plots of the entire sample, and then on the right, you’ll see a zoomed-in view of the hinge of that entire sample. On the top, you’ll see the suicides and controls mapped to 1000Genomes, and on the bottom, you’ll see just those suicide deaths and the control samples, along with those that we excluded for this analysis.</p>
<p>We took multiple steps to filter out potential false positives. One, we jointly imputed cases and controls from a common SNP list. We also created limits around a centroid in the CEU data (the European 1000Genomes data), where 99% of the CEU samples fell within those limits. We then excluded cases and controls that did not fall within those limits. We also, for the GWAS analysis, we used mixed models with genomic relatedness matrices and ancestry PCs. For this analysis, we wanted to use two control cohorts because we were concerned that there would be effects specific to each control cohort, false positive effects. And so, we wanted to filter out that signal. And so we conducted a control-control GWAS before the case-control GWAS, and then the suggestive signal was filtered out of that subsequent case-control GWAS.</p>
<p>I’m showing you the results here of the 3,400 suicides and about 14,000-15,000 controls. These are the results from the GWAS indicating six variants from two loci associated with suicide death at a genome-wide significant level. We had another 52 variants that were nominally significant, and those matched to 19 genes. As you can see in the Manhattan plot here, the genome-wide significance level is in green, and our FDR correction in purple is at 0.1. Our lambda for this analysis was 1.015.</p>
<p>Here are the top SNPs meeting genome-wide significance on chromosomes 13 and 15. Most of these are intergenic. Our LD score regression SNP-based heritability for this analysis was 0.246 with a standard error of 0.036, so highly significant.</p>
<p>These are our gene-based results from the GWAS, 10 met FDR correction at 0.1. We used MAGMA to detect 19 genes associated with suicide out of about 18,000 total genes. Our lambda for this test was 0.994. Associations of chromosome 13 SNPs were found with three genes, including <em>DACH1</em>. Eleven of the 19 genes have prior evidence of association with suicidal behaviors, and there was a significant association with Schizophrenia results from the GWAS Catalog. Ten of the genes also show evidence of significant differential gene expression in post-mortem brain in either schizophrenia, autism, or bipolar disorder, and this is corrected at 0.05.</p>
<p>We wanted to leverage the fact that we had two waves of suicide death samples and two control cohorts to cross-validate and predict polygenic risk for suicide death based on case-control status. So what we did here is, to orient each of these plots on the y-axis, you’ll see, centered around zero, polygenic risk score for suicide, and on the x-axis, you’ll see the p-value threshold cutoff for the polygenic risk score, starting at 10% of the variance all the way up to all of the variance. In the first plot on the left, we used Wave 1 and Generation Scotland as the training sample to score Wave 2 and UK10K, and then the procedure was reversed on the right-hand plot. In both cases, suicide PRS significantly differentiated cases from controls.</p>
<p>For this next analysis, I wanted to characterize the polygenic risk in suicide cases for different disorders potentially relevant to suicide death. So here, these plots are a little different, these are plotting suicide in red, Generation Scotland and UK10K controls in green and blue. These PRS are based on summary data from large-extant GWAS of multiple phenotypes. What we find is significant increases in the phenotypes we would expect to be related to suicide, including depression, autism spectrum disorder, disinhibition, schizophrenia, and alcohol use.</p>
<p>We also wanted to dig into some of the clinical data and the mode-of-suicide data, and there are four sort-of types of suicide, because some of the epidemiological research suggests pretty complex etiologies and potentially risk subtypes. So I looked at ICD diagnoses from the electronic health records in the suicide deaths, and mode of death as well, with respect to polygenic risk for these conditions. One notable finding worth mentioning is that suicide by violent trauma, which is more rare, it’s a more rare form of suicide, was associated with both a clinical diagnosis of schizophrenia and polygenic risk of schizophrenia.</p>
<p>I’m putting up here some of the more prevalent diagnoses associated in the suicide sample here to show that while schizophrenia rates are very high in this sample, the diagnoses associated with suicide death are typically quite different. We see a lot of pain-related depression, anxiety diagnoses, as well as cardiovascular and metabolic disorders.</p>
<p>Just to wrap up, we have some future goals here with a larger cohort to complete a replication with PGC controls. We are working on a multi-trait-based conditional and joint analysis using GWAS summary data. I’m interested in a trans-ancestry analysis when we finally have enough samples with significant admixture, and we have located, with Dr.&nbsp;Stahl’s help, Genomic Psychiatry controls. We are also doing some genetic subtyping with our GWAS and we are working on leveraging suicide death PRS to enhance prediction models of suicide mortality as opposed to suicide attempt, with colleagues Eli Stahl and Niamh Mullins. And with that, I will wrap up and thank all of the contributors to the study and consultants.</p>
<p><strong>Eli Stahl</strong>: Thank you, Anna. We have another short period that we can take questions. So, please, again, type any questions you may have into the Q&amp;A box or the chat box. Alternatively, you can raise your hand, and I can unmute you.</p>
<p>We do have one question that was typed into the chat box from Paul Nestat. If you want to read it and answer it, or I can do it if you want.</p>
<p><strong>Anna Docherty</strong>: I don’t have the…can you…</p>
<p><strong>Eli Stahl</strong>: okay Question: Was there a reason that controls weren’t drawn from the Utah Medical Examiner’s other death investigations (non-suicide)? Presumably, if they have been saving blood from suicide deaths, they also did for others. In Maryland, we plan to use MVA passenger deaths as controls when we look at suicide bloods from the medical examiner, for instance.</p>
<p><strong>Hilary Coon</strong>: Right, that’s actually… I can answer that. That’s a perfect control group, and we have been trying to pursue permissions for that. Our medical examiners have been down by about four medical examiners. They recently hired some more folks, so they’re just way too busy, way overworked, and every time I ask them for more, they say, “We’ll think about that.” So, we have been definitely working on it. We have a couple of other potential opportunities for some biorepository samples, but we’re really, really interested in getting a Utah-specific control group. We recognize this is a big issue. We have a small Utah control group, unrelated people from the CEPH CEU, or Utah people. We have another control group that’s individuals who are elderly healthy, study. A lot of the studies in Utah are families because that’s kind of a big strength here. So that’s a bit of a challenge. We’ve been collecting unrelated people from various family studies also as controls. So, yeah, we’re working on it and definitely working on our medical examiner as well. Lots of you know, “Donuts down” to the medical examiner to see if they can help us out with some of those samples.</p>
<p><strong>Eli Stahl</strong>: Okay, if there are any other questions? Any other questions at this time? So, let’s move on and hear from Doug Ruderfer.</p>
<p><strong>Doug Ruderfer</strong>: Excellent! Well, thank you Eli for organizing this, and it’s happy to be part of this session. This deserves a whole session for the worldwide meeting and happy to have great colleagues that are dedicated to the issues. We came into this for the same reasons that here we mentioned, for all the important public health reasons and for that necessity to kind of learn about ongoing biology of this devastating outcome. We came at it from a slightly different direction, and that’s partly out of a requirement, and everyone’s familiar with this idea of population samples, are, do have limited powerful phenotypes that are of low prevalence, and for our own work at Vanderbilt, we have about three million unique patients that have been seen at least once at the hospital, but many of those patients actually come for particular procedures, or give birth, because a trauma center, or go to the ER, and don’t have a lot of actual data. If we had three million new patients and we had DNA on all of them, we could have a pretty well-powered genome-wide association study even for a 1% disorder, but we clearly don’t.</p>
<p>If we think about the individuals where we have longitudinal data, multiple events, what we call our “frequent visitors”, or medical home patients, we substantially reduce the number of individuals to about 1 million patients, and of course, collecting DNA on all of those would still be a challenge, though we have a pretty robust collection process where we have about 1/4 million individuals that have DNA in our biobank, named BioVu.</p>
<p>But as you see, these numbers reducing at this very moment of course is increasing rapidly; about half those have been genotyped, and again, this has changed dramatically over the last year or so and has also changed over, kind of even, within this talk I’m gonna give, so we don’t really have enough cases in the Biobank to constitute a significantly powered genetic study, but what we were trying to think about is how we can leverage this information, this extensive clinical information, that clinical data that we have, and the large overall sample size that exists, to improve our ability to study the genetics of these low prevalence disorders…</p>
<p>And that’s kind of the question that led us to think about suicide as a particular phenotype of interest and one that has been hard to ascertain, but also many other potential phenotypes, and just in the interest of time, to cut to the chase, what we were trying to do here and what our approach was, we think about our sample of a million patients. We can identify cases as “gold standard” chart-validated, best quality possible set of individuals, and then we can utilize the clinical information that we have from electronic health records that could be structured data, which is what easy to utilize, work with notes, labs, of diagnosis, medications, procedural codes, to build a predictive model. And the idea is that we can build a predictive model and we can demonstrate that the model’s both predictive and accurate, meaning that if we say a person’s in case, they are case, and well calibrated, meaning that we say to individuals 20% probability of being a case, then 20% of individuals are actually in cases.</p>
<p>Then what we’ve done is we’ve actually converted our binary outcome into a quantitative measure, which theoretically should be well more powered than using case-control status. So that’s the argument, the theory we’re trying to work behind. There’s some really nice theoretical work by Yang, Wray, and Visscher actually doing these calculations and comparing effective power between a quantitative trait and a binary trait, showing that particularly for low prevalent phenotypes you have many, many, many fold more power to study the quantitative trait than the binary trait itself, and I suggest looking at the paper, it’s actually pretty fantastic.</p>
<p>This leveraging kind of work, those going on by my great colleague Colin Walsh, we wanted to apply this to suicide, again, for all the public health reasons and for the interest, and at the time when we started this a few years ago, there was limited knowledge in terms of the underlying genetics for this particular trait. In general, it’s great to see that that has advanced rapidly over the last couple years. In reviewing charts of all individuals that had self-injury codes, about 8,000 individuals, they looked through every single one to identify the ones they could confirm by hand, so validate through the charts, that had a suicide attempt. There’s about 3,000 individuals among our dataset that had that chart-validated suicide attempt. We could build a model now to predict suicide attempt, leveraging the rest of the data that we have, including demographic information, medications, diagnoses, and hospital utilization. And it turns out that using random forests performed the best among a number of approaches, and provided a prediction accuracy closer to 0.9 or so, and a good-calibrated model, so fitting the expectation that we think in theory should provide us a quantitative trait that would be well-powered compared to our binary outcome.</p>
<p>And we applied this to the sample that we had at the time, which is about 25,000 European individuals that were genotyped. And to keep in mind, among those individuals that were genotyped, 73 were cases. What we’re trying to do here is leverage this clinical data to build a quantitative trait to demonstrate that we might have power to do GWAS on 73 cases using the quantitative trait.</p>
<p>So we needed an out-group to compare this to, we needed a classically defined patient outcome to do this thing. So we leveraged work with our colleague Manny RIvas at Stanford to use the UK Biobank. Among the 500,000 individuals, about 150,000 of them filled out an online mental health assessment, and received, or the ones who responded “yes” to a self-injury question, the question, “Have you ever harmed yourself with the intention of ending your life?” which is a good suicide attempt question because it includes intent. 3,500 individuals responded “yes”, after quality control of their genetic data and reducing to a homogeneous, white, British ancestry, there were about 2,400 cases of suicide attempt. And we compared this to all the controls. If we compared it to only the controls that filled out the online mental health assessment, the results do not change.</p>
<p>So this provides us now with a classically ascertained self-reported patient suicide attempt phenotype and the dataset to compare those, well-powered, although again, neither of these studies should be well-powered to a very large-scale GWAS for a polygenic trait.</p>
<p>So the first thing we can do is actually do a genome-wide association study in the UK Biobank, and not surprisingly, we did not find any genome-wide significant loci. We can then take this result, build polygenic risk scores from the UK Biobank, and apply it to our case-control status in our BioVU sample, and we do see nominally significant enrichment, even though it’s 73 cases, that these individuals carry higher genetic risk for suicide based on UK Biobank GWAS than the controls. And if we then compare now to a predicted probability of suicide risk, we see a much more significant correlation between underlying genetic risk of suicide and the probability of suicide attempt. And this holds even if we remove those 73 cases, so the power is not generated from those cases, it’s actually generated from the clinical data that informs that probability, as part of controls. As we expect. We can do a genome-wide Association study in just our BioVU sample, and we do see two genome-wide significant hits that are tabled at the moment because neither of them have been replicated in the UK Biobank, but are worth investigating further as we increase sample size, which Niamh will tell you more about directive of hers for us to do that.</p>
<p>And more interestingly, we can look at our QQ plots, so we can look at our heritability, and we do find both significant and comparable heritability estimates between our UK Biobank sample and our BioVU sample, around four or five percent.</p>
<p>And I think more interestingly, you can do a direct genetic correlation calculation between these two samples, and then if a significant genetic correlation between the two different samples in two different ways, coming close to one with high confidence intervals, and we also can show that significant genetic correlations with traits that we would expect based on our understanding of clinical risk for suicide attempt, including depressive symptoms, major depression, neuroticism, schizophrenia, insomnia, and somewhat interesting, a negative relationship with age of first birth, which has shown up in other disorders, including schizophrenia.</p>
<p>So, to summarize, we think of this type of model, again, if you can be accurate, if you can have good calibration, can capture genetic contribution, then effectively, you can do a GWAS with a small number of 73 cases, identified some heritability, and showed that it was significantly genetically correlated with a standard patient-reported outcome. There are lots of opportunities to utilize this approach for phenotypes that are either hard to ascertain or of low prevalence, and we think about in both our work and also in the UK Biobank now that they have primary care information.</p>
<p>The nice thing about this type of approach, again if it works, is your power increases with sample size, not purely with case numbers. And we’re trying to do this now with the additional 60,000 European samples we have in our own BioVU sample to further demonstrate that power increase.</p>
<p>We do have work that is currently funded with both Colin and our colleague Roy Perlis at MGH to do the same approach and apply it to treatment resistance in depression and schizophrenia and weave in suicide.</p>
<p>The realm have been going further with our colleague Adi Bejan, thinking about how do we both identify more individuals with notes. One of the things were missing currently is all historical suicide attempts that we can capture, but thousands of more individuals have historical suicide attempts we can capture suicidal thoughts or ideation from using clinical narrative notes. And we can demonstrate, we can both improve our prediction model, but also demonstrate the accuracy of that using NLP with the predictions that we currently have. So, kind of reiterating improved models, improved power for genetic studies as we move forward.</p>
<p>So, with that, I’ll just thank the wonderful colleagues that in all of the suicide genetics here, my colleagues directly Colin, Adi, and Mani, everybody in my lab, and of course, all the funding and everybody else for listening.</p>
<p><strong>Eli Stahl</strong>: Thank you, Doug. Again, let’s have a brief question period now. You can raise your hand as an attendee, and I can unmute you to join the conversation. Or you can type your questions into the Q&amp;A or chat dialog boxes. So, there is a question in the Q&amp;A: Doug, can you see it? Or I can read it to you if you want.</p>
<p><strong>Doug:</strong> I can see it. So, right. So, I think that we had started to…Okay, so let me read the question in case everyone is not looking at it. The question from Adrian Campos: what data is missing from UK Biobank to calculate the [indistinguishable] comparable suicide liability model? Before GWAS, and until recently, much of the data was missing. So, it was mostly self-report, patient report, information which we toyed around using, but the extensive longitudinal information becomes really valuable for lots of the model that we’re currently using, which is based more on diagnostic codes, medications, and procedural codes, which now, given I think for some of the individuals, have their primary care information. I think that’s more, it’s way more possible to leverage that information to build similar models in UK Biobank, and that’s kind of our hope to move that forward.</p>
<p><strong>Eli Stahl</strong>: Thank you. Are there any other questions at this time? Okay, we can move now to our final presentation, Niamh Mullins.</p>
<p><strong>Niamh Mullins</strong>: Thank you. Okay, can you see my slides? And can you hear me? So, this is a within-case design, and it’s aimed at detecting associations specific for suicide attempt, and not the psychiatric disorder, and variants occurring at a higher frequency in suicide attempters compared with other psychiatric cases. I ran a GWAS of suicide attempt in each disorder followed by meta-analysis between them. In total, there were over six and a half thousand cases of suicide attempt.</p>
<p>There were a couple of genome-wide significant associations. One in the GWAS of suicide attempt in MDD. It had a MAF of 2%; Another in suicide attempt and bipolar disorder, which had a frequency of 20%. In the GWAS of suicide attempt in schizophrenia, there were no genome-wide significant associations.</p>
<p>In a meta-analysis of suicide attempt in mood disorders, there were two GWAS hits. One of them on chromosome 2, again it had a low MAF and only one SNP passing the threshold. But this one on chromosome 4 is the same locus that came up in the GWAS of suicide attempt in bipolar disorder, and the association strengthened in the meta-analysis. Ten SNPs passed the genome-wide significance thresholds, and there’s a nice peak here of SNPs in LD, and the region is a location of a long non-coding RNA. In support of our study design, it has not been associated with either depression or bipolar disorder, suggesting its specificity for a suicide attempt.</p>
<p>So, this looked like a very promising locus, and we wanted to try and replicate this association. So, we went to our collaborators who work on the UK Biobank sample and the iPSYCH sample in Denmark. They pulled out cases with mood disorders, split them into suicide attempters and non-attempters to replicate our study design. The numbers were very decent; we had good powerful replication, but there was no association between this top SNP and suicide attempt in either of the replication datasets.</p>
<p>One of the most interesting findings from this paper was that polygenic scores for depression are associated with suicide attempt in each of these psychiatric disorders. So, what I’m showing here is a polygenic score for depression predicting suicide attempt versus non-attempt from left to right in bipolar disorder, MDD, schizophrenia. So, even within depression, for example, depressed cases who made a suicide attempt have higher genetic risk for depression compared with depressed cases who don’t make a suicide attempt. And even though this is old PGC data, there is no overlap here between the discovery and target samples.</p>
<p>It was also interesting that this finding seems to be specific to the depression polygenic score. So, here in the pink bars, I’m testing whether a polygenic score for bipolar disorder differs between suicide attempters and non-attempters with bipolar disorder. There’s no difference between the two groups. What I’m showing in green is that there is a difference in polygenic scores for schizophrenia between suicide attempters and non-attempters with schizophrenia. But it’s actually the non-attempters who have higher genetic risk for schizophrenia, and as I said, the suicide attempters with schizophrenia have higher genetic risk for depression. So essentially, the take-home message from the polygenic scoring analyses is that across psychiatric disorders, suicide attempters have a higher genetic liability for depression, not just a higher genetic liability for the psychiatric disorder that they are affected by. And even though we haven’t reached their sample sizes where we’re replicating genome-wide significant associations for suicide attempt across cohorts, we are starting to get insights into the genetic etiology of the phenotype using polygenic methods. This was a collaborative effort from many people in the PGC and three of the working groups, so I’d like to acknowledge all of the collaborators, and the GWAS summary statistics are all on the PGC website.</p>
<p>For the final part of the session today, we wanted to introduce the new initiative that all of the presenters are part of, and many others as well, and that’s the International Suicide Genetics Consortium. This is a working group that was set up last year, and our goal is to really scale up genetic studies of suicide to the sample size that we all know is needed. Our initial aim was to conduct a meta-analysis of all the GWAS summary statistics that we could get on suicide attempt or death.</p>
<p>We have 15 participating groups, and so far, we’ve gathered data on over 27,000 cases of suicide attempt or suicide, including a range of psychiatric diagnoses as well as several genetic ancestries. The group was set off by myself and Doug, along with help and direction from Cathryn Lewis and Eli Stahl.</p>
<p>These are all of our members, and all of the studies presented today are represented. We also have the Generation Scotland study, Army STARRs, we have two African American cohorts: Janssen are working with us, Columbia University, the German Borderline Genetics Consortium, Karolinska. Most of the samples are European; we do have two East Asian cohorts as well. Recently, we received data from the new Australian Genetics of Depression study, and of course, the PGC are involved, including the Eating Disorders Working Group. So, it’s a worldwide effort.</p>
<p>You’ll notice that we have many different types of cohorts here, and we really wanted to try and leverage all of the data that we could for this project. So, we designed the study under three analysis models. Depending on the type of cohort, it can be analyzed under one or more of these models, and the difference between them is really the type of controls that are being used. Model one is a GWAS of cases of suicide attempt with a specific psychiatric disorder versus non-attempters with the same disorder, that’s a within case-control design, the same style as the PGC GWAS. Model two is the GWA of suicide attempt or death versus general population controls. And model 3 is suicide attempt or death versus screened non-psychiatric controls, and in both model two and three, the controls are also screened for suicide attempt if that information is available. These are the numbers we have in each of the models. So, we’ve far exceeded the sample size of any of the previous individual GWAS on suicide, which is a great start. We did anticipate from the outset that there would be a difference in statistical power between these models, with greater power from model two and three, not only because of their sample size but also the phenotypic difference between cases and controls in these two models.</p>
<p>One limitation of model 3, which is the case versus screened control model, is in distinguishing whether any associations are really for suicide or whether they might reflect genetic liability to psychiatric disorders. But we will look for convergence of evidence across the models, and model 1 is designed to distinguish between those scenarios. So, we’re using these three models to leverage all the data that we can to accelerate discovery and to complement each other.</p>
<p>Most of the cases in the study are of suicide attempt, some of death by suicide, but we didn’t include individuals with suicidal thoughts only in the case group. In terms of the ascertainment of suicide attempt, most of the studies had used a psychiatric interview. All of the collaborators conducted the GWAS in their own sample. It’s a within-ancestry GWAS under the appropriate model depending on the type of controls they have. We all used a standardized analysis protocol that we came up with to minimize heterogeneity across the analyses. The GWAS summary statistics were shared within the consortium, and we ran a meta-analysis across all studies, regardless of ancestry, within each of the three models. So, those are our data, and this is our setup. We’re quite a new consortium, and we have a big analysis plan ahead of us.</p>
<p>These are the preliminary results from the three models:</p>
<ul>
<li><p>There are six genome-wide significant loci, all of them from either model two or three.</p></li>
<li><p>There was significant SNP heritability in model two at six percent, and thirteen percent in the model of cases versus screened controls.</p></li>
<li><p>In model one, there were no genome-wide significant SNPs, and only a hint of SNP heritability, which was non-significant, So as we anticipated, model one likely has the lowest power out of the three.</p></li>
</ul>
<p>The gene labels here are just the nearest gene to the index SNP. Some of these regions have been associated with psychiatric disorders before, like the MHC and SHANK2. So, we have secondary analyses underway now to really delve into these loci and to dissect these results further using polygenic methods as well. So, watch this space. We have a presentation next month at the World Congress of Psychiatric Genetics, so we’re looking forward to sharing more results then. We’re also really excited that there’s an entire session on the genetics of suicide for the first time, so I think that’s a sign of all the work that’s going on in this area. These are all members of the ISGC who we’ve all been working together over the last year or so to get this study up and running, and if anybody else would like to get involved, feel free to send us an email.</p>
<p><strong>Eli Stahl:</strong> Thank you, Niamh. We’d like to take any questions, perhaps initially specifically for Niamh, and then open it up to any of the panelists. There are two questions. Please raise your hand as a participant or attendee, and I can unmute you to join the conversation. Or you can type a question into either the Q&amp;A dialog box or the chat dialog box.</p>
<p>So, there are two questions. One is in the chat box from Dave Curtis. Dave, shall I unmute you and allow you to ask your question. Let’s see, I need to, I believe, try to unmute you. I’m not able to do so.</p>
<p><strong>Dave Curtis</strong>: I was just assuming, idea about, you take the, you have a people diagnosed with depression and say some have a suicidal send some don’t but although they, they’ve all got depression it might be that the attempters have more severe depression than the non-attempters, and so what you may be picking up is not really suicidality but kind of general depressive severity. And then if you’re taking a different diagnosis, schizophrenia, you may have some people to get schizophrenia, tend to be more or less depressed, and the more depressed ones tend to be more suicidal, but again it might, you know, I’m not really sure if we’re picking up something specific about suicidality or whether it isn’t other features of psychopathology, like how generally depressed somebody is, how generally disinhibited they are, how impulsive they are, have borderline they are, how substances usually they are, and all those other quantitative traits that we expect to vary with between people within the same diagnosis.</p>
<p><strong>Niamh Mullins</strong>: Yes, oh, that’s definitely what we’re seeing across the polygenic scoring, suicide attempters always have higher genetic risk for depression, regardless of their psychiatric disorder. And that is fitting with the clinical picture. For example, the presence of depressive symptoms in schizophrenia are a risk factor for suicide, longer depressive episodes in bipolar disorder are a risk factor. We did find in the PGC data that the suicide attempters with schizophrenia did have higher factor scores for depression. So, yes, it’s fitting with the clinical picture.</p>
<p>And our analyses, though were aimed to try and tease apart the genetics of suicide attempts compared with the psychiatric disorder, and so it can pick up associations specific to suicide attempts, but also, given what we know about the polygenic nature of these traits, and polygenicity between many disorders. We don’t necessarily expect all of these genetic variants to be specific for suicide attempts but don’t overlap with other disorders. And within the case model of depression, for example, not only can it pick up associations specific for suicide but associations that are pleiotropic with depression but occur at a higher frequency in suicide attempters. And so what we’ve seen from this study and many others is that the disorder that suicide attempt seems to be most closely genetically related to is depression.</p>
<p><strong>Eli Stahl</strong>: Any follow up? So the question in the Q&amp;A comes from Adrian Campos. If Adrian, you would like to ask your question yourself, I can allow you to unmute yourself now if you would like.</p>
<p><strong>Adrian Campos</strong>: Yeah, please, can you hear me?</p>
<p><strong>Eli Stahl</strong>: Please ask your question as well.</p>
<p><strong>Adrian Campos</strong>: I was just wondering whether there was any evidence for differences in inflation or power between different types of ascertainment for suicide attempt. For example, you mentioned some groups with psychiatric interviews versus using self-report or medical health records.</p>
<p><strong>Niamh Mullins</strong>: Yeah, so this is in the ISGC study, and we do have several different methods of ascertainment. Most of them are psychiatric interviews. Some studies did online recruitment or self-report. Some studies used hospital records like ICD codes. So, yeah, there’s possibly some heterogeneity in the definition of the phenotype from different methods of ascertainment, and we do intend to look into that. For example, we could look at the genetic correlations between the different cohorts depending on the different methods of ascertainment, and we’ll certainly look at this. One challenge with doing that is the statistical power in the sample size to do genetic correlation analyses for each of the cohorts individually, and it does pose a challenge to testing that.</p>
<p><strong>Adrian Campos:</strong> Alright, thank you.</p>
<p><strong>Eli Stahl</strong>: Thank you. We are at time. I’m not sure what happened at 11 o’clock, but if there are any other questions, perhaps there is just a tiny amount of time remaining. And if not, I would like to thank you all again for attending the PGC Worldwide Lab Meeting: Insights into the Genetics of Suicide. Thank you, and have a good day.</p>
<hr>
</section>
<section id="sec-tourette" class="level1">
<h1>Tourette Syndrome</h1>
<p><strong>Title</strong>: Genetics of Tourette Syndrome</p>
<p><strong>Presenter(s)</strong>: Matt Halvorsen</p>
<p>Hi, everyone. My name is Matt Halvorsen, and today I’ll be presenting on the genetics of Tourette syndrome.</p>
<p>So first, a definition of Tourette syndrome or TS. It belongs to a spectrum of neurodevelopmental conditions, referred to as tic disorders, and this is from the Diagnostic and Statistical Manual of Mental Disorders, or DSM, Fifth Edition. First, let’s define tics. Tics are defined as sudden rapid recurrent motor movements or vocalizations, and the basic criteria for being diagnosed with TS is having both motor tics and vocal tics that present for at least one year. Other key tic disorders to keep in mind are chronic tic disorder, or CTD. The feature of CTD is basically motor tics or vocal tics, but not both, lasting for at least one year. So tic disorders have a wide spectrum.</p>
<p>TS prevalence and comorbidities. World estimate prevalence of TS is in the range of 0.3% to 1% in children. Psychiatric comorbidities. TS cases most often frequently are diagnosed with obsessive-compulsive disorder, or OCD - that’s 50% of cases - and attention deficit hyperactivity disorder, or ADHD - and 54% of cases.</p>
<p>So brief history of Tourette syndrome. First described by and named after Georges Albert Edouard Brutus Gilles de la Tourette in 1885. He described the condition in nine separate patients, but even before that, tics have been described in individuals as far back as the 15th century. I’ve highlighted here the book <em>Malleus Maleficarum</em>,which translates to the ‘Hammer of Witches’ and is basically encyclopedia of witchcraft written during that time period. And in it, a case of a priest is described that has a tics and they blame it on demonic possession. Nowadays we have, instead of stacks of <em>Malleus Maleficarum</em> books, stacks of DSM-5 books. And in those books, both tics and TS are recognized as neurodevelopmental in origin rather than some origin that’s superstitious.</p>
<p>So what evidence do we have that Tourette syndrome has a large genetic component? Well, first-line evidence is from twin studies. And in twin studies, what we do is we take twins, and we determine you know, in instances where one of them has a disorder like Tourette syndrome, how often does the other twin have that disorder? And so, you get these concordance rates that range from 0 to 1; if the values are 0, then basically there’s no genetic contribution to risk, and if value is 1, then genetics entirely explains risk. So these estimates that we’re listing here, they’re not the first ones that were derived for Tourette syndrome, but they’re one of the more recent ones. This is a large meta-analysis of twin studies, and it features a bunch of different disorders and traits in terms of these concordance values but here we’re just going to concentrate on values for TS. Monozygotic twins, this concordance estimate, and when I say monozygotic I mean identical twins, so the estimate here is 0.63, and then dizygotic twins, so twins that are not identical, basically siblings, that estimate is 0.34. So based on this and other more specifically TS-focused twin studies appear to explain a large portion, but not all of risk for TS and for tic disorders in general.</p>
<p>So another line of evidence is more generalized family studies. And so when we talk about family studies, for instance, we’re talking about taking population-scale registry data, finding individuals of Tourette syndrome, and then finding their relatives in the data and saying, “Okay, are individuals that are first-degree relative, second-degree, third-degree… if you’re a first-degree relative of somebody with TS or CTD, are you yourself more likely to be a case of a TS or CTD?” And two different studies that came out in the last couple years. One focused on the Swedish registry, one focused on the Danish registry, report that for being a first-degree relative, yes, your likelihood of being a case if your first-degree relative of a TS/CTD case goes up a lot and they both also report risk elevations of your second-degree relative or a third-degree relative of a TS/CTD case. So, in short, what we’re seeing here in these separate studies focused on different populations is essentially clustering of TS and CTD diagnosis within families in a population. And this is consistent with a significant genetic contribution to risk, so given that family-based studies support a genetic contribution, the next question is: Can we identify specific genetic variation that contribute to TS risk? We could do that by using modern DNA genotyping and sequencing technologies to collect genetic data, and then we can conduct formal case-control studies on a large scale on the data collected. So then I guess what’s the point in doing this? So there are two. One is to be able to assess TS genetic risk profiles on a per-patient basis, and then the other point is to produce a map of genes whose activity is perturbed in TS. This will help us understand TS a lot better, and it’ll help us treat it a lot better, potentially on a resolution that is patient-specific.</p>
<p>So, in terms of the work that’s been done so far, first, I’ll talk a little bit about genome-wide association studies, or GWAS, of TS. And when we talk about GWAS, we’re talking in this case primarily about variants that are common in a population and what we’re looking for is we’re looking for instances where variants are found to a genome-wide significant degree in cases more often than controls. And when I say genome-wide significant, this is controlling for if you’re looking at 7 million variants that are common, each of those is a test and you have to adjust your significance threshold based on the number of tests performed. So, this is the significance threshold, so to speak, that the variant has to pass to be significant. Most recent study is from 2019, we have 4,800 cases and 9,500 controls and there’s only one genome-wide significant locus. And when they try to see if it replicates in a separate cohort of around 1,000 cases and 6,000 controls, they’re not able to see a significant case-control difference in a separate cohort, and unfortunately, that’s kind of a standard requirement for implicating a variant in a trait or a disease is seeing if in a separate cohort you see the same thing.</p>
<p>So additional findings. One critical one is for the total common variant contribution to TS heritability. There were two different estimates produced for this total contribution. One is for a subset of cases that were more family history-depleted and one is for individuals that were more enriched for family history. And what they saw is that in terms of the total contribution to TS heritability from common variants, you know there’s a notable contribution in family history-depleted cases, and the number is a bit higher in family history-enriched cases, where you might have siblings that also have Tourette syndrome. So that would suggest that these families where their multiple affecteds have a higher load of risk genetic risks coming from common variation.</p>
<p>And so another thing that the folks that put together this paper were also able to calculate what they call polygenic risk scores for Tourette syndrome in separate samples outside of the case control cohort that they analyzed. And they found it was predictive of Tourette case status, CTD case status, and - this part is pretty important - the recorded worst-ever tic severity. So they’re actually able to use PRS to actually predict just how bad the tics are per individual, so it’s kind of like a treating of it as like a true sort of quantitative trait rather than you have Tourette’s and whenever you don’t have Tourette’s.</p>
<p>So let’s see, next up, we’ll talk about rare variant studies that have been done on TS. So first rare copy number variant studies, or CNV studies, of TS. CNVs, just briefly, are defined as deletions or duplications of DNA sequence in the genome. They tend to be quite a bit more rare and deleterious, and so it makes sense to focus on them in terms of their contribution to genetic risk for Tourette syndrome. In the most recent paper, Huang at al., 2018, has around 2,400 cases and around 4,000 controls. They implicated two separate specific rare CNVs with Tourette risk, and it’s estimated that you’d find these specific CNVs in around 1% of TS cases total. One is deletions overlapping the gene <em>NRXN1</em> and other is duplications overlapping the gene <em>CNTN6</em>. More generally, they saw that, in general, cases have an excess of CNVs relative to controls, specifically very large CNVs. Presumably, the larger the CNV, the more damaging it’s going to be - it’s going to delete or duplicate more genes in the process. And also, there’s an excess in case of relative controls of previously reported pathogenic CNVs, so all this is consistent with a contribution of CNVs to Tourette’s risk and probably converging on specific Tourette’s risk genes.</p>
<p>So another type of rare variant study design that’s been used a lot is whole exome sequencing. Whole exome sequencing involves sequencing the one percent of the human genome that codes for proteins and then one other thing to note about these studies is that they’re often so-called “trio-based.” The goal here ends up being identifying variants that are present in the TS case and absent from both parents, so these kind of variants we refer to as <em>de novo</em> variants and they’re generally the focus of these exome studies, in particular the Tourette’s studies, that we’ll talk about. So I’ll mention two. One is from Willsey et al., 2017, they identified one risk gene that we can highlight here, and that’s <em>WWC1</em>. A more recent paper, a year after that, Wang et al., 2018, identified an additional risk gene <em>CELSR3</em>. What’s key is that in both, they report in excess of damaging <em>de novo</em> variants relative to controls. I will mention briefly, too, that when they look at families that have multiple affecteds, these so-called multiplex families, they actually do not see that excess relative to controls. And so kind of the take home there is we’re talking about polygenic risk, you know the burden of common variants in these multiplex families being higher. What this suggests here is that in those families, common variant risk, the burden of common variance, is higher, but these rare or <em>de novo</em> variants, their burden is lower in these families. So rare variants might have more of an influence in these simplex families without family history, and common variant burden might have more of an influence in multiplex families with multiple affecteds. In general, results from CNV and WES suggest that rare variants can be used to identify specific risk genes for TS and that there are more to be found.</p>
<p>So upcoming TS genetic analysis to highlight. GWAS, there is an analysis being assembled now the total cohort includes 13,500 cases and at least 50,000 controls. CNV, we have an analysis being assembled right now as well and this one’s going to include over 10,000 cases and at least 20,000 controls. So a substantial increase in sample size relative to the studies that we’ve described here before, and the sample size should be adequate for risk variant discovery for both studies.</p>
<p>So summary. TS is a neurodevelopmental condition defined by the presence of a persistent vocal and motor tics that last for at least one year. Consistent with a contribution of genetics to TS and CTD diagnosis are highly concordant between twin pairs and cluster within families in large populations. We see that common genetic variants explain a portion of TS trait heritability in cases relative to non-TS controls and we also see that TS cases are enriched for rare variants that damage protein coding genes. Thank you for listening.</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
          // target, if specified
          link.setAttribute("target", "_blank");
          // default icon
          link.classList.add("external");
      }
    }
});
</script>
</div> <!-- /content -->



</body></html>