<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>PGC Video Textbook - Software Tutorials: Genome Builds (Video Transcript)</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar docked nav-fixed">


<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">PGC Video Textbook</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link active" href="./index.html" rel="" target="" aria-current="page">
 <span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html" rel="" target="">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./contact.html" rel="" target="">
 <span class="menu-text">Contact us</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://pgc.unc.edu/" rel="" target="">
 <span class="menu-text">PGC Website</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item">Software Tutorials: Genome Builds (Video Transcript)</li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
      <a href="./index.html" class="sidebar-logo-link">
      <img src="./pgc_logo_website_v3.jpeg" alt="" class="sidebar-logo py-0 d-lg-inline d-none">
      </a>
      <div class="sidebar-tools-main">
    <div class="dropdown">
      <a href="" title="" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label=""><i class="bi bi-github"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="">
            Source code
            </a>
          </li>
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="">
            Report a bug
            </a>
          </li>
      </ul>
    </div>
</div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./welcome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Welcome to the PGC Video Textbook!</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./toc.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Table of Contents</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Chapters</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter1.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 1: Introduction</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter2.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 2: The Genome</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter3.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 3: Technologies</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter4.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 4: Study Designs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter5.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 5: GWAS Analysis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter6.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 6: Polygenic Scores</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter7.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 7: Ancestry-Specific Analyses and Considerations</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter8.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 8: Post-GWAS Bioinformatics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter9.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 9: Advanced Topics</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter10.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 10: Other Considerations</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true">
 <span class="menu-text">Software Tutorials</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_cnvs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">CNVs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_conditional.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Conditional Analysis</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_datasets.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Datasets</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_ewas.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">EWAS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_geneset.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gene Set Identification</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_liftover.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Genome Builds</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_gwas.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">GWAS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_genomicSEM.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Genomic SEM</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_MR.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Mendelian Randomization</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_mtag.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">MTAG</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_prs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PRS</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./software_correlation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">SNP Heritability and Genetic Correlation</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true">
 <span class="menu-text">Additional Resources</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Glossary</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Tools/Software Resources</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="https://pgc.unc.edu/for-researchers/download-results/" class="sidebar-item-text sidebar-link">
 <span class="menu-text">PGC Summary Statistics</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Software Tutorials: Genome Builds (Video Transcript)</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<p><strong>Title</strong>: Liftover to Map Genomic Coordinates Between Genome Builds with Rtracklayer Bioconductor Package</p>
<p><strong>Presenter(s)</strong>: Saniya Khullar (Biostatistics and Medical Informatics, School of Medicine and Public Health, University of Wisconsin-Madison)</p>
<p>[First of the video has no sound]</p>
<p><strong>Saniya Khullar</strong>:</p>
<p>Hi guys, so today we’ll be talking about liftover analysis. Okay, so in liftover analysis, what happens is that you usually have a different reference build, like hg19 (GRCh37), and you’re trying to convert to hg38 (GRCh38), (hg19 is also sort of like hg37), and you’re trying to convert to a different reference build. So, how do we map from these different genomic coordinate systems? Liftover. Let me kind of show you how that is possible.</p>
<p><em>Looking up reference hg build for Bipolar Disorder Genome-Wide Association Study = hg19 (GRCh37)</em></p>
<p>So, there are different genomic coordinates out there. For instance, if I’m looking up this study on bipolar disorder, it’s a genome-wide association study that is from 2019 for bipolar disorder. So, I navigate to the folder here, and I can see, you know, the first five rows, for instance. So, it’s like chromosome, SNP, base pair, A1, A2. These are all the columns here, the headings. And so, this is a SNP here, and this, the first one is chromosome number and SNP ID, then base pair. But we don’t often know like what is the reference build. So, there are many different reference builds out there. For instance, if you go to this caviar on any site in general,</p>
<p>it’s going to say human reference version freeze is. So, which version is this hg19, which is grch37 or hg38? And they seem very, very confusing. And this is something I also didn’t really know. It’s not something you might think about often, but these are different mappings.</p>
<p>So, if we were to look, for instance, at this example here, let’s just look at this bipolar disorder. So, in other videos, I’ve talked about how you can view the first few rows of the genome-wide association study because there are so many different lines in this 2019. You can see that they’re just so many rows in this file. So you know about 13,413,245 rows, including the header. So, let’s see, like this is chromosome 10 and position 2,394,26. So, the following information that we can gather is that this is chromosome 10. You know, base pairs, a chromosome number, and then base pair. So, we have chr10, and the base pair is 239-2426. So, that’s the first one we can look at. And the next one is just the next row, which is, um, you know, so that’s read.</p>
<p>Okay, so the next row is, um, so this one is corresponding to, um, this over here. So, this one here. Well, let’s see, this is corresponding to this, and then we can also just do the next one, which is just chromosome 10 again. And this position, 2855051. And then we can also take the corresponding snip ids, which is rs. This is the SNP ID here. And then this is the SNP ID here. So, um, for the next one, so let’s just, um, see what happens. So, what we are doing then is I’m sort of taking that and, um, in this information here, and I just sort of put it in here. So, I took the chromosome ken, um, I took the chromosome can, and then this position here, which is what you see, um, which is what you see here. So, I just took this first little information and I put it here.</p>
<p>And then I also put the corresponding rs id right, you know, below it as well. And then I also took chromosome 10 and this one here, and I put it here in this next row along with the corresponding snip id below. So, I want to show you what happens when we try to map it because I don’t know like, um, as of now, I mean, um, I wouldn’t know like what is the reference build.</p>
<p>So, I’m going to start with hg19, and what we’ll see is that this returns chr102392426.</p>
<p>This maps here, this 2392426, yep, this maps, and the RSIDs that were picked up is rs18539560, which actually is the same. So, that’s great. And similarly, chromosome 10 and this position 52855051,</p>
<p>it’s great, and the SNP ID of rs80145673 maps. That’s awesome. And I can assume that that would probably, you know, be the case going on, not just for those two. But why can I just assume that and why is this really huge, you know, that it’s an hg19 they map properly for hg19, you know what we are trying out is hg19 grch37. And I have another video where I kind of, uh, walk through in more depth how to use this site to map, you know, genetic variants and do those conversions. So, hg38, let’s try it now on hg38 and see, using hg38 what we get. So, does it all work out?</p>
<p>Well, okay, we’ll find the SNPs here, the SNP 18. Okay, so we can look up on rs185339560 in hg38 instead. And what we can see is that this SNP is good. Okay. And similarly, the snip rs80145673. Okay, the SNPs are mapping. Now let’s see about the positions. So, this SNP here, the position that was picked up for it is 2350232. Does that actually map here? Does this position actually map with this? It doesn’t. It doesn’t map. This is different, you know, this, this does not, this is not the same, you know, we were trying to see if, you know, there’s a map from here to here, that is definitely not the case. These are not mapping properly. And then in the next example, what we will see is similarly, if we were to look at this snip here, 6,7, rs80145672.</p>
<p>Which one does it map to? You know, this one here. Let me change the color, um,</p>
<p>Okay, so if we look at this one here, this 8281252859, does it match? No, it does not match. It is, it’s not the same. It’s not. And in fact, what you’re going to see is that um, it sort of says that if it were to look up, you know, chromosome 10 position 2392426, if it were to look this position up, which is what we want.</p>
<p>It says it’s not found in hg-38, but it’s found in hg19, and similarly, this position here, chr10:2855. You know, this position here, it’s 55051, is not found in this, but it’s found in hg19. So what that goes to show me is that we… So what we show here is that the same SNPs can be detected in hg19 and hg38. You know, I mean, for the most part, I think they can be detected, but the positions that they correspond to could differ. So, these SNPs are the same. You know, these two SNPs are the same here and here. They map, the positions do not map, and they’re not found in this hg38. So what this goes to show us, what this goes to show us, is, instead, like, it goes to show us that, in reality, the build is probably the build is probably hg19, because these positions, this chr10:239, this is a proper map, this part here is a proper map, this, and this is a proper map for the same SNP. So that’s great. It’s a great map. It’s great. It’s awesome. We’re happy. That’s great. And same with in this other situation with this position, so we see that the SNPs are the same. And, you know, for hg19, it even told us that hg38 even told us that, ‘Hey, you probably are going to find that in hg19.’ So what you’re going to see is that the same SNP can have different genomic coordinates. How do we map between different reference builds (e.g., hg19 and hg38 for humans)? Answer: Liftover.</p>
<p>So, one build is very different from another, and there’s this term that we need, if you need to map between different data files of different types, one file may be in hg19. So, guys, um, just to note, so we have hg19, which is the same thing as GRCh37, and hg38, which is the same thing as GRCh38. So, it’s kind of weird why this is the same and why this is kind of, you know, different, but these are the different builds that they are out there. So how do we map them exactly? So this is a concept that is called liftover. This tool converts genome coordinates. What is liftover? (Mapping between different genomic coordinates) and genome annotation files between assemblies. So, hg19 is a different assembly from hg18. So, basically, um, so this is UCSC, it’s a genomic visualization and interpretations. To motivate that, I want to say is that a common analysis task is to convert genomic coordinates between different assemblies. Probably the most common situation is that you have some coordinates for a particular version of a reference genome, and you want to determine the corresponding coordinates on a different version of the reference genome for that species. For example, you have a bed file that’s one of the input files with exon coordinates for human build GRCh37, which is hg19, and wish to update to GRCh38, which is hg38. So, what you’re wanting to do is you may want to convert from the build GRCh (you can write it either way) 37, which is hg19. You may want to convert it to another file where it’s instead the type of coordinates, so GRCh38, which is hg38. So, I’ve shown to you that the same SNP can have different coordinates because it’s like a different mapping system. It’s like a translation that you need to do. So, you need to do the mapping between them. So, you might need to map in some cases this way. You might need to map it in this direction in some applications. But in other applications, you may need to go the other way. If you’re going in this way, the conversion would be hg19 to hg38. You would need to do this conversion if that’s the direction you’re going in.</p>
<p>But if you’re going the other direction and you’re trying to map it, um, you know, going in this direction, now what you would need to do is you would need to convert from hg38 to hg19. So you would need to do this conversion, um, here if you want to go in that direction.</p>
<p>So this was sort of inspired, and this is a very helpful site. So let’s kind of go over, um, you know, what are reference assemblies? What is a reference assembly, and not just this, we also have reference builds and sequence alignments between humans and canines, which is what this is trying to do, canine family like the dog family. We even have it for cows for different types of mapping. If you want to see are not chromosomal or DNA, what is the coordinate system and where do certain regions tend to match or align properly, like how do you change or adapt the coordinates exactly? And that’s important because some of the genomic association studies, some of the other files that are working with single nucleotide polymorphisms or genetic regions, they can be using different assemblies, so I’m going to talk to you about it and also how you can deal with that.</p>
<p>So let’s go over what a reference assembly actually is. It’s as complete as much as possible representation of the nuclear-type sequence of a representative genome for a specific species. So here we’re focusing on Homo sapiens. HG is what it is. So there are powerful shortcuts when mapping reads as they can be mapped to the assembly rather than to each other to piece the genome of a new individual together. This has a number of benefits, the most obvious of which is that it’s far more efficient than attempting to build a genome from scratch. By its very nature, however, using this approach means that there’s no perfect reference assembly for an individual due to polymorphisms like SNPs. Um, you know, this leads to the publication of new, um, so it’s due to the presence of repetitive structural elements like duplications, inverted repeats, tandem repeats. A given assembly is almost always incomplete and is constantly being improved on, so this leads to the publication of the new assembly versions every so often, such as GRCh37, which is 2009, GRCh38, which is December 2013 for the Human Genome Project. It’s also important, so every few years when data is getting better, they try to, um, you know, update it routinely. So like GRCh37 from NCBI, you know, so different organizations can publish different reference assemblies. So this is coming from the NCBI, while HG19 is coming from UCSC. They are identical, say for a few minor differences such as in the mitochondrial sequence, the naming of chromosome 1 versus chr1. So there are some resources available.</p>
<p>Towards to convert coordinates from one assembly to another. So I’m going to talk about like how you can, um, basically access, you know, a chain file from UCSC, but use this, um, R function and Bioconductor package called R track layer to do the mappings across assemblies. And this is important because of what I showed you where Bipolar Disorder 2019 GWAS (and typically many GWAS files) is in hg18 (GRCh37) when you try looking up in hg38, this is the sit-in positions they are not found. You know a SNP with that location is not found, you know, but in this file. So what this goes to show me is that the Bipolar Disorder, this is instead, it’s hg19 data.</p>
<p>And let’s say I want to convert. I want to go and I want to convert these coordinates here. I want to figure out for all these base pairs, I want to convert each of these to the hg38 build. I want to go through and convert. I want to go through and convert each of these. How do I go about doing that? That’s what I’m interested in, is they’re going and converting it because as you see here what it’s saying is that this information is not found in hg38 but in hg19. So this is hg19; this is hg19 data here. HG19, which is the same thing as GRCh37.</p>
<p>So, we need to convert this; that’s what we need to do. So, um, I’m also going to motivate it here where it says that the conversion rates from hg38 to hg19 average 95 were lower than the conversion rates here. Um, then they talk about a number of significant single nucleotide variants, that’s the same thing as SNPs, could not be converted between this and this. We recommend the newer version, which is hg38. So it came out like, you know, four years later for a next-generation sequencing single nucleotide variant analysis.</p>
<p>So our finding suggests caution when translating single nucleotide variants between different versions of the human reference genome. So we even see what that means. So if based on what I thought a SNP is, if I had gone with, um, if I looked these up in hg19, the coordinates that I get here are very different than what I would get if, instead, I look these up using, so basically if I look these up using hg38, these SNPs have different coordinates. You see this, and this is very different from the same SNP, this and the, I mean the same, that’s the same, so between the builds, you know, this is hg um 19, hg38, the same exact build is there, you know, this is the same, and but what’s different is that these are different, these are not the same, and similarly in the other situation as well where you know chr2855051 for this is very different from what you get, this, this value here is different from this part here, that they’re not the same even for the same SNP, so you have to be very careful.</p>
<p>So let’s see how do we go about doing this given that the build is different, you know, hg38 versus hg19, so let’s go about doing that now. So what this site was talking about is it was talking about doing ucsc liftover, UCSC Genome Browser Resource for Liftover Analysis (Chain Files) analysis as well. So I go here to the website here, so this is, you know, the UCSC, um, University of California’s, um, website here, and I think so it’s hgdownload.soe.ucsc.edu.downloads, our Goal: go from hg19 (GRCh37) to hg38 (GRCh38) conversion for our Bipolar GWAS Data (on SNPs), so I can see this, and what we’re trying to do, our goal here is to go from, you know, for the bipolar 29, a bipolar disorder, um, you know, one of the, um, brain-related, um, disorders, you know, it affects mood stability. So for bipolar disorder, we’re trying to go from hg19, so this is the 2019, um, GWAS, genome-wide study, but trying to go from the hg19 all the way to, which is the same thing as GRCh387, to hc38. So whatever the purpose is that you need to do this conversion, that’s what I’m trying to show you how we can do this. So, um, that’s what our goal is, you know, trying to do.</p>
<p>So, yeah, just a little spiel on bipolar disorder, A little background on Bipolar Disorder (our example application), so it’s still important because it’s a disorder associated with episodes of mood swings ranging from depressive lows to demanding highs. Exact, um, cause is not known but a combination of genetics, environment, and altered brain structure and chemistry may play a role. You know, high energy, reduced need for sleep, and loss of touch with reality and depressive episodes may include symptoms such as low energy, low motivation, and loss of interest in daily activities. Episodes last for days, um, two months at a time and may also be associated with suicidal thoughts. And treatment is usually lifelong and often requires a combination of medications and psychotherapy, so it’s very, very, um, you know, devastating. So manic phase may last weeks or months and depressive, um, phase could also last that long too, so just to bring some awareness as well about what this is. And, um, there’s a genome-wide association study on it that was actually published, 2019 Genome-Wide Association Study on Bipolar Disorder Background and Data, that we are using here. It’s published in Nature and a really good paper, so it’s highly habitable, and they did the genome-wide association study with 20,352 cases and 31,358 controls of European descent, which is why in, um, you know, when I read in this file here, what you’re going to see is that I put in the cases and controls. I sort of put in, okay, twenty thousand three hundred fifty-two cases and, and um, and for controls, uh, 31,358 controls, and this is the total sample size is 46,582 individuals from this study here and that’s when I’m reading. I’m reading in this SNP file because we’re going to be using R, so when I read it in, what we’re going to see is that this is sort of like meeting in. We have the chromosome. Actually, let me also just show you view file, while that is loading I want to kind of show you um, what we do here. So we want to do the hg, Navigating to Chain Files in UCSC Genome Browser (mapping to reference builds of many organisms too!), 19 all the way to hdm38 so we want to go from this to this build. So you can do all sorts of things you can do all sorts of conversions but um, so I’m going to go here and I’m going to click lift over. So left over is like kind of like translating. It’s like a translator for translating the different genetic genomic coordinates, so I’m going to go to liftover files and then it’s going to kind of tell me how I can download all the files if I want to and this is like hg19 to all sorts of things there’s boss tallest this is like</p>
<p>Actually, I’m in the I need to navigate to my downloads so that’s important. You need to be in the right folder. So then I’m just going to go back up and I’m just going to run this um, and I’m just going to say yes because I had done this earlier, you know, before I’m showing this to you guys. So now all you have to do is do gunzip. So what you’re going to see then, you know, is you’re going to actually end up seeing this. You’re going to end up seeing this hg19 to hg38.over.chain. So it’s a chain file. So why do you need that? Well, we’re going to need it to load it into R to do the conversions.</p>
<p>So what the formula is that I, I typically, um, use, um, oh, not the formula, the package, I mean, it comes from our track layer. So our track layer is a package available on Bioconductor, and along with, Introducing RTrackLayer Bioconductor package and liftover, many things that it does is it allows you to change coordinate systems using our track layer leftover. So before I get into that again, I kind of want to show you that basically, whatever you want to download, you can just download it. Like, you can even hg19 to hg18 over, you can do all these things, but with this file, if you click hg19, it’s going to show going from hg 19 to, um, other, um, reference builds, you know. So if instead we wanted to do the other way around, if we wanted to, What if we start with hg38 instead of hg19 data? Viewing Hg38Tohg19.over.chain.gz files and other files starting with hg38 coordinates instead, go back and do we wanted to go back and instead go from, we would choose this. If like, let’s say so we were looking up before hg19 to hg38, but we would choose hg38 to hg19. In fact, they even show this example here, that if you want to go from hg38 to hg19, then you go there, and then you could just download that file, and that’s going to help you go the other way instead, but since we have hg19 and we want to go to hd 38, that’s why we downloaded this file over here, so that we could convert our bipolar disorder genomic association study results, and the results from this study were kind of cool, um, and very informative. So this is what the genomic association, Looking at some results from the 2019 Bipolar GWAS paper (Manhattan plots, etc.), study results look like. These are, um, snip single nucleotide polymorphism so, genomic association studies negative log10 p-value, and then the chromosome for all snips for all single nucleotide polymorphisms, also known as variance snps or variance, which is smvs. So the important thing is that, like, it’s just a single nucleotide so one DNA base, atgc, you know, being different and at least one percent of the population, so it’s quite, um, frequent, so it’s a variation. So this is the genomic association study and the build happens to just be hg19, so that’s why we downloaded it. So our track layer is the package that we will need in Bioconductor, so if you don’t have it, you’re going to need to download that. You’re going to need to download an R track layer and Bioconductor, and you’re going to need to import the chain, so let me kind of show you in the context of, you know, the bipolar disorder example here.</p>
<p>So this is the bipolar disorder. This is, um, you know, like all these different rows, you know, 13. million 413,244 entries, the chromosome, the snip, the base pair, a1, a2, other information, odds ratio, standard error, p-value, which is important, and other things that you may need here as needed. So, um, these are all these are summary statistics that are essentially um, when you see the, basically, just to kind of show you guys again that you see the chromosome, the snip, the position. These are these, each of these dots is a single nucleotide polymorphism, and you take the p-value here from this, Reading it in R (loading in the file), column, and then for these chromosome and numbers, and these are the base pairs, you just sort of like, you know, you take the negative logarithm of this p-value, and that gives you your y value, and then you just continue drawing these dots and then seeing what reaches genome-wide significance, which is usually 5 times 10 to the negative 8. So you can do that, and that</p>
<p>information for the snip or anything but in this example it’s a smith</p>
<p>then i also need to create the start field</p>
<p>which i’m going to say what is the name of the start field well i just called it start field that’s the name of the</p>
<p>column here start field so in quotes you just have to say like what is the name name</p>
<p>of column with start dot field</p>
<p>so this is the start of the chromosome of snap position and in here the star is</p>
<p>the same as the end but we just sort of need to do it because g ranges like it’s not really explicit way to say that hey</p>
<p>this is the same you know star and n so end field is equal to um</p>
<p>n field here so you’re going to just say this is the</p>
<p>end dot field end of snip and this is the same thing you know these are the same</p>
<p>um if it’s easy you could also just have done this it’s equivalent because they’re exactly the</p>
<p>same it’s up to you the width is just one um but if they have more than one then</p>
<p>what would it be then and that’s why i want to show you the end field um you know because what if you have like</p>
<p>different like if you have ranges like enhances the motors uh distal regions other things then how</p>
<p>would you do that and then there’s just one more argument which is</p>
<p>keep dot extra columns and i’m saying true why because i don’t just want to</p>
<p>have my uh my chromosome column start and end i just that’s not all the i just don’t want to have so the basically um</p>
<p>let me sort of show you once i add this and this let me show you um</p>
<p>okay so the essentially i’m telling it to make it based on um</p>
<p>this here and</p>
<p>this part here and it’s going to know it’s going to figure out that hey the width is actually under the same same</p>
<p>same same same it’s the width as one but i just don’t want to have just these three pieces of information into it oh</p>
<p>before i forget i should also kind of i also want to add this in here before i forget um</p>
<p>snips file source equals</p>
<p>just so that we remember</p>
<p>so we want to have all the other information too we want all these other columns even if some of them may be</p>
<p>redundant we want to still keep these you want to have the snips and we want to have the um alleles</p>
<p>um the frequency of each allele the odds ratio we want to just keep that information right now so what i’m going</p>
<p>to do is i’m just going that’s why we say keep extra columns is true because we don’t just want these three we don’t</p>
<p>just want this column um and this and this we also want like the source and all this other</p>
<p>information so we’re just going to say keep extra columns true</p>
<p>might take a little bit of time to make this object but it’s going to be very very efficient and then the next thing</p>
<p>that we’re going to do after that is yeah let’s just see what it did like gr g was snips</p>
<p>ah i put an extra g here so yeah now what we see is that you know</p>
<p>we have the essentials to seek names ranges and strands so this is all that we would have had if we had said false</p>
<p>to this if we said false and we would have just had those columns one more thing i want</p>
<p>to tell you because you might forget about this is this new chromosome column guys this new chromosome column was this</p>
<p>one but what if instead we put chr instead like what if i just forgot to</p>
<p>update it you know we did the updating by having the space uc hr what if we forgot about it and just put chr and you</p>
<p>know we wanted to see how um but actually i’ll do that after so i’m just going to show you so this is</p>
<p>our gr wasp snips so i’m going to show you this one i’m</p>
<p>going to show you this one is a bad one</p>
<p>and i’m going to put in um [Music] the chr here and then show you</p>
<p>what happens so it’s not going to complain right now but i’m just going to show you that like</p>
<p>it’s going to probably like work i i i let him experience that it</p>
<p>doesn’t work so then i’m like wait why isn’t this working and this is my my first time running this package okay so we can try</p>
<p>it the bad way as well where we just don’t label the number and we don’t you know we don’t put chr in front but the</p>
<p>one that works is going to have the chr in front of it so the next thing that we’re go and you</p>
<p>can see is genomic ranges so it has all this information and the g ranges object with everything here and all the</p>
<p>metadata columns it’s really nice so then the thing we’re going to do is</p>
<p>we’re going to then just run left over so i’m going to start with the bad example first and i’m just going to</p>
<p>show you that it’s not going to work</p>
<p>see discarded unchained sequences that’s sort of what happens is that it’s this doesn’t even know what to do</p>
<p>so it’s like i don’t i don’t know it’s like this is retaining zero and then you’ll be like wait why why</p>
<p>isn’t it working and then then you might come to the conclusion that hey this actually uses chromosome</p>
<p>names like this so if you instead try to do it with you know numbers instead of</p>
<p>the chr than the number so it really depends on the format of the file that you get so this does not work so instead</p>
<p>we’re going to need to use the one with chr in front of it we’re going to need to use this format</p>
<p>where the seek names is instead chr10 instead of just the numbers this this</p>
<p>bad one this bad one was a baddie so</p>
<p>it did not work for us so instead what we’re going to do is we’re just</p>
<p>going to try this with the with this on g or was i just want to show it just because you</p>
<p>might again i want to show you kind of what happens and it’s just not me trying to show you a bunch of rules you know</p>
<p>you might have a little panic like me i’m like why isn’t this working and like am i doing something wrong am i breaking</p>
<p>it and then i’m like oh wait i forgot chr i didn’t really think it would be that sensitive</p>
<p>so this hopefully should be the leftover analysis that you get</p>
<p>and you see here i have this old hg19 base here so we can then just see what we’ll end up getting afterwards now</p>
<p>so now okay great there was no error there was nothing like this coming so i was so happy at the time too i’m so</p>
<p>happy okay great</p>
<p>so now we see a basic names is this and because we kept the original we still have this original column here i know a</p>
<p>lot is redundant and because these are huge files you might want to remove redundant things but now what you’re going to see is that</p>
<p>okay this is the same and if you didn’t like um they kind of saved the old one i</p>
<p>mean we have it here which is two three nine but you might be confused so all you need to know is that okay this is</p>
<p>the old base pair for this first row the old one was two you know these end</p>
<p>up being the same but i remember bp is not sometimes as informative so the old one is two three nine two four two six</p>
<p>which is what we were looking at here two three um two three</p>
<p>um nine two four two six that’s the same one here and remember that we were wanting it to be two three</p>
<p>five zero two three two we’re converting from hg19 all the way to this we wanted it to</p>
<p>be this and is that what we get do we get two three five zero two three two</p>
<p>well yes in fact we do we get that that’s that’s awesome so i did the</p>
<p>mapping correctly you know it did that so this is the start and end are the new</p>
<p>coordinates that we are getting here that’s what we wanted this is the same new coordinate that we</p>
<p>were wanting to get that’s awesome</p>
<p>so in essence you can just um you can get rid of end and start it’s just keeping</p>
<p>it there just because you gave start an end and you might have regions where maybe you have a few at least a thousand a few or not at least a thousand you</p>
<p>could have many days pairs of destroying and could be different so what i’m gonna say here is results</p>
<p>nu hg baby8 vp is just results</p>
<p>um you know start and then i’m just going to say head of results so i can</p>
<p>see the we see first five rows and then i can also do dim of results is</p>
<p>to see the dimensions of results</p>
<p>and it kept the original columns which is great so now we have all this information and</p>
<p>group one you know we may have to drop that afterwards but we have what we want we have the mapping and just to also</p>
<p>Interpreting and validating liftover results (sanity checking)</p>
<p>check here so if you recall we also had tried this out for caviar we</p>
<p>had tried out like um that you know hg38</p>
<p>um for this snip zero one four five six seven three</p>
<p>it should be um two eight we said that the position remember the corresponding position it</p>
<p>could not find so let me put it this way just to you know</p>
<p>show this other example too is that this position here</p>
<p>was the old one for the second row so guys just remember that you know we’re looking in the second row here um so</p>
<p>when you’re looking at our output this is fifth second third fourth fifth sixth rows rows rows</p>
<p>but um but what happens is they’re just filling over so you know this is a continuation of the same row there’s so many columns</p>
<p>so this is the second row which is corresponding to this the snip um</p>
<p>corresponding to this neb um you know h014</p>
<p>and what we see here is up here and just write down this again</p>
<p>this one is this here and the one</p>
<p>oh sorry this is hg38 what it has said is that this um okay</p>
<p>with the same snip here sometimes i can use myself i’m like</p>
<p>showing different coordinates and i get mixed up too this part here is like</p>
<p>this here that seems like you’re holding a constant and</p>
<p>this is the old this was the old that was not found</p>
<p>in hg 38 but it was found in hg-19 that’s what we had tried putting in</p>
<p>before for that snip and the new one that they recommended for us was actually they recommended this to us</p>
<p>and when they re recommended this value to us</p>
<p>this is just the same as this it’s just the same thing</p>
<p>that’s exactly what we what we wanted and that’s what we got we got this is exactly the same so this is an example</p>
<p>of how this mapping has worked on multiple and these two different examples</p>
<p>so it’s just allowed us essentially to just do the conversions and i do recommend</p>
<p>you know having this here and having this point here so then you can also see you know um</p>
<p>the say in the start is or at the end is just the same so you can see the conversion of coordinates</p>
<p>so now we can kind of work with either we have what we want we have a given set of snaps we have</p>
<p>we have the old hd base pair information we have</p>
<p>the new hd information and this is just ahead the first three rows</p>
<p>and then i think five or six rows head shows might be kept using python with that and then we also have</p>
<p>um you know the snip ids</p>
<p>as well as i think this is like a deletion this d stands for deletion and that just shows how we can do this</p>
<p>mapping and not just some chromosome can but on all the chromosomes</p>
<p>so that is just an example here of how we</p>
<p>can do this mapping have a given snip we have the old hd 19</p>
<p>base pair now we have the new hg 38 base pair so we have this conversion that we</p>
<p>did here and that’s the best thing ah</p>
<p>so that is just um you know the best thing that we did</p>
<p>we did that conversion so just to revise with you guys again</p>
<p>just very quickly is that um we just like kind of we glowed in a</p>
<p>genomic study um we just have to also load in the chain file which we downloaded and gun zip gu and gun zip it</p>
<p>so once you download it you just do gun steps you go here to just download it</p>
<p>i think you go um let me see</p>
<p>um you go to ucsc site here yeah you know you can go here</p>
<p>to downloads under downloads you know you can check out um you know the browsers and then you can see the human</p>
<p>genome and then you go for whichever one you want to go to so again this is going from hg 38 to any other</p>
<p>reference genome even for cats dogs cows anything and this one is going from hg</p>
<p>19 to any other reference genome and again hg19 is the same thing</p>
<p>as grch37 and this is just the same thing as</p>
<p>grch38 so that’s why you can go to any of them and</p>
<p>you just have to click on the leftover files here or here as needed and then download the</p>
<p>file that you need from here the conversion file</p>
<p>you download it and when you download it all you have to do is just gun zip it</p>
<p>go where it is and do gun zip so make sure you have ubuntu videos on downloading that as well so then you can</p>
<p>just unzip it and then after that um you have to run this code in r</p>
<p>where you know once you load it in you know you have to make sure you add a chr in front of your column so that you end</p>
<p>up using um so you’re going to end up using this data here</p>
<p>on and it actually disappears after so you see how the chromosome</p>
<p>it takes that information um on the start field and enfield and the new chromosome column and</p>
<p>they’re no longer here actually that’s why i was saying to try to store the old base pair information as well and they</p>
<p>try to store it because that might be gone you see like all these columns that we essentially had used here these</p>
<p>column headings they kind of disappear in the final output so sometimes it’s just good to keep track</p>
<p>of the old columns so if you’re doing hg 38 to hd19 if you want to call you want to see if this is hd 38 base pair and i</p>
<p>put old to represent that that’s what we had before and this is the new</p>
<p>so you know you just create the g range with some data frame you need your chain object again and your chain object is</p>
<p>just you importing you know please import the chain um file</p>
<p>and this is just whatever you want it to be he nineteen two um a cow a horse a</p>
<p>mouse you know the mouse genome as well it can be anything so you don’t want this this one is not</p>
<p>good you wanna get rid of that this one is a baddie um it’s it has a chr information</p>
<p>not like numbers oh my story ability’s gone down the ability to draw stars</p>
<p>anyways so you kind of don’t want this you want</p>
<p>this part here your snips file your sequence um names field the name of the column with the</p>
<p>chromosome information the name of the column with the start field name of the column with the end field and then this</p>
<p>could be the same for a snip this can just be exactly the same exactly the same it’s just telling it</p>
<p>how to do it keep extra columns is true if you don’t want to lose this other information um and then what you do is you just run</p>
<p>lift over this is the lift over function again the leftover</p>
<p>and then you put in um your snips file object here this just comes in</p>
<p>this this object comes here that’s all this object comes in here</p>
<p>oh i don’t um red nice and cheerful</p>
<p>this comes here and then the chain object has whichever um you know hg you</p>
<p>know file to whatever</p>
<p>you know that chain file so that’s sort of how this does the conversions</p>
<p>and um and then in the end we have what we wanted and that’s a great you know so</p>
<p>yay we just finished doing the conversion so that’s sort of how i got by with trying to do the conversion and</p>
<p>then still kind of working on that and i just found out about how to do this conversion so i’m just sharing it with</p>
<p>you guys to help you guys out as well if you have similar uh difficulties so how do you change genomic coordinates how do</p>
<p>you do that and that i hope that this helped you guys in the process of doing that</p>
<p>um please let me know if you have any questions um and subscribe if you liked this as well</p>
<p>so thank you guys um and i’m sonia and hope that this is helpful</p>
<p>English (auto-generated)</p>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
          // target, if specified
          link.setAttribute("target", "_blank");
          // default icon
          link.classList.add("external");
      }
    }
});
</script>
</div> <!-- /content -->



</body></html>